% this file is called up by thesis.tex
% content in this file will be fed into the main document

%: ----------------------- name of chapter  -------------------------
\chapter{Pilot sequencing (Sanger method)} % top level followed by section, subsection
\label{pilot-seq}

%: ----------------------- paths to graphics ------------------------

% change according to folder and file names
\ifpdf
    \graphicspath{{3_pilot/figures/PNG/}{3_pilot/figures/PDF/}{3_pilot/figures/}}
\else
    \graphicspath{{3_pilot/figures/EPS/}{3_pilot/figures/}}
\fi

%: ----------------------- contents from here ------------------------

In preparation of high-throughput transcriptome sequencing of the
swimbladder nematode \textit{A. crassus} expressed sequence tags
(ESTs) were generated using traditional Sanger-technology. In total
945 reads from adult \textit{A. crassus} (5 libraries from 4 cDNA
preparations, including 541 sequences generated by students in a
laboratory course) and 288 reads from liver-tissue of the host species
\textit{Anguilla japonica} (3 libraries from 3 cDNA preparations) were
sequenced.

\subsection*{Initial quality screening}
\label{qual-pil}

The initial quality screening of \textit{A. crassus}-sequences
revealed a high number of sequences that had to be discarded due to
failed sequencing reactions (sequences beeing too short after quality
trimming by \texttt{trace2seq}) in the library prepared by
students. For sequences of \textit{Anguilla japonica} and the other
libraries from \textit{A. crassus} failed sequencing reactions were
less common.

In the next screening-step for \textit{A. crassus} 125 (13.23\%) and
for \textit{Anguilla japonica} 64 (22.22\%) of the sequences were
excluded because of homopolymer-runs considered artificial. This
resulted in 452 of the nematode and 195 of the host reads regarded of
sufficient quality for further processing after base-calling and
quality screening.

\subsection*{rRNA screening}
\label{rRNA-pil}

The further screening of sequences revealed a high abundance of rRNA
(see Figure \ref{rRNA-con}) ranging from 71.67\% to 91.67\% of
obtained sequences.  High abundances of rRNA were also found in the
libraries from host liver tissue (see table \ref{tab:num}), ranging
from 71.67\% to 77.42\% . This contamination in libraries from both
species was mainly responsible for a low ammount of sequences beeing
of sufficient quality for submission to NCBI-dbEST. At this point for
the \textit{A. japonica}-dataset 36 sequences were submitted to
NCBI-dbEST under the Library Name ``\textit{Anguilla japonica} liver''
and were assigned the accession LIBEST\_027503 by the curators.

\figuremacro{rRNA-con}{Proportion of rRNA in different libraries for
  \textit{A. crassus} and \textit{A. japonica}}{rRNA abundance as
  proportion of the raw sequencing-reads (rRNA from total) and as
  proportion of the reads after quality screening (rRNA from
  good). Libraries starting with ``Ac\_'' are from \textit{A.crassus},
  libraries starting with ``Aj\_'' are from \textit{A. japonica}.}

% latex table generated in R 2.13.0 by xtable 1.5-6 package
% Mon Sep 19 15:44:26 2011
\small\begin{table}[ht]
\begin{center}
\begin{tabular}{rrrrrr}
  \hline
 & short & poly & rRNA & fishpep & good \\ 
  \hline
Ac\_197F(n=96) &   4 &  17 &  58 &   1 &  16 \\ 
  Ac\_106F(n=96) &  25 &   9 &  48 &   0 &  14 \\ 
  Ac\_M175(n=116) &  30 &  19 &  41 &   3 &  23 \\ 
  Ac\_FM(n=96) &  12 &  29 &  34 &   1 &  20 \\ 
  Ac\_EH1(n=541) & 297 &  51 & 143 &   8 &  42 \\ 
  Ac\_total(n=945) & 368 & 125 & 324 &  13 & 115 \\ 
  Aj\_Li1(n=96) &  10 &  23 &  50 &  &  13 \\ 
  Aj\_Li2(n=96) &  10 &  26 &  43 &  &  17 \\ 
  Aj\_Li3(n=96) &   9 &  15 &  66 &  &   6 \\ 
  Aj\_total(n=288) &  29 &  64 & 159 &  &  36 \\ 
   \hline
\end{tabular}
\caption[Screening statistics for pilot sequencing] {\textbf{Screening
    statistics for pilot sequencing} - Number of ESTs discarded at
  each screening-step for single libraries and totals for
  species. Short, sequence to short in \texttt{trace2seq}; poly,
  sequences with artificial homopolymer-runs from poly-A tails; rRNA,
  with hits to rRNA databases; fishpep with better hits to
  host-protein-databases than to nematode protein databases; good,
  sequences regarded ``valid'' after all screening steps. Note that
  the 13 sequences in the \textit{A. crassus}-dataset, for which
  fish-origin was inferred, were still submintted to NCBI-dbEST.}

\label{tab:num}
\end{center}
\end{table}

\subsection*{Screening for host-contamination}
\label{host-pil}

For the \textit{A. crassus}-dataset screening for host-sequences at
this stage was regarded necessary based on the notion that a big
proportion of the tissue prepared in RNA extraction consisted of
eel-blood inside the gut of the worms (see also Figure
\ref{worm_diff}). Additionally a bimodal distribution of GC-content in
the \textit{A. crassus}-dataset was observed with one of the modes
consistent with the mean GC-content of the ESTs from the Japanese eel.

Comparison of \texttt{Blast}- results for these sequences versus
nempep4 and a fishprotein-database (derived from NCBI non-redundant),
showed that 13 sequences were more likely to originate from host
contamination than from \textit{A. crassus}. These 13 sequences in the
\textit{A. crassus} data-set were submitted to NCBI-dbEST with a
comment, that host origin had been inferred. This reduced the dataset
esentially to 115 ESTs. However it has to be noted that these 13 ESTs
are still accessible through the same library name ``Adult
\textit{Anguillicola crassus}'' and library-identifier LIBEST\_027505
and are taxonomically associated to \textit{A. crassus} on NCBI-dbEST.

\figuremacro{GC-pil}{GC-content of sequences from \textit{A. japonica}
  and \textit{A. crassus}}{The Japanese eel has a slighly higher
  GC-content than the parasite: This sequence characteristic is useful
  for seperation of sequences from the host-parasite interface, note
  the higer GC-content of the sequences from \textit{A.crassus}, for
  which host origin was inferred from similarity searches (red line
  labeled \textit{A.crassus/A.japonica}).}

After screening of host-sequences the GC-content of
\textit{A. crassus} ESTs had a unimodal distribution (see Figure
\ref{GC-pil}). \textit{A. crassus} had a lower mean GC-content $(37.32
\pm 8.36$ mean $\pm$ sd) than \textit{Anguilla japonica} $(45.79 \pm
8.36$ mean $\pm$ sd; two-sided t-test $p<0.001$). The distribution of
the GC-contents for sequences, for which host-origin was inferred was
in agreement with the GC-distribution for host sequences.

\texttt{Blast}-annotations obtained (by similarity searches against
NCBI-nr, bit-score threshold of 55) for the sequences of putative host
origin were also largely in agreement with the expectations for
eel-blood: One sequence could be identified being higly similar to
``Hemoglobin anodic subunit'' from the European eel. Others were
annotated with best hits to highly exressed houskeeping genes from
fish or vertebrates (see table \ref{tab:hostan}). Two sequences in the
set had lower similarities only to proteins predicted from
genome-sequences of Chordates, and one sequence of the 13 lacked any
similarity to NCBI-nr above the treshold of 55 bits.

115 of the sumbitted sequences for ``Adult \textit{Anguillicola
  crassus}'' (LIBEST\_027505) were regarded ``valid'' i.e. not clearly
host origin. 

However it should be noted, that two ESTs (Ac\_EH1f\_01D10 and
Ac\_EH1r\_01D10; forward and reverse read of the same clone) were
annotated with ``ref$|$ZP\_05032178.1$|$; Exopolysaccharide synthesis,
ExoD superfamily'' from \textit{Brevundimonas} sp. BAL3. The family
Caulobacteraceae, comprises bacteria living in oligothroph freshwater
and sequences are probably derived from a commensal, symbiont or
pathogen of eels or swimbladder-nematodes. These off-target data was
left in the submission file.

For 66 (58.4\%) of the remaining 113 ESTs annotations were obtained
from orthologous sequences. All these orthologous sequences were from
other species in the phylum nematoda. 

% Sun Oct  2 07:46:25 2011
\begin{sidewaystable}[ht]
\begin{center}
\begin{tabular}{lrp{3.5cm}lrr}
  \hline
  sequence & hit identifier & hit description &species & bit-score & e-value \\
  \hline
  Ac\_EH1f\_005B07 & gb$|$AAQ97992.1$|$ & cyclin G1 & \textit{Danio rerio} & 67.0 & 9e-10 \\ 
  Ac\_EH1f\_01A02 & gb$|$ACO10003.1$|$ &  Nicotinamide riboside kinase 2 & \textit{Osmerus mordax} &  333 & 1e-89 \\ 
  Ac\_EH1f\_01C10 & gb$|$ADF80517.1$|$ & ferritin M subunit & \textit{Sciaenops ocellatus}  &  328 & 5e-88 \\ 
  Ac\_EH1r\_004A04 & ref$|$XP\_003340320.1$|$ & cytoplasmic 1-like actin & \textit{Monodelphis domestica}  &  102 &3e-20 \\ 
  Ac\_EH1r\_005B07 & gb$|$ABN80454.1$|$ & cyclin G1  &\textit{Poecilia reticulata}  & 90.5 & 8e-17 \\ 
  Ac\_EH1r\_009C03 & ref$|$NP\_001122208.1$|$ & THAP domain containing protein 4 & \textit{Danio rerio}  &  176 & 1e-42 \\ 
  Ac\_EH1r\_01A07 & sp$|$P80946.1$|$ & Hemoglobin anodic subunit beta &\textit{Anguilla anguilla} & 283 &1e-74 \\ 
  Ac\_FMf\_08F03 & ref$|$XP\_003226802.1$|$ & cohesin subunit SA-2-like isoform 2 & \textit{Anolis carolinensis}  &  219 &8e-56 \\ 
  Ac\_M175\_01H02 & emb$|$CAQ87569.1$|$ & NKEF-B protein & \textit{Plecoglossus altivelis}  &  365 &3e-99 \\ 
  Ac\_197Ff\_01E04 & ref$|$XP\_002121150.1$|$ & CUB and sushi domain-containing protein 3 & \textit{Ciona intestinalis} & 80.5 & 2e-13 \\ 
  Ac\_EH1f\_01D07  & ref$|$XP\_002606965.1$|$ & hypothetical protein & \textit{Branchiostoma floridae} & 82.8 &3e-14 \\ 
  Ac\_M175\_01B06 & ref$|$XP\_422710.2$|$ & hypothetical protein & \textit{Gallus gallus} &  123 &1e-26 \\
  \hline
\end{tabular}
\end{center}
\caption[Annotaion of putative host-derived sequences in the
\textit{A. crassus}-dataset]{\textbf{Annotaion of putative
    host-derived sequences in the \textit{A. crassus}-dataset} -
  Sequences excluded because of inferred host-origin comparing
  similarity to nematode- and fish-proteins. The annotation obtained
  against NCBI-nr are in agreement with this inference of host origin,
  as only best hits to vertebrate proteins are found.}
\label{tab:hostan}
\end{sidewaystable}

%%% Local Variables: ***
%%% mode:latex ***
%%% TeX-master: "../thesis.tex"  ***
%%% tex-main-file: "../thesis.tex" ***
%%% End: ***
     
