% this file is called up by thesis.tex
% content in this file will be fed into the main document

%: ----------------------- name of chapter  -------------------------
\chapter{Additional tables and figures}
% top level followed by section, subsection
\label{cha:add-tf}

%: ----------------------- paths to graphics ------------------------

% change according to folder and file names
\ifpdf
    \graphicspath{{9_backmatter/figures/PNG/}{9_backmatter/figures/PDF/}{9_backmatter/figures/}}
\else
    \graphicspath{{9_backmatter/figures/EPS/}{9_backmatter/figures/}}
\fi

\section{Additional tables}
\label{add-table}

\subsection{Transcriptomic divergence in a common garden experiment}

\begin{longtable}{lp{5cm}rrrl}
  \caption[GO-terms enriched in DE between eel-hosts]{\textbf{GO-terms
      enriched in DE between eel-hosts} - The top 10 enriched
    GO-categories are given for genes DE between the different
    eel-hosts.}\\
  \hline
  GO.ID & Term & Annotated & Significant & Expected & p-value \\
\endfirsthead
\multicolumn{6}{c}%
{{\bfseries \tablename\ \thetable{} -- continued from previous page}} \\
\hline
GO.ID & Term & Annotated & Significant & Expected & p-value \\ 
\hline 
\endhead
\hline
\multicolumn{6}{|r|}{{Continued on next page}} \\ 
\hline
\endfoot
\endlastfoot
\hline
  \multicolumn{6}{l}{Molecular function} \\ 
  GO:0004190 & aspartic-type endopeptidase activity &   7 &   2 & 0.03 & 0.00044 \\ 
  GO:0070001 & aspartic-type peptidase activity &   7 &   2 & 0.03 & 0.00044 \\ 
  GO:0030248 & cellulose binding &   1 &   1 & 0.00 & 0.00478 \\ 
  GO:0030600 & feruloyl esterase activity &   1 &   1 & 0.00 & 0.00478 \\ 
  GO:0052689 & carboxylic ester hydrolase activity &  27 &   2 & 0.13 & 0.00694 \\ 
  GO:0045505 & dynein intermediate chain binding &   2 &   1 & 0.01 & 0.00955 \\ 
  GO:0016788 & hydrolase activity, acting on ester bond... & 193 &   4 & 0.92 & 0.01060 \\ 
  GO:0016787 & hydrolase activity & 604 &   7 & 2.89 & 0.01256 \\ 
  GO:0030235 & nitric-oxide synthase regulator activity &   3 &   1 & 0.01 & 0.01429 \\ 
  GO:0044183 & protein binding involved in protein fold... &   3 &   1 & 0.01 & 0.01429 \\ 
  \hline
  \multicolumn{6}{l}{Biological process}  \\ 
  GO:0002478 & antigen processing and presentation of e... &   7 &   2 & 0.04 & 0.00055 \\ 
  GO:0019886 & antigen processing and presentation of e...  &   7 &   2 & 0.04 & 0.00055 \\ 
  GO:0019884 & antigen processing and presentation of e... &   8 &   2 & 0.04 & 0.00073 \\ 
  GO:0002495 & antigen processing and presentation of p... &   9 &   2 & 0.05 & 0.00093 \\ 
  GO:0002504 & antigen processing and presentation of p... &   9 &   2 & 0.05 & 0.00093 \\ 
  GO:0048002 & antigen processing and presentation of p... &  13 &   2 & 0.07 & 0.00199 \\ 
  GO:0019882 & antigen processing and presentation &  15 &   2 & 0.08 & 0.00266 \\ 
  GO:0008219 & cell death & 406 &   7 & 2.16 & 0.00274 \\ 
  GO:0016265 & death & 406 &   7 & 2.16 & 0.00274 \\ 
  GO:0048102 & autophagic cell death &  19 &   2 & 0.10 & 0.00428 \\ 
  \hline
  \multicolumn{6}{l}{Cellular compartment}  \\ 
  GO:0005768 & endosome & 109 &   4 & 0.48 & 0.00094 \\ 
  GO:0043230 & extracellular organelle &   2 &   1 & 0.01 & 0.00880 \\ 
  GO:0065010 & extracellular membrane-bounded organelle &   2 &   1 & 0.01 & 0.00880 \\ 
  GO:0070062 & extracellular vesicular exosome &   2 &   1 & 0.01 & 0.00880 \\ 
  GO:0043025 & neuronal cell body & 105 &   3 & 0.46 & 0.00951 \\ 
  GO:0000323 & lytic vacuole & 106 &   3 & 0.47 & 0.00976 \\ 
  GO:0044297 & cell body & 109 &   3 & 0.48 & 0.01054 \\ 
  GO:0000328 & fungal-type vacuole lumen &   3 &   1 & 0.01 & 0.01317 \\ 
  GO:0061200 & clathrin sculpted gamma-aminobutyric aci... &   3 &   1 & 0.01 & 0.01317 \\ 
  GO:0061202 & clathrin sculpted gamma-aminobutyric aci... &   3 &   1 & 0.01 & 0.01317 \\ 
  \hline\\
\end{longtable}

\begin{longtable}{lp{5cm}rrrl}
  \caption[GO-terms enriched in DE between
  populations]{\textbf{GO-terms enriched in DE between
      worm-populations} - The top 10 enriched GO-categories are given
    for genes DE between the different worm populations.}\\
  \hline
  GO.ID & Term & Annotated & Significant & Expected & p-value \\
\endfirsthead
\multicolumn{6}{c}%
{{\bfseries \tablename\ \thetable{} -- continued from previous page}} \\
\hline
GO.ID & Term & Annotated & Significant & Expected & p-value \\ 
\hline 
\endhead
\hline
\multicolumn{6}{|r|}{{Continued on next page}} \\ 
\hline
\endfoot
\endlastfoot
\hline
  \multicolumn{6}{l}{Molecular function}  \\ 
  GO:0016491 & oxidoreductase activity & 189 &   9 & 1.67 & 1.7e-05 \\ 
  GO:0004129 & cytochrome-c oxidase activity &  17 &   3 & 0.15 & 0.00038 \\ 
  GO:0015002 & heme-copper terminal oxidase activity &  17 &   3 & 0.15 & 0.00038 \\ 
  GO:0016676 & oxidoreductase activity, acting on a hem... &  17 &   3 & 0.15 & 0.00038 \\ 
  GO:0016616 & oxidoreductase activity, acting on the C... &  42 &   4 & 0.37 & 0.00042 \\ 
  GO:0004622 & lysophospholipase activity &   4 &   2 & 0.04 & 0.00044 \\ 
  GO:0016675 & oxidoreductase activity, acting on a hem... &  19 &   3 & 0.17 & 0.00054 \\ 
  GO:0016614 & oxidoreductase activity, acting on CH-OH... &  46 &   4 & 0.41 & 0.00060 \\ 
  GO:0004607 & phosphatidylcholine-sterol O-acyltransfe... &   5 &   2 & 0.04 & 0.00074 \\ 
  \hline
  \multicolumn{6}{l}{Biological process} \\ 
  GO:0034186 & apolipoprotein A-I binding &   5 &   2 & 0.04 & 0.00074 \\ 
  GO:0046688 & response to copper ion &  25 &   4 & 0.24 & 7.3e-05 \\ 
  GO:0006123 & mitochondrial electron transport, cytoch... &  11 &   3 & 0.11 & 0.00012 \\ 
  GO:0010035 & response to inorganic substance & 233 &   9 & 2.23 & 0.00019 \\ 
  GO:0010038 & response to metal ion & 182 &   8 & 1.74 & 0.00020 \\ 
  GO:0008202 & steroid metabolic process &  64 &   5 & 0.61 & 0.00028 \\ 
  GO:0034370 & triglyceride-rich lipoprotein particle r... &   4 &   2 & 0.04 & 0.00052 \\ 
  GO:0034372 & very-low-density lipoprotein particle re... &   4 &   2 & 0.04 & 0.00052 \\ 
  GO:0009408 & response to heat &  76 &   5 & 0.73 & 0.00063 \\ 
  GO:0009266 & response to temperature stimulus & 117 &   6 & 1.12 & 0.00065 \\ 
  \hline
  \multicolumn{6}{l}{Cellular compartment}  \\  
  GO:0034375 & high-density lipoprotein particle remode... &   5 &   2 & 0.05 & 0.00087 \\ 
  GO:0034364 & high-density lipoprotein particle &   4 &   2 & 0.03 & 0.00037 \\ 
  GO:0032994 & protein-lipid complex &   5 &   2 & 0.04 & 0.00061 \\ 
  GO:0034358 & plasma lipoprotein particle &   5 &   2 & 0.04 & 0.00061 \\ 
  GO:0031090 & organelle membrane & 505 &  11 & 4.08 & 0.00078 \\ 
  GO:0044421 & extracellular region part & 174 &   6 & 1.41 & 0.00197 \\ 
  GO:0005576 & extracellular region & 250 &   7 & 2.02 & 0.00258 \\ 
  GO:0005739 & mitochondrion & 605 &  11 & 4.89 & 0.00372 \\ 
  GO:0005743 & mitochondrial inner membrane & 162 &   5 & 1.31 & 0.00807 \\ 
  GO:0031967 & organelle envelope & 313 &   7 & 2.53 & 0.00914 \\ 
  GO:0031975 & envelope & 314 &   7 & 2.54 & 0.00930 \\ 
   \hline
\end{longtable}


%% print(eel.detail, hline.after=1:length(ortho.l[[2]])*3)
% latex table generated in R 2.14.0 by xtable 1.6-0 package
% Sun Dec 11 18:21:30 2011
\begin{longtable}{p{7cm}rrrr}
  \caption[Group-means for OC genes DE between eel
  species]{\textbf{Group-means for OC genes DE between eel species} -
    Group means are given for host combination \textit{An. japonica}
    (Aj) and \textit{An. anguilla} (Aa) with European (EU) and
    Taiwanese (TW) worm populations.}\\
  \hline
  & Aa:EU & Aa:TW & Aj:EU & Aj:TW \\
  \hline
\endfirsthead
\multicolumn{5}{c}%
{{\bfseries \tablename\ \thetable{} -- continued from previous page}} \\
\hline
& Aa:EU & Aa:TW & Aj:EU & Aj:TW \\ 
\hline 
\endhead
\hline \multicolumn{5}{|r|}{{Continued on next page}} \\ 
\hline
\endfoot
\endlastfoot
Contig1005.mean & 518.35 & 630.47 & 1512.31 & 831.26 \\ 
  Cytochrome P450 family protein & 1123.86 & 1204.98 & 2647.29 & 1620.76 \\ 
  ce.ortho.mean & 557.65 & 662.20 & 1658.80 & 1004.08 \\ 
   \hline
Contig12201.mean & 514.90 & 549.58 & 116.02 & 99.56 \\ 
  Lipase family protein & 502.48 & 553.48 & 119.47 & 101.09 \\ 
  ce.ortho.mean & 501.19 & 549.00 & 119.20 & 99.67 \\ 
   \hline
Contig26.mean & 11007.58 & 5406.06 & 3206.43 & 2541.48 \\ 
  Aspartic protease BmAsp-1, identical & 12994.14 & 7671.50 & 4466.98 & 4926.97 \\ 
  ce.ortho.mean & 12670.54 & 7237.48 & 4206.98 & 4402.80 \\ 
   \hline
Contig3754.mean & 490.23 & 901.35 & 922.95 & 663.19 \\ 
  MGC79044 protein, putative & 660.74 & 1110.31 & 1180.48 & 884.49 \\ 
  ce.ortho.mean & 488.55 & 883.91 & 971.48 & 682.95 \\ 
   \hline
Contig3896.mean & 123.17 & 85.71 & 109.09 & 60.18 \\ 
  Transcription factor AP-2 family protein & 119.36 & 86.89 & 111.08 & 59.46 \\ 
  ce.ortho.mean & 119.08 & 85.79 & 111.17 & 58.87 \\ 
   \hline
Contig566.mean & 642.74 & 484.47 & 337.05 & 691.06 \\ 
  Eukaryotic aspartyl protease family protein & 651.38 & 496.17 & 377.95 & 733.26 \\ 
  ce.ortho.mean & 654.89 & 491.93 & 381.14 & 724.47 \\ 
   \hline
Contig6778.mean & 39.00 & 768.10 & 1028.40 & 92.46 \\ 
  Nematode cuticle collagen N-terminal domain containing protein & 621.79 & 1259.66 & 1508.45 & 447.50 \\ 
  ce.ortho.mean & 38.62 & 752.61 & 1056.15 & 95.26 \\ 
   \hline
Contig6934.mean & 449.66 & 639.22 & 632.23 & 572.12 \\ 
  Serine/threonine-protein phosphatase & 788.16 & 1133.91 & 1236.79 & 1041.83 \\ 
  ce.ortho.mean & 448.17 & 628.16 & 663.55 & 591.01 \\ 
   \hline
Contig7580.mean & 240.34 & 1318.57 & 2215.65 & 38.30 \\ 
  Cuticular collagen Bmcol-2 & 286.57 & 1490.40 & 2531.07 & 227.23 \\ 
  ce.ortho.mean & 231.55 & 1298.61 & 2272.71 & 38.23 \\ 
  \hline\\
\label{eel.sing.diff}
\end{longtable}

%% print(pop.detail, hline.after=1:length(ortho.l[[3]])*3)
% latex table generated in R 2.14.0 by xtable 1.6-0 package
% Sun Dec 11 18:22:04 2011
\begin{longtable}{p{7cm}rrrr}
  \caption[Group-means for OC genes DE between worm
  populations]{\textbf{Group-means for OC genes DE between worm
      populations} - Group means are given for host combination
    \textit{An. japonica} (Aj) and \textit{An. anguilla} (Aa) with
    European (EU) and Taiwanese (TW) worm populations.}\\
\hline
& Aa:EU & Aa:TW & Aj:EU & Aj:TW \\ 
\hline
\endfirsthead
\multicolumn{5}{c}%
{{\bfseries \tablename\ \thetable{} -- continued from previous page}} \\
\hline
& Aa:EU & Aa:TW & Aj:EU & Aj:TW \\ 
\hline 
\endhead
\hline \multicolumn{5}{|r|}{{Continued on next page}} \\ 
\hline
\endfoot
\endlastfoot
 Contig13267.mean & 103.86 & 38.57 & 111.01 & 83.54 \\ 
  ABC transporter family protein & 101.36 & 37.67 & 114.79 & 94.25 \\ 
  ce.ortho.mean & 101.74 & 37.76 & 115.19 & 89.28 \\ 
   \hline
Contig157.mean & 362.46 & 394.14 & 369.26 & 449.27 \\ 
  Probable 3-hydroxyacyl-CoA dehydrogenase B0272.3, putative & 361.60 & 378.14 & 381.70 & 545.36 \\ 
  ce.ortho.mean & 362.40 & 367.51 & 380.95 & 504.83 \\ 
   \hline
Contig2099.mean & 289.41 & 327.82 & 367.54 & 556.00 \\ 
  Malate/L-lactate dehydrogenase family protein & 316.68 & 360.99 & 418.67 & 754.71 \\ 
  ce.ortho.mean & 319.36 & 357.47 & 421.73 & 699.56 \\ 
   \hline
Contig236.mean & 266.65 & 164.76 & 183.18 & 840.76 \\ 
  Lecithin:cholesterol acyltransferase family protein & 2797.98 & 2969.10 & 2306.91 & 6119.67 \\ 
  ce.ortho.mean & 2716.28 & 2886.46 & 2225.58 & 5278.32 \\ 
   \hline
Contig2442.mean & 284.39 & 360.83 & 521.53 & 408.18 \\ 
  Putative uncharacterized protein & 782.07 & 1102.11 & 1432.12 & 960.61 \\ 
  ce.ortho.mean & 797.22 & 1131.03 & 1448.22 & 970.06 \\ 
   \hline
Contig2531.mean & 21.38 & 53.89 & 25.65 & 35.20 \\ 
  Cutical collagen 6, putative & 20.78 & 52.54 & 26.07 & 37.82 \\ 
  ce.ortho.mean & 20.86 & 51.95 & 26.08 & 36.53 \\ 
   \hline
Contig3453.mean & 269.89 & 209.33 & 277.53 & 1032.13 \\ 
  Lecithin:cholesterol acyltransferase family protein1 & 2797.98 & 2969.10 & 2306.91 & 6119.67 \\ 
  ce.ortho.mean & 2716.28 & 2886.46 & 2225.58 & 5278.32 \\ 
   \hline
Contig566.mean & 642.74 & 484.47 & 337.05 & 691.06 \\ 
  Eukaryotic aspartyl protease family protein & 651.38 & 496.17 & 377.95 & 733.26 \\ 
  ce.ortho.mean & 654.89 & 491.93 & 381.14 & 724.47 \\ 
   \hline
Contig6043.mean & 1003.44 & 841.34 & 942.26 & 631.00 \\ 
  Putative uncharacterized protein1 & 977.73 & 834.03 & 964.85 & 670.11 \\ 
  ce.ortho.mean & 978.45 & 823.82 & 967.65 & 647.85 \\ 
   \hline
Contig6386.mean & 68.17 & 31.29 & 68.01 & 48.09 \\ 
  Matrixin family protein & 66.79 & 30.60 & 69.64 & 53.52 \\ 
  ce.ortho.mean & 72.76 & 36.38 & 72.47 & 55.31 \\ 
   \hline
Contig6759.mean & 47.39 & 12737.30 & 115.48 & 28013.11 \\ 
  Cytochrome c oxidase subunit 2 & 5647.97 & 19163.28 & 9116.07 & 43335.23 \\ 
  ce.ortho.mean & 5865.67 & 19455.08 & 9437.50 & 41673.94 \\ 
   \hline
Contig6778.mean & 39.00 & 768.10 & 1028.40 & 92.46 \\ 
  Nematode cuticle collagen N-terminal domain containing protein & 621.79 & 1259.66 & 1508.45 & 447.50 \\ 
  ce.ortho.mean & 38.62 & 752.61 & 1056.15 & 95.26 \\ 
   \hline
Contig6934.mean & 449.66 & 639.22 & 632.23 & 572.12 \\ 
  Serine/threonine-protein phosphatase & 788.16 & 1133.91 & 1236.79 & 1041.83 \\ 
  ce.ortho.mean & 448.17 & 628.16 & 663.55 & 591.01 \\ 
   \hline
Contig7580.mean & 240.34 & 1318.57 & 2215.65 & 38.30 \\ 
  Cuticular collagen Bmcol-2 & 286.57 & 1490.40 & 2531.07 & 227.23 \\ 
  ce.ortho.mean & 231.55 & 1298.61 & 2272.71 & 38.23 \\ 
   \hline
Contig8758.mean & 390.97 & 715.11 & 602.46 & 494.53 \\ 
  Protein B0207.11, putative & 383.10 & 687.32 & 626.45 & 510.14 \\ 
  ce.ortho.mean & 389.74 & 701.10 & 633.78 & 511.74 \\ 
   \hline\\
   \label{pop.sing.diff}
\end{longtable}


\clearpage


\section{Additional figures}
\label{add-figures}

\subsection{Pyrosequencing of the \textit{A. crassus} transcriptome}

\figuremacro{tGO_DN_DS_BP_classic_10_all}{GO biological process graph
  for enriched terms in contigs under positive selection}{Subgraph of
  the GO-ontology biological process category induced by the top 10
  terms identified as enriched contigs under positive selection. Boxes
  indicate the 10 most significant terms. Box colour represents the
  relative significance, ranging from dark red (most significant) to
  light yellow (least significant). In each node the
  category-identifier, a (eventually truncated) description of the
  term, the significance for enrichment and the number of DE / total
  number of annotated gene is given. Black arrows indicate a is
  ``is-a'' relationship.}

\figuremacro{tGO_DN_DS_CC_classic_10_all}{GO cellular compartment
  graph for enriched terms in contigs under positive
  selection}{Subgraph of the GO-ontology cellular compartment category
  induced by the top 10 terms identified as enriched contigs under
  positive selection. Boxes indicate the 10 most significant
  terms. Box colour represents the relative significance, ranging from
  dark red (most significant) to light yellow (least significant). In
  each node the category-identifier, a (eventually truncated)
  description of the term, the significance for enrichment and the
  number of DE / total number of annotated gene is given. Black arrows
  indicate a is ``is-a'' relationship.}

\figuremacro{tGO_DN_DS_MF_classic_10_all}{GO molecular function graph
  for enriched terms in contigs under positive selection}{Subgraph of
  the GO-ontology biological process category induced by the top 10
  terms identified as enriched contigs under positive selection. Boxes
  indicate the 10 most significant terms. Box colour represents the
  relative significance, ranging from dark red (most significant) to
  light yellow (least significant). In each node the
  category-identifier, a (eventually truncated) description of the
  term, the significance for enrichment and the number of DE / total
  number of annotated gene is given. Black arrows indicate a is
  ``is-a'' relationship.}

\figuremacro{tGO_EEL_EXP_BP_classic_10_all}{GO biological process
  graph for enriched terms in pyrosequencing-DE genes between
  worm-origin}{Subgraph of the GO-ontology biological process category
  induced by the top 10 terms identified as enriched in DE genes
  between worms from Asia and Europe. Boxes indicate the 10 most
  significant terms. Box colour represents the relative significance,
  ranging from dark red (most significant) to light yellow (least
  significant). In each node the category-identifier, a (eventually
  truncated) description of the term, the significance for enrichment
  and the number of DE / total number of annotated gene is
  given. Black arrows indicate a is ``is-a'' relationship.}

\figuremacro{tGO_EEL_EXP_CC_classic_10_all}{GO cellular compartment
  graph for enriched terms in pyrosequencing-DE genes between
  worm-origin}{Subgraph of the GO-ontology cellular compartment
  category induced by the top 10 terms identified as enriched in DE
  genes between worms from Asia and Europe. Boxes indicate the 10 most significant
  terms. Box colour represents the relative significance, ranging from
  dark red (most significant) to light yellow (least significant). In
  each node the category-identifier, a (eventually truncated)
  description of the term, the significance for enrichment and the
  number of DE / total number of annotated gene is given. Black arrows
  indicate a is ``is-a'' relationship.}

\figuremacro{tGO_EEL_EXP_MF_classic_10_all}{GO molecular function
  graph for enriched terms in pyrosequencing-DE genes between
  worm-origin}{Subgraph of the GO-ontology molecular function category
  induced by the top 10 terms identified as enriched in DE genes
  between worms from Asia and Europe. Boxes indicate the 10 most
  significant terms. Box colour represents the relative significance,
  ranging from dark red (most significant) to light yellow (least
  significant). In each node the category-identifier, a (eventually
  truncated) description of the term, the significance for enrichment
  and the number of DE / total number of annotated gene is
  given. Black arrows indicate a is ``is-a'' relationship.}

\figuremacro{tGO_SEX_EXP_BP_classic_10_all}{GO biological process
  graph for enriched terms in pyrosequencing-DE genes between
  worm-sex}{Subgraph of the GO-ontology cellular compartment category
  induced by the top 10 terms identified as enriched in DE genes
  between female and male worms. Boxes indicate the 10 most
  significant terms. Box colour represents the relative significance,
  ranging from dark red (most significant) to light yellow (least
  significant). In each node the category-identifier, a (eventually
  truncated) description of the term, the significance for enrichment
  and the number of DE / total number of annotated gene is
  given. Black arrows indicate a is ``is-a'' relationship.}

\figuremacro{tGO_SEX_EXP_CC_classic_10_all}{GO cellular compartment
  graph for enriched terms in pyrosequencing-DE genes between
  worm-sex}{Subgraph of the GO-ontology cellular compartment category
  induced by the top 10 terms identified as enriched in DE genes
  between female and male worms. Boxes indicate the 10 most
  significant terms. Box colour represents the relative significance,
  ranging from dark red (most significant) to light yellow (least
  significant). In each node the category-identifier, a (eventually
  truncated) description of the term, the significance for enrichment
  and the number of DE / total number of annotated gene is
  given. Black arrows indicate a is ``is-a'' relationship.}

\figuremacro{tGO_SEX_EXP_MF_classic_10_all}{GO molecular function
  graph for enriched terms in pyrosequencing-DE genes between
  worm-sex}{Subgraph of the GO-ontology cellular compartment category
  induced by the top 10 terms identified as enriched in DE genes
  between female and male worms. Boxes indicate the 10 most
  significant terms. Box colour represents the relative significance,
  ranging from dark red (most significant) to light yellow (least
  significant). In each node the category-identifier, a (eventually
  truncated) description of the term, the significance for enrichment
  and the number of DE / total number of annotated gene is
  given. Black arrows indicate a is ``is-a'' relationship.}

\clearpage

\subsection{Transcriptomic divergence in a common garden experiment}

\figuremacro{tGO_sex_BP_classic_10_all}{GO biological process graph
  for enriched terms in DE according to sex}{Subgraph of the
  GO-ontology biological process category induced by the top 10 terms
  identified as enriched in DE genes between male and female
  worms. Boxes indicate the 10 most significant terms. Box colour
  represents the relative significance, ranging from dark red (most
  significant) to light yellow (least significant). In each node the
  category-identifier, a (eventually truncated) description of the
  term, the significance for enrichment and the number of DE / total
  number of annotated gene is given. Black arrows indicate a is
  ``is-a'' relationship.}

\figuremacro{tGO_sex_CC_classic_10_all}{GO cellular compartment graph
  for enriched terms in DE according to sex}{Subgraph of the
  GO-ontology cellular compartment category induced by the top 10
  terms identified as enriched in DE genes between male and female
  worms. Boxes indicate the 10 most significant terms. Box colour
  represents the relative significance, ranging from dark red (most
  significant) to light yellow (least significant). In each node the
  category-identifier, a (eventually truncated) description of the term,
  the significance for enrichment and the number of DE / total number
  of annotated gene is given. Black arrows indicate a is ``is-a''
  relationship.}

\figuremacro{tGO_sex_MF_classic_10_all}{GO molecular function graph
  for enriched terms in DE according to sex}{Subgraph of the
  GO-ontology molecular function category induced by the top 10 terms
  identified as enriched in DE genes between male and female
  worms. Boxes indicate the 10 most significant terms. Box colour
  represents the relative significance, ranging from dark red (most
  significant) to light yellow (least significant). In each node the
  category-identifier, a (eventually truncated) description of the term,
  the significance for enrichment and the number of DE / total number
  of annotated gene is given. Black arrows indicate a is ``is-a''
  relationship.}

\figuremacro{tGO_eel_BP_classic_10_all}{GO biological process graph
  for enriched terms in DE according to eel-host}{Subgraph of the
  GO-ontology biological process category induced by the top 10 terms
  identified as enriched in DE genes between different host
  species. Boxes indicate the 10 most significant terms. Box colour
  represents the relative significance, ranging from dark red (most
  significant) to light yellow (least significant). In each node the
  category-identifier, a (eventually truncated) description of the term,
  the significance for enrichment and the number of DE / total number
  of annotated gene is given. Black arrows indicate a is ``is-a''
  relationship.}

\figuremacro{tGO_eel_CC_classic_10_all}{GO cellular compartment graph
  for enriched terms in DE according to eel-host}{Subgraph of the
  GO-ontology cellular compartment category induced by the top 10 terms
  identified as enriched in DE genes between different host
  species. Boxes indicate the 10 most significant terms. Box colour
  represents the relative significance, ranging from dark red (most
  significant) to light yellow (least significant). In each node the
  category-identifier, a (eventually truncated) description of the term,
  the significance for enrichment and the number of DE / total number
  of annotated gene is given. Black arrows indicate a is ``is-a''
  relationship.}

\figuremacro{tGO_eel_MF_classic_10_all}{GO molecular function graph for
  enriched terms in DE according to eel-host}{Subgraph of the
  GO-ontology molecular function category induced by the top 10 terms
  identified as enriched in DE genes between different host
  species. Boxes indicate the 10 most significant terms. Box colour
  represents the relative significance, ranging from dark red (most
  significant) to light yellow (least significant). In each node the
  category-identifier, a (eventually truncated) description of the
  term, the significance for enrichment and the number of DE / total
  number of annotated gene is given. Black arrows indicate a is
  ``is-a'' relationship.}

\figuremacro{tGO_pop_CC_classic_10_all}{GO cellular compartment graph
  for enriched terms in DE according to worm-population}{Subgraph of
  the GO-ontology biological process category induced by the top 10
  terms identified as enriched in DE genes between different parasite
  populations. Boxes indicate the 10 most significant terms. Box colour
  represents the relative significance, ranging from dark red (most
  significant) to light yellow (least significant). In each node the
  category-identifier, a (eventually truncated) description of the
  term, the significance for enrichment and the number of DE / total
  number of annotated gene is given. Black arrows indicate a is
  ``is-a'' relationship.}

\figuremacro{sex_all_heat}{Clustering of expression values for contigs
  DE between female and male worms}{A heatmap of variance/mean
  stabilised expression values. Deprograms are based on euclidean
  distance and hierarchically clustering. Green indicates expression
  below the mean, red above the mean. Experimental conditions are
  indicated by black bars for groups of samples (columns) below the
  plot. Presence GO-term annotation for contigs (rows) are given as
  black bars right to the plot: isOxidoreductase = GO:0016491,
  oxidoreductase activity; isMitochondrial = GO:0005739,
  mitochondrion; isELDevelopment = GO:0002164, larval development or
  GO:0009791, post-embryonic development; isResponsetoStim =
  GO:0050896, response to stimulus; isPhosphatase = GO:0016791,
  phosphatase; isMembrane = GO:0016020, membrane; isAntigenProc =
  GO:0002478, antigen processing and presentation of exogenous peptide
  antigen; isEndosome = GO:0005768, endosome; isProtLipComp =
  GO:0032994, protein-lipid complex. Grey bars indicate no annotation
  available.}

\figuremacro{sex_ortho_heat}{Clustering of expression values for OC
  contigs DE between female and male worms}{A heatmap of variance/mean
  stabilised expression values. Deprograms are based on euclidean
  distance and hierarchically clustering. Green indicates expression
  below the mean, red above the mean. Experimental conditions are
  indicated by black bars for groups of samples (columns) below the
  plot. Presence GO-term annotation for contigs (rows) are given as
  black bars right to the plot: isOxidoreductase = GO:0016491,
  oxidoreductase activity; isMitochondrial = GO:0005739,
  mitochondrion; isELDevelopment = GO:0002164, larval development or
  GO:0009791, post-embryonic development; isResponsetoStim =
  GO:0050896, response to stimulus; isPhosphatase = GO:0016791,
  phosphatase; isMembrane = GO:0016020, membrane; isAntigenProc =
  GO:0002478, antigen processing and presentation of exogenous peptide
  antigen; isEndosome = GO:0005768, endosome; isProtLipComp =
  GO:0032994, protein-lipid complex. Grey bars indicate no annotation
  available.}

\figuremacro{eel_all_heat}{Clustering of expression values for contigs
  DE between worms in \textit{An. japonica} and
  \textit{An. anguilla}}{A heatmap of variance/mean stabilised
  expression values. Deprograms are based on euclidean distance and
  hierarchically clustering. Green indicates expression below the
  mean, red above the mean. Experimental conditions are indicated by
  black bars for groups of samples (columns) below the plot. Presence
  GO-term annotation for contigs (rows) are given as black bars right
  to the plot: isOxidoreductase = GO:0016491, oxidoreductase activity;
  isMitochondrial = GO:0005739, mitochondrion; isELDevelopment =
  GO:0002164, larval development or GO:0009791, post-embryonic
  development; isResponsetoStim = GO:0050896, response to stimulus;
  isPhosphatase = GO:0016791, phosphatase; isMembrane = GO:0016020,
  membrane; isAntigenProc = GO:0002478, antigen processing and
  presentation of exogenous peptide antigen; isEndosome = GO:0005768,
  endosome; isProtLipComp = GO:0032994, protein-lipid complex. Grey
  bars indicate no annotation available.}

\figuremacro{eel_ortho_heat}{Clustering of expression values for OC
  contigs DE between worms in \textit{An. japonica} and
  \textit{An. anguilla}}{A heatmap of variance/mean stabilised
  expression values. Deprograms are based on euclidean distance and
  hierarchically clustering. Green indicates expression below the
  mean, red above the mean. Experimental conditions are indicated by
  black bars for groups of samples (columns) below the plot. Below
  contig-names uniprot names are given for ortholog genes in
  \textit{B. malayi}}

%: ----------------------- contents from here ------------------------


%%% Local Variables: ***
%%% mode:latex ***
%%% TeX-master: "../thesis.tex"  ***
%%% tex-main-file: "../thesis.tex" ***
%%% End: ***
