
% this file is called up by thesis.tex
% content in this file will be fed into the main document

\chapter{Materials \& methods} % top level followed by section, subsection


% ----------------------- paths to graphics ------------------------

% change according to folder and file names
\ifpdf
    \graphicspath{{8/figures/PNG/}{8/figures/PDF/}{8/figures/}}
\else
    \graphicspath{{8/figures/EPS/}{8/figures/}}
\fi

% ----------------------- contents from here ------------------------


\section{Sampling of worms from  wild eels}

\subsection{Sampling in Taiwan}

Cultured eels were acquired from an aquaculture directly adjacent to
Kaoping river (22.6418N; 120.4440E) 15km stream upwards from it's
estuary, on the 29th of April 2008. On the same day wild eels were
picked up at Tunkang Biotechnology Research Centre Fisheries Research
institute in Tunkang, Pintung, Taiwan, where they had been sheltered
since the time of purchase during the 2nd two weeks of April 2008 from
a fisherman, fishing in the estuary of Kao-Ping river (22.5074N;
120.4220E). All eels were transported to the Institute of Fisheries
Science at the National Taiwan University in Taipei in aerated plastic
bags, where they were sheltered until dissection.

Dissection of eels was carried out during May 2008. Eels were
decapitated, length (to the nearest 1.0mm) and weight (to the nearest
0.1g) were measured, and sex was determined by visual inspection of
the gonads. The swimbladder was opened, adult worms were removed from
the lumen with a forceps, their sex was determined, and they were
counted. All adult \textit{A. crassus} were preserved in
RNAlater(Quiagen, Hilden, Germany) in individual plstic tubes.

\subsection{Sampling of European worms}

Worms from the European eel were sampled in Sniardwy Lake, Poland
(53.751959N ,21.730957E) by Urszula Weclawski and from the
Linkenheimer Altrhein, Germany (49.0262N; 8.310556E), following a
procedure similar to the one described above for worms from Taiwan.

\section{RNA-extraction and cDNA synthesis for Sanger- and
  454-sequencing}

Total RNA was extracted from single, whole worms using the RNeasy kit
(Quiagen, Hilden, Germany), following the manufacturers
protocol. Alternatively parts of the liver of the host species
\textit{Anguilla japonica}, which also had been preserved in RNAlater
were used for RNA extraction, following the same protocol.

The Evrogen MINT cDNA synthesis kit (Evrogen, Moscow, Russia ) was
then used to amplify mRNA transcripts according to the manufacturers
protocol. It uses an adapter sequence at 3' the end of a a poly
dT-primer for first strand synthesis and adds a second adapter
complementary to the bases at the 5' end of the transcripts by
terminal transferase activity and template switching. Using these
adapters it is possible to specifically amplify mRNA enriched for
full-length transcripts. 

\section{Cloning for Sanger-sequencing}

The obtained cDNA preparations were undirectionally cloned into
TOPO2PCR-vectors (Invitrogen, Carlsbad, USA) and TOP10 chemically
competent cells (Invitrogen, Carlsbad, USA) were transformed with this
construct. The cells were plated on LB-medium-agarose containing
Kanamycin (5mg/ml), xGal
(5-bromo-4-chloro-3-indolyl-$\beta$-D-galactopyranoside) and IPTG
(Isopropyl-$\beta$-D-1-thiogalactopyranosid). After 24h of incubation
at $36\,^{\circ}\mathrm{C} $ cells were picked into 96-well
micro-liter-plates containing liquid LB-medium and Kanamycin (5mg/ml)
and incubated for another 24h. Subsequently 2ml of the cells were used
as template for amplification of the insert by PCR using the primers
\begin{description}
\item[Forward] M13F(GTAAAACGACGGCCAGT) and
\item[Reverse] M13R(GGCAGGAAACAGCTATGACC)
\end{description}
in a concentration of 10$\mu$M. The protocol for PCR cycling is shown
% in table \ref{tab:PCR}.

\begin{table}[h]
  \centering
  \begin{tabular}{lllll} 
    \textbf{Inital denaturation} &  $ 94\, ^{\circ}\mathrm{C} $ & 5min &  &\\ 
    \hline
    \textbf{Denaturation} &  $ 94\, ^{\circ}\mathrm{C} $ &30s& & \\ 
    \textbf{Annealing} &   $ 54\, ^{\circ}\mathrm{C} $ & 45s & 35 cycles &\\ 
    \textbf{Elongation} &   $ 72\, ^{\circ}\mathrm{C} $ & 2min &  &\\ 
    \hline
    \textbf{Filnal Elongation} &   $ 72\, ^{\circ}\mathrm{C} $ & 10min &\\ 
  \end{tabular}   
  \caption{PCR protocol for insert amplification}
  \label{tab:PCR}
\end{table}

Amplification products were controlled on gel and cleaned using SAP
(Shrimp Alkaline Phosphatase) and ExoI (Exonuclease I). Sequencing
reactions were performed using the BigDye-Terminator kit and
PCR-primers (forward or reverse) in a concentration of 3.5$\mu$M and
sequenced on an ABI 3730 DNA Analyzer (Applied Biosystems, Foster
City, California, USA).  For \textit{A. crassus} the following
libraries were prepared:
 
\begin{description}
\item{Ac\_197F:} Female from Taiwanese aquaculture
\item{Ac\_106F:} Female from Taiwanese aquaculture
\item{Ac\_M175:} Male from Taiwanese aquaculture
\item{Ac\_FM:} Female from Taiwanese aquaculture
\item{Ac\_EH1:} Same cDNA preparation as Ac\_FM, but sequenced by
  students in a practical
\end{description}

For \textit{Anguilla japonica} the following three libraries:
\begin{description}
\item{Aj\_Li1:} liver of an eel from aquaculture
\item{Aj\_Li2:} liver of an eel from aquaculture
\item{Aj\_Li3:} liver of an eel from aquaculture
\end{description}

\section{Pilot Sanger-sequencing}

The original sequencing-chromatographs ("trace-files") were renamed
according to the NERC environmental genomics scheme. "Ac" was used as
project-identifier for \textit{Anguillicoloides crassus}, "Aj" for
\textit{Anguilla japonica}. In \textit{Anguillicoloides} sequences
information on the sequencing primer (forward or reverse PCR primer
\textit{Anguilla japonica} sequences were all sequenced using the
forward PCR primer) was stored in the middle
"library"-field, resulting in names of the following form:\\

\texttt{Ac\_[\textbackslash{}d|\textbackslash{}w]\{2,4\}(f|r)\_\textbackslash{}d\textbackslash{}d\textbackslash{}w\textbackslash{}d\textbackslash{}d}\\
\texttt{Aj\_[\textbackslash{}d|\textbackslash{}w]\{2,4\}\_\textbackslash{}d\textbackslash{}d\textbackslash{}w\textbackslash{}d\textbackslash{}d}\\

The last field indicates the plate number (two digits), the row (one
letter) and the column (two digits) of the corresponding clone. For
first quality trimming trace2seq, a tool derived from trace2dbEST
(both part of PartiGene \cite{parkinson_partigene--constructing_2004})
was used, briefly it performs quality trimming using
phred\cite{ewing_base-calling_1998} and trimming of vector sequences
using cross-match\cite{PHRAP}. The adapters used by the MINT kit were
trimmed by supplying them in the vector-file used for trimming along
with the TOPO2PCR-vector. After processing with trace2seq additional
quality trimming was performed on the produced sequence-files using a
custom script. This trimming was intended to remove artificial
sequences produced when the sequencing reaction starts at the 3' end
of the transcript at the poly-A tail. These sequences typically
consist of numerous homo-polymer-runs throughout their length caused
by "slippage" of the reaction.
The basic perl regular expression used for this was:\\

\texttt{/(.*A\{5,\}|T\{5,\}|G\{5,\}|C\{5,\}.*)\{\$lengthfac,\}/g}\\

Where \texttt{\$lengthfac} was set to the length of the sequence
devided by 70 and rounded to the next integer. So only one
homo-polymer-run of more then 5 bases was allowed per 105 bases.

Sequences were screened for host contamination by a comparison of
\texttt{BLAST} searches against the version of nempep4 and a fish
protein database. Sequences producing better bit scores againt fish
proteins than nematode proteins were labeled as host-contamination.

Only the trace-files corresponding to the sequences still regarded as
good after this step were processed with trace2dbEST. Additionally to
the processing of traces already included in trace2seq sequences were
preliminary annotated using \texttt{BLAST} versus the NCBI-NR
non-redundant protein database and EST-submission-files were produced.

\section{454-pyro-sequencing}

\subsection*{Nematode samples, RNA extraction, cDNA synthesis and Sequencing}

\textit{A. crassus} from \textit{JAn. japonica} were sampled from
Kao-Ping river and an adjacent aquaculture in Taiwan as described in
\cite{heitlinger_massive_2009}. Worms from An. anguilla were sampled
in Sniardwy Lake, Poland (53.751959N, 21.730957E) and from the
Linkenheimer Altrhein, Germany (49.0262N, 8.310556E). After
determination of the sex of adult nematodes, they were stored in
RNA-later (Quiagen, Hilden, Germany) until extraction of RNA. RNA was
extracted from individual adult male and female nematodes and from a
population of L2 larvae (Table 1). RNA was reverse transcribed and
amplified into cDNA using the MINT-cDNA synthesis kit (Evrogen,
Moscow, Russia).  For host contamination screening a liver-sample from
an uninfected A. japonica was also processed.  Emulsion PCR was
performed for each cDNA library according to the manufacturerâ€™s
potocols (Roche/454 Life Sciences), and sequenced on a Roche 454
Genome Sequencer FLX. All samples were sequenced using the FLX
Titanium chemistry, except for the taiwanese female sample T2, which
was sequenced using FLX standard chemistry, to generate between 99,000
and 209,000 raw reads. For the L2 larval library, which had a larger
number of non-A. crassus, non-An. anguilla reads, we confirmed that
these data were not laboratory contaminants by screening Roche 454
data produced on the same run in independent sequencing lanes.


\subsection*{Trimming, quality control and assembly}

Raw sequences were extracted in fasta format (with the corresponding
qualities files) using sffinfo (Roche/454) and screened for adapter
sequences of the MINT-amplification-kit using cross-match \cite{PHRAP}
(with parameters -minscore 20 and -minmatch 10). Seqclean
\cite{tgicl_pertea} was used to identify and remove poly-A-tails, low
quality, repetitive and short (<100 base) sequences. All reads were
compared to a set of screening databases using BLAST (expect value
cutoff E<1e-5, low complexity filtering turned off: -F F). The
databases used were (a) a host sequence database comprising an
assembly of the An. japonica Roche 454 data, an unpublished assembly
of An. anguilla Sanger dideoxy sequencesd expressed sequence tags
(made available to us by Gordon Cramb, University of St Andrews) and
transcripts from from EeelBase \cite{pmid21080939} a publically
availble transcriptome database for the European eel; (b) a database
of ribosomal RNA (rRNA) sequences from eel species derived from our
Roche 454 data and EMBL-Bank; and (c) a database of rRNA sequences
identified in our A. crassus data by comparing the reads to known
nematode rRNAs from EMBL-Bank. This last database notably also
contained xenobiont rRNA sequences. Reads with matches to one of these
databases over more than 80\% of their length and with greater than
95\% identity were removed from the dataset. Screening and trimming
information was written back into sff-format using sfffile (Roche
454). The filtered and trimmed data were assembled using the combined
assembly approach \cite{pmid20950480}, combining assemblies from the
mira \cite{miraEST} and newbler \cite{pmid16056220}. ****Give the
details here and we will trim the text later **** . The two assemblies
were combined into one using Cap3 \cite{Cap3_Huang} at default
settings and contigs labeled by whether they derived from both
assemblies or one assembly only.

\subsection*{Post-assembly classification and taxonomic assignment of
  contigs}

After assembly contigs were assessed a second time for host and other
contamination by comparing them (using BLAST) to the three databases
defined above, and also to nembase4, a nematode transcriptome database
derived from whole genome sequencing and EST assemblies
\cite{parkinson_nembase:resource_2004, pmid21550347}. For each contig,
the highest-scoring match was recorded as long as it spanned more than
50\% of the contig. We also compared the contigs to the NCBI
non-redundant nucleotide (NCBI-nt) and protein (NCBI-nr) databases,
recording the taxonomy of all best matches with expect values better
than 1e-05.

\subsection*{Protein prediction and annotation}


Protein translations were predicted from the contigs using prot4EST
(version 3.0b) \cite{wasmuth_prot4est:_2004}. Proteins were predicted
either by joining single high scoring segment pairs (HSPs) from a
BLAST search of uniref100 \cite{pmid18836194}, or by ESTscan
\cite{estscan}, using a training data the \textit{Brugia malayi}
complete proteome back-translated using a codon usage table derived
from the BLAST HSPs, or, if the first two methods failed, simply the
longest ORF in the contig. For contigs where the proein prediction
required insertion or deletion of bases in the original sequence, we
also imputed an edited sequence for each affected contig. Annotations
with Gene Ontology (GO), Enzyme Commission (EC) and Kyoto
Encyclopaedia of Genes and Genomes (KEGG) terms were inferred for
these proteins using Annot8r (version 1.1.1)
\cite{schmid_annot8r:_2008}, using the annotated sequences available
in uniref100 \cite{pmid18836194}. Up to 10 annotations based on a
BLAST similarity bitscore cut-off of 55 were obtained for each
annotation set. The complete \textit{B. malayi} proteome (as present
in uniref100) and the complete \textit{C. elegans} proteome (as
present in wormbase v.220) were also annotated in the same
way. SignalP V4.0 \cite{pmid21959131} was used to predict signal
peptide cleavage sites and signal anchor signatures.

\subsection*{Single nucleotide polymorphism analysis}

We mapped the raw reads against the the complete set of contigs,
replacing imputed sequences for originals where relevant, using ssaha2
(with parameters -kmer 13 -skip 3 -seeds 6 -score 100 -cmatch 10
-ckmer 6 -output sam -best 1). From the ssaha2 output, pileup-files
were produced using samtools
\cite{journals/bioinformatics/LiHWFRHMAD09}, discarding reads mapping
to multiple regions. VarScan \cite{pmid19542151} (pileup2snp) was used
with default parameters on pileup-files to output lists of single
nucleotide polymorphisms (SNPs) and their locations.

% \subsection*{Respiration analysis}
% We extracted contigs annotated with ``GO:0009060 aerobic respiration''
% and ``GO:0009061 anaerobic respiration''

\subsection*{Gene-expression analysis}

For Roche 454 data, read counts for each transcript were obtained from
the mapping to imputed sequence performed for SNP
analyses. Tag-sequences were mapped using BWA \cite{pmid19451168}. And
read counts extracted using Samtools
\cite{journals/bioinformatics/LiHWFRHMAD09}. For deepSAGE
NlaIII-tag-sequencing, total RNA was prepared as described above from
a female nematode from the Polish sampling site. A deepSAGE library
was constructed following the protocol supplied by Illumina. Briefly
after synthesis of cDNA on oligo(dT)-beads, cDNA was digested with the
NlaIII (recognition site CATG), and the oligo(dT)-anchored 3' ends of
mRNAs retained. After ligation of an adaptor containing an MmeI
restriction site, the type II enzyme MmeI was used to cut 17 bases
from the 3' end fragment, genrating a 21 base tag, expected to be
unique for most mRNAs. The R-package DESeq \cite{pmid20979621} was
used to normalize for library size and analyse statistical
significance of differential expression of both Roche 454 and deepSAGE
data. Spearman correlation coefficients were calculated for raw
counts.


%%% Local Variables: ***
%%% mode:latex ***
%%% TeX-master: "../thesis.tex"  ***
%%% tex-main-file: "../thesis.tex" ***
%%% End: ***
 






