
% this file is called up by thesis.tex
% content in this file will be fed into the main document

\chapter{Materials \& methods} % top level followed by section, subsection


% ----------------------- paths to graphics ------------------------

% change according to folder and file names
\ifpdf
    \graphicspath{{8/figures/PNG/}{8/figures/PDF/}{8/figures/}}
\else
    \graphicspath{{8/figures/EPS/}{8/figures/}}
\fi

% ----------------------- contents from here ------------------------


\section{Sampling of worms from  wild eels in Taiwan}

During sampling in Taiwan (sampling locations published in
\cite{heitlinger_massive_2009}) and Germany, single adult
\textit{A. crassus} were preserved in RNAlater(Quiagen, Hilden,
Germany), after their sex had been determined. 


\section{General RNA-extraction and cDNA synthesis}

Total RNA was extracted from single, whole worms using the RNeasy kit
(Quiagen, Hilden, Germany), following the manufacturers
protocol. Alternatively parts of the liver of the host species
\textit{Anguilla japonica}, which also had been preserved in RNAlater
were used for RNA extraction, following the same protocol.

The Evrogen MINT cDNA synthesis kit (Evrogen, Moscow, Russia ) was
then used to amplify mRNA transcripts according to the manufacturers
protocol. It uses an adapter sequence at 3' the end of a a poly
dT-primer for first strand synthesis and adds a second adapter
complementary to the bases at the 5' end of the transcripts by
terminal transferase activity and template switching. Using these
adapters it is possible to specifically amplify mRNA enriched for
full-length transcripts. 


\section{Cloning and Sanger-sequencing}

The obtained cDNA preparations were undirectionally cloned into
TOPO2PCR-vectors (Invitrogen, Carlsbad, USA) and TOP10 chemically
competent cells (Invitrogen, Carlsbad, USA) were transformed with this
construct. The cells were plated on LB-medium-agarose containing
Kanamycin (5mg/ml), xGal
(5-bromo-4-chloro-3-indolyl-$\beta$-D-galactopyranoside) and IPTG
(Isopropyl-$\beta$-D-1-thiogalactopyranosid). After 24h of incubation
at $36\,^{\circ}\mathrm{C} $ cells were picked into 96-well
micro-liter-plates containing liquid LB-medium and Kanamycin (5mg/ml)
and incubated for another 24h. Subsequently 2ml of the cells were used
as template for amplification of the insert by PCR using the primers
\begin{description}
\item[Forward] M13F(GTAAAACGACGGCCAGT) and
\item[Reverse] M13R(GGCAGGAAACAGCTATGACC)
\end{description}
in a concentration of 10$\mu$M. The protocol for PCR cycling is shown
% in table \ref{tab:PCR}.

\begin{table}[h]
  \centering
  \begin{tabular}{lllll} 
    \textbf{Inital denaturation} &  $ 94\, ^{\circ}\mathrm{C} $ & 5min &  &\\ 
    \hline
    \textbf{Denaturation} &  $ 94\, ^{\circ}\mathrm{C} $ &30s& & \\ 
    \textbf{Annealing} &   $ 54\, ^{\circ}\mathrm{C} $ & 45s & 35 cycles &\\ 
    \textbf{Elongation} &   $ 72\, ^{\circ}\mathrm{C} $ & 2min &  &\\ 
    \hline
    \textbf{Filnal Elongation} &   $ 72\, ^{\circ}\mathrm{C} $ & 10min &\\ 
  \end{tabular}   
  \caption{PCR protocol for insert amplification}
  \label{tab:PCR}
\end{table}

Amplification products were controlled on gel and cleaned using SAP
(Shrimp Alkaline Phosphatase) and ExoI (Exonuclease I). Sequencing
reactions were performed using the BigDye-Terminator kit and
PCR-primers (forward or reverse) in a concentration of 3.5$\mu$M and
sequenced on an ABI 3730 DNA Analyzer (Applied Biosystems, Foster
City, California, USA).  For \textit{A. crassus} the following
libraries were prepared:
 
% \begin{list}{\labeldescription}{\leftmargin = 5em}
%   \setlength{\itemsep}{1pt} \setlength{\parskip}{1pt}
%   \setlength{\parsep}{0pt}}
% \item{Ac\_197F:} Female from Taiwanese aquaculture
% \item{Ac\_106F:} Female from Taiwanese aquaculture
% \item{Ac\_M175:} Male from Taiwanese aquaculture
% \item{Ac\_FM:} Female from Taiwanese aquaculture
% \item{Ac\_EH1:} Same cDNA preparation as Ac\_FM, but sequenced by
%   students in a practical
% \end{list}

% For \textit{Anguilla japonica} the following three libraries:
% \begin{list}{\labeldescription}{\leftmargin = 5em}
%   \setlength{\itemsep}{1pt} \setlength{\parskip}{1pt}
%   \setlength{\parsep}{0pt}}
% \item{Aj\_Li1:} liver of an eel from aquaculture
% \item{Aj\_Li2:} liver of an eel from aquaculture
% \item{Aj\_Li3:} liver of an eel from aquaculture
% \end{list}

The original sequencing-chromatographs ("trace-files") were renamed
according to the NERC environmental genomics scheme. "Ac" was used as
project-identifier for \textit{Anguillicoloides crassus}, "Aj" for
\textit{Anguilla japonica}. In \textit{Anguillicoloides} sequences
information on the sequencing primer (forward or reverse PCR primer
\textit{Anguilla japonica} sequences were all sequenced using the
forward PCR primer) was temporarily stored in the middle
"library"-field, resulting in names of the following form:\\

% \texttt{Ac\_[\textbackslash{}d|\textbackslash{}w]\{2,4\}(f|r)\_\textbackslash{}d\textbackslash{}d\textbackslash{}w\textbackslash{}d\textbackslash{}d}\\
% \texttt{Aj\_[\textbackslash{}d|\textbackslash{}w]\{2,4\}\_\textbackslash{}d\textbackslash{}d\textbackslash{}w\textbackslash{}d\textbackslash{}d}\\

The last field indicates the plate number (two digits), the row (one
letter) and the column (two digits) of the corresponding clone. For
first quality trimming trace2seq, a tool derived from trace2dbEST
(both part of PartiGene \cite{parkinson_partigene--constructing_2004})
was used, briefly it performs quality trimming using
phred\cite{ewing_base-calling_1998} and trimming of vector sequences
using cross-match\cite{PHRAP}. The adapters used by the MINT kit were
trimmed by supplying them in the vector-file used for trimming along
with the TOPO2PCR-vector.  After processing with trace2seq additional
quality trimming was performed on the produced sequence-files using a
custom script. This trimming was intended to remove artificial
sequences produced when the sequencing reaction starts at the 3' end
of the transcript at the poly-A tail. These sequences typically
consist of numerous homo-polymer-runs throughout their length caused
by "slippage" of the reaction.
The basic perl regular expression used for this was:\\

\texttt{/(.*A\{5,\}|T\{5,\}|G\{5,\}|C\{5,\}.*)\{\$lengthfac,\}/g}\\

Where \texttt{\$lengthfac} was set to the length of the sequence
devided by 70 and rounded to the next integer. So only one
homo-polymer-run of more then 5 bases was allowed per 105 bases.
Results of this screening were checked by blasting the sequences
excluded as artificial against nempep4, a nematode rRNA database and a
fish-protein database. Two sequences which were identified as false
positives (hitting proteins in nempep4) were moved manually to the
sequences still categorized as good. These were screened against
\textit{Anguillicoloides} rRNA or fish rRNA using
cross-match\cite{PHRAP} with standard parameters for screening.
Finally GS content was tabulated for the sequences intended for
submission and screening statistics were calculated.

After this step sequences were screened for host contamination by a
comparison of BLAST searches against nempep4 and a fish protein
database. Sequences producing better bit scores againt fish proteins
than nematode proteins were removed.

Only the trace-files corresponding to the sequences still regarded as
good after this step were processed with trace2dbEST. Additionally to
the processing of traces already included in trace2seq sequences were
preliminary annotated using BLAST versus the NCBI-NR non-redundant
protein database and a EST-submission-file was produced. This file
parsed for the information on the sequencing primer (stored in the
library-field) and the corresponding primer-entries in the file were
replaced.



 

% ---------------------------------------------------------------------------
%: ----------------------- end of thesis sub-document ------------------------
% ---------------------------------------------------------------------------



 






