
% this file is called up by thesis.tex
% content in this file will be fed into the main document

%: ----------------------- introduction file header -----------------------
\chapter{Introduction}
\label{chap:intro}
% the code below specifies where the figures are stored
\ifpdf
    \graphicspath{{1_introduction/figures/PNG/}{1_introduction/figures/PDF/}{1_introduction/figures/}}
\else
    \graphicspath{{1_introduction/figures/EPS/}{1_introduction/figures/}}
\fi

% ----------------------------------------------------------------------
%: ----------------------- introduction content ----------------------- 
% ----------------------------------------------------------------------

\section{The study organism: \textit{Anguillicola crassus}} 
\label{stud-org}

\subsection{Ecological significance} 
\label{eco-sig}

\textit{Anguillicola crassus} Kuwahara, Niimi and Ithakagi 1974
\cite{kuwahara_Niimi_Itagaki_1974, moravec_anguillicoloides} is a
swimbladder nematode naturally parasitizing the Japanese eel
(\textit{Anguilla japonica}) indigenous to East-Asia. In the last 30
years anthropogenic expansions of its geographic- and host-range to
new continents and host-species attracted interest of limnologists and
ecologists. The newly accquired hosts are, like the native host,
freshwater eels of the genus \textit{Anguilla}, and the use of the
dfinitive host seems to be limited to this genus
\cite{sures_development_1999}. However the nematode displayed a high
versatility and plasticity in most other aspects of it's life, and
this has been proposed as one of the reasons for its success invading
new continents \cite{taraschewski_hosts_2007}.

\textit{A. crassus} colonized Europe in the eraly 1980ies and spread
through almost all populations of the European eel (\textit{Anguilla
  anguilla}) during the following decades (reviewed in
\cite{kirk_impact_2003}). This spread includes populations of the
European eel in North
Africa\cite{gargouri_ben_abdallah_spatio-temporal_2006,
  loukili_dynamics_2007}. At the present day \textit{A. crassus} is
found in all but the northernmost population of the European eel in
Iceland \cite{kristmundsson_parasite_2007}. It has to be noted
however, that low water temperature \cite{knopf_impact_1998} and
salinity \cite{kirk_effect_2000} limit the dispersal of
\textit{A. crassus} larvae and thus high epidemiological prameters are
rather expected in freshwater and in southern latitudes.

Wielgoss et al. \cite{wielgoss_population_2008} studied the population
structure of \textit{A. crassus} using microsattelite markers and
inferred details about the colonization process and history. These
details are in very good agreement with previous knowledge about the
history of introduction and dispersal. Therefore the process of
introduction and spread can be considered very well illuminated:

\textit{A. crssus} was first recorded in 1882 in North-West Germany,
and this record was published in a German fishery magazine in 1985
\cite{fischer_teichwirt}. The import of Japanese Eels from Taiwan to
the habor of Bremerhaven in 1980, was soon identified as most likely
source of introduction
\cite{koops_anguillicola-infestations_1989}. Taiwan as the most likely
geographical source of the introduction was in turn also inferred from
population structure by Wielgoss et al. Furthermore, from the fact
that genetic diversity is highest in northern regions of Germany and
gradually declines to the south, they concluded a single introduction
event to Germany as source for all populations of \textit{A. crassus}
in the comprehensive set of investigated populations of the European
eel. This signal was persistent together with a strong signal for
anthropogenic mixing of eel and parasite populations due to restocking
\cite{pmid20646147}. However a recent study of Laetsch et all !!CITE
found additional haplotypes for Cytochrome C oxidase subunit II in
Turkey, and a second introduction to the Eastern Meditereanean seems
possible. These Turkish haplotypes cluster with Taiwanese haplotypes
and the introduction source would be similar to the main itroduction.

\figuremacro{world-ac}{Transcontinental dispersal of
  \textit{A. crassus:}}{Invasions of different continents by different
  source-populationa are illsutrated using arrows. Red color indicates
  the range of the eel species targeted by the invasion. Modified form
  \cite{mnderle_kologische_2005}, based on data reviewed in
  \cite{kirk_impact_2003} and newer findings in
  \cite{wielgoss_population_2008} and \cite{sasal_parasite_2008}}

A second colonization of \textit{A. crassus}, succeeded in
North-America. Since the 1990s populations of the American eel
(\textit{Anguilla rostrata}) have been invaded as novel hosts
\cite{fries_notes:_1996,barse_exotic_1999,
  barse_swimbladder_2001}. Wielgoss et al. identified Japan as the
most likely source of this American population of
\textit{A. crasssus}. Laetsch et al. CITE!! showed that all source
populations for different intoductions are from one of two clearly
seperated clades of \textit{A. crassus}.

Finally \textit{A. crassus} has been detected in three indigenous
species of freshwater eels on the island of Reunion near Madagascar
\cite{sasal_parasite_2008}.

Copepods and ostracods serve as intermediate hosts of
\textit{A. crassus} in Asia, as well as in the introduced ranges
\cite{moravec_first_2005}. In these hosts L2 larvae develop to L3
larvae infective for the final host. Once ingested by an eel they
migrate through the intestinal wall and via the body cavity into the
swimbladder wall \cite{haenen_effects_1996}, i.a. using a trypsin-like
proteinase\cite{polzer_identification_1993}. In the swimbladder wall
L3 larvae hatch to L4 larvae. After a final moult from the L4 stage to
adults (via a short preadult stage) the parasites inhabit the lumen of
the swimbladder, where they eventually mate. Eggs containing L2 larvae
are released via the eel's \textit{ductus pneumaticus} into it's
intestine and finally into the water \cite{de_charleroy_life_1990}.

\figuremacro{l_cycle}{Life-cycle of \textit{A. crassus}}{Adult females
  deposit already hatched L2 in the lumen of the swimbladder. Larvae
  migrate through the \textit{ductus pneumaticus} and the intestine
  into the open water.Copepodes serve as intermediate host where
  infectiv L3-larvea develop. These can be transported and accumulated
  in paratenic hosts or directly ingested by an eel. They migrate
  through the eel's intestinal wall into the swimbladder wall. After
  the final molt to adults worms arrive in the lumen of the
  swimbladder, feed on blood and reproduce. Modified from
  \cite{mnderle_kologische_2005}.}

One of the possible differences between Asian and European population
of \textit{A. crassus} is the widespread use of paratenic hosts in
European waters \cite{thomas_paratenic_1992,
  pietrock_dynamics_2002}. Such a use of paratenic hosts has not been
reported from the Asian range of the parasite and there are some
speculation that the use and availability of paratenic hosts could be
a factor explaining the success of invasion or even the higher
epidemiological parameters in Europe
\cite{thomas_paratenic_1992}. However the lack of evidence for the use
of paratenic host in Asia could as well be a a result of the lack of
appropriate studies in Asian water systems.

The impact of \textit{A. crassus} on the European eel has been a major
focus of research during the past decades. Pathogenic effects on the
eels such as a thickening \cite{wurtz_tara_2000} and infammation
\cite{beregi_radiodiagnostic_1998} of the swimbladder wall, can lead
to mortality of eels, when combined with co-stressors
\cite{gollock_physiological_2005}. Especially the changes in the
tissue of the swimbladder wall have been shown to influence swimming
behavior and it has been speculated that eel may fail to complete
their spawning migration \cite{palstra_swimming_2007}. Anguillicolosis
(the condition caused by \textit{Anguillicola}) has therefore been
speculated to be a cofactor in the decline of European eel stocks
\cite{sures_science_letter} caused by overfishing of glass-eels
\cite{pmid12713741}.

High prevalences of the parasite of above 70\%
(e.g. \cite{wrtz_distribution_1998}), as well as high intesities of
infections were reported, throughout the newly colonized area
\cite{lefebvre_anguillicolosis:_2004}. In the natural host in Asia
prevalences and intesities are lower \cite{mnderle_occurrence_2006}.

These differences in abundance and intensity of \textit{A. crassus}
infections in East Asia compared to Europe are commonly attributed to
the different host-parasite relations in the final eel host permitting
a differential survival of the larval and the adult parasites
\cite{knopf_differences_2004}. High epidemiological parameters are
attributed to the inadequate immune-response of the European Eel
\cite{knopf_swimbladder_2006}.  While the Japanese eel is capable of
killing larvae of the parasite after vaccination
\cite{knopf_vaccination_2008} or under high infection pressure
\cite{heitlinger_massive_2009}, responses in \textit{An. anguilla}
have hallmarks of pathology. Recently, data from experimental
infections of European eels with \textit{A. crassus} have been
published \cite{fazio_regulation_2008} that show that in this host the
parasite undergoes (under experimental conditions) a density-dependent
regulation keeping the number of worms within a certain range.

Interestingly the differences in the two host also affect the size and
life-history of the worm: In European eels the nematodes are bigger
and develop and reproduce faster \cite{knopf_differences_2004}.

\figuremacro{worm_diff}{Difference between worms in the swimbladder of
  the European eel and the Japanese eel}{Note the bigger size and
  higher number of worm in a typically infected European eel. In
  comparison in the Japanese eel worms are smaller and intensities of
  infection are much lower. The dark brown matter is ingested
  eel-blood visible through the transparent nematode body- and
  intestinal wall, the white matter are developing eggs and larvae in
  ovaries of female \textit{A. crassus}.}


\subsection{Evolutionary significance}
\label{ev-sig}

With a view on the potential co-evolution (i.e. adaptation), of the
eel-hosts to \textit{A. crassus} the katadromous reproduction of
freshwater eels might play an important role. Individuals of both
species \textit{An. anguilla} and \textit{An. japonica} migrate
thouthands of kilometers to reproduce in the area of the Sargasso sea
\cite{pmid19779192}. The Japanese eel in its endemic area migrates to
the west of the southern West Mariana Ridge \cite{pmid20735676}. Eel
larvae then migrate to their freshwater habitates with the help of
oceanic currents. While hybrids between the two Atlantic eel species
have only been reported from Iceland \cite{pmid21299662}, European
eels as a species are considered panmictic \cite{pmid20735687}:
Signals for population structure, interpreted as evidence against
panmixia first \cite{pmid11234011}, have been shown to be an artifact
of temporal variation between cohorts of juvinile eels
\cite{pmid19417764, pmid21299662, pmid16024374}. Such panmixia would
reduce the effectiveness of selection, when unifected populations are
participating in reproduction, making local adaptation impssible.

A decline of epidemiological parameters for European populations of
\textit{A. crassus} has been hypothised based on data published over
two decades \cite{lefebvre_anguillicolosis:_2004}. However this
decline has not been confirmed in an explicit meta-analysis. If it
would be present, possible expanations would include lower population
density of the eel, an evolution of the eel host towards better
resitance, and an evolution of \textit{A. crassus} towards lower
virulence.


!!! Fit here: Memory componetnt of the vertebrate immune system has
been thought to be a driving positive selection on antigenes of
microorganisms \cite{conway_measuring_2002}. The immune systems of
teleost and of eels especially differs in many details from the
mammalian immune system (i.e. it lacks all but the M-class of
antibodies, response to macro-parasites is carried out mainly by
neutrophile rather than eosinophile granulocytes
\cite{nielsen_eel_2006}).

\subsubsection{Interest in \textit{A. crassus} based on its
    phylogeny}
\label{phyl-int}

The genus \textit{Anguillicola} comprises five morphospecies
\cite{taraschewski_revision_1988}: In East Asia in additon to
\textit{A. crassus}, \textit{A. globiceps} Yamaguti, 1935
\cite{yamaguti_globiceps} parasitises \textit{Anguilla
  japonica}. \textit{A. novaezelandiae} is endemic to New Zealand and
South-Eastern Australia in \textit{Anguilla australis} and
\textit{A. australiensis} Johnston et Mawson, 1940
\cite{johnston1940some} parasitizes the long-fin eel \textit{Anguilla
  reinhardtii} in North-Eastern Australia. Finally
\textit{A. papernai} is known from the African longfin eel
\textit{Anguilla mossambica} in Southern Afrika and Madagascar.

In 2006 Moravec promoted the the former subgenus
\textit{Anguillicoloides} comprising all species but
\textit{A. globiceps} to the rank of a genus
\cite{moravec_anguillicoloides}. This subdivision of the
Anguillicolidae in two genera was revised based on the notion that
monophyly of \textit{Anguillicoloides} had to be rejected,
\textit{Anguillicolides crassus} was restored to \textit{Anguillicola
  crassus} in CITE!! Laetsch. In the same study on the phylogeny of
the \textit{Anguillicolidae} \textit{A. crassus} was identified as the
basal species in the genus, analysing nuclear genes SSU and LSU (see
figure \ref{nLSU-phylo}) or as forming a clade with the oceanic
species with \textit{A. globiceps} and \textit{A. papernai} in a
sister clade (see figure \ref{mCOXI-phylo}).

\figuremacro{nLSU-phylo}{Phylogeny of the genus \textit{Anguillicola}
  based on nLSU}{Phylogram infered from large ribosomal subinit of
  \textit{Anguillicola} and outgroups using Bayesian Inference. Lables
  on internal branches indicate Bayesian posterior
  probabilities. From Laetsch et al. CITE!!}

\figuremacro{mCOXI-phylo}{Phylogeny of the genus \textit{Anguillicola}
  based on COXI}{Phylogram infered for \textit{Anguillicola} based on
  mitochondrial Cytochrome C oxidase subunit I and outgroups using
  Bayesian Inference. Lables on internal branches indicate Bayesian
  posterior probabilities. From Laetsch et al. CITE!! } 

Neiter of these phylogenetic hypotheses is consistent with the
phylogeny of the eel-hosts without host-switching: Assuming the
establishment of \textit{Anguillicola} in an ancestral Indo- pacific
host at least three host-switch events are needed, even to explain
classical (non-recent, non-anthropogenic) host-parasite associations.
Two of these host-capture events must have spanned the major splits in
the eel phylogeny \cite{minegishi_molecular_2005}: Oceanic
\textit{Anguillicola} must have captured hosts transitioning between
the clade of \textit{An. reinhardtii} and \textit{An. japonica} to the
clade in which \textit{An. australis} is found. Finally the the most
basal have species of frehwater eels \textit{An. mossambica} must have
been captured.

The recent anthropogenic host-switch of \textit{A. crassus} from
\textit{An. japonica} to \textit{An. anguilla} and
\textit{An. rostrata} constitues additional acquisition of
phylogenetically well separated hosts. This affinity for
host-switching may be an evolutionary relict found only in one clade
of \textit{A. crassus} !!CITE Laetsch.

The to date most likely phylogenetic hypothesis places the genus
\textit{Anguillicola} (the only genus in the family Anguillicolidae)
at a basal position in the Spirurina (clade III \textit{sensu}
\cite{blaxter_molecular_1998}), one of 5 major clades of nematodes
\cite{nadler_molecular_2007, wijov_evolutionary_2006}. The Spirurina
exclusively exhibit a animal-parasitic lifestyle and comprise
improtant human pathogens as well as prominent parasites of livestock
(e.g. the Filaroidea and Ascarididae). The finer subdivision of the
Spirurina into Spirurina A, and the Sister clades Spriurnina B and C
from Laetsch et al. can be seen in figure \ref{clade-phylo}.

Within the Spirurina B an enormous phylogentic diversity of the
definitive hosts can be observed ranging from fresh-water fish as
hosts for the Anguillicolidae to cartilaginous fish for
Echinocephalus, mammals parasitized by Gnathostoma and Linstowinema to
reptiles as hosts for Tanqua. In addition to this diversity, a common
characteristic of Spirurina A and C is a complex life-cycle involving
freshwater or marine intermediate hosts. The observation of these
complex traits render the assumption of evolution of the parasitic
Spirurina from a free-living ancestor less parsimonous.

\figuremacro{clade-phylo}{Phylogeny of nematode clade III based on
  nSSU}{Phylogram inferred from nuclear small ribosomal subunit for
  Spirurina using Bayesian Inference. Branches are collapsed to
  highlight major groups. Lables on internal branches indicate
  Bayesian posterior probabilities. From Laetsch et al. CITE!!}

This phylogenetic position makes the Anguillicoloidae an interesting
system in the endeavour to understand the emergence of parasitism in
Spirurina and as an ``outgroup'' for functional studies of parasitism
in this clade.

\subsubsection{Divergence of \textit{A. crassus} populations}
\label{div-ac}

Common-garden experiments (also termed ``transplant expreiments'') are
a method to identify genetic components of phenotypic differences
between potentally diverged population of a species, used for almost
as long as scientists investigate evolution
\cite{kerner_classic_common_garden, bonnier_classic_common_garden}. In
the reciprocal version of these experiments, representatives of each
population intented to be studied are raised in the other population's
natural environment. A modification of this would be to raise each
population in an experimental setup under conditions resembling the
environment of the other populaton.

When applied to parasites infecting two different hosts such an
experiment can be best described as ``cross-inoculation experiment
under common garden conditions'' \cite{kaltz_shykoff_rev}. In a recent
study using this method both European and Japanese eels were infected
under laboratory conditions with worms from three geographic origins;
Southern Germany, Poland and Taiwan.

In these experimets differences between the two European populations
and and the Taiwanese population of worms manifested. Differences were
especially (but not solely) visible in the early stages of the
life-cycle. In the European eel the number of L3 larvae from
the Taiwanese population of worms was higher than from European
worms. From the Taiwanese population less L4 larvae were
observed at 25 dpi and the levels of this larval stage were stable
during the infection, in contrast the numbers of L4 for the European
populations decreased with the time. Additionally up to 50 dpi there
were less living adults observerd for worm from the Taiwanese
population and fewer dead adult worms were recorded for the
Taiwanese population beginning from 50 dpi.

In the Japanese eel fewer L3 larvae at 25dpi were observed from the
Taiwanese population compared to the European population of
worms. Additioally more L4 larvae at this point in time and fewer
living adults bat 25 and 150 dpi, as well as fewer adults beginning
from 50 dpi from wroms of Taiwanese origin compared to worms of
European origin.

\figuremacro{ula_stages}{Differences in developmental speed}{three
  populations of \textit{A. crassus} (rows) were raised in two
  different hosts (column). Bars represent means of recovered
  individuals from three life-clycle stages. Differences are pointed
  out in the main text. Data courtesy of Urszula Weclawski.
}

These findings taken can be consolidated to the interpretation that an
increase in the speed of development was observed in the European
populations of \textit{A. crassus} compared to the Taiwanese source
population.

Interpretation of morphological characters in these studies proved
difficult. Size of the worms seems to be mainly determined by the
uptake of host-blood and is thus largely object to phenotypic
modification, with a genetic component hard to detect.

\section{DNA sequencing}

\subsection{A very short history of high-throughput DNA-sequencing}
\label{his-seq}

For almost tree decades the method developed by Frederick Sanger
\cite{pmid271968} was the only practical choice for determining the
sequence of nucleic acid. The method uses specifficlly labeled (first
radioactive lables were used later fluorescent) chain termination
nucleotides. If such a molecule is incorporated into a strand of DNA,
synthesis stops and the length of the partial DNA-sequence can be
determined on a single-base resolution agarose gel along with the
corresponding base at that position. Although modern machines use the
chain-termitaion method combined with capillary gel electorphoresis
\cite{pmid2326186} in a highly paralized way, costs and labour
constrained sequencing to a well established laboratory-model
organisms. In addition to the sequencing reaction itself, the need for
cloning into DNA vectors for purification and amplification made costs
and labour associtated with this method prohibitive for a large scale
application in non-model organisms. After phages \cite{pmid1264203} in
the first years of DNA sequencing. The bacterium \textit{Escherichia
  coli} in 1997 \cite{pmid9278503}, the baker's yeast
\textit{Saccharomyces cerevisiae} in 1996 \cite{pmid8849441}, the
nematode \textit{Caenorhabditis elegans} in 1998 \cite{pmid9851916},
the fruit fly \textit{Drosophila melanogaster} in 2000
\cite{adams2000genome} and the mouse \textit{Mus musculus} in 2002
\cite{pmid12466850} were the model orgaisms, for which multi-national
consortia sequenced genomes in multi-million dollar projects. This
``first generation of genomics'' culminated in the publication of the
human genome in 2001 \cite{pmid11181995}.


\subsection{DNA-sequencing in Nematodes}
\label{sec:dna-sequ-nemat}

In 1998 \textit{Caenorhabditis elegans} had become the first
multicellular organism with a sequenced genome
\cite{pmid9851916}. Soon it was noted, that in addition to it's use as
a general model system for the metazoa and bejond, knowledge gained in
this species has the potential to be even more valuable in the phylum
nematoda \cite{blaxter_caenorhabditis_1998}. The breadth and detail of
genomic information available for \textit{C. elegens} to date is
illustrated by a recent publication of the Gerstein et
al. \cite{pmid21177976} providing detailed annotation of the diverse
functional genomic elements at single base resolution and their
interactions.

The genome sequence of \textit{Caenorhabditis elegans} was soon
complemented by the gneome of \textit{Caenorhabditis briggsae}
\cite{stein_genome_2003}, a second nematode from the genus
\textit{Caenorhabditis} sequenced a satellite system for comparative
genomics instide this genus. As a second satellite model in clade V
the necromenic \textit{Pristionchus pacificus} (living in close
association with beetles) has a published draft genome
\cite{pmid18806794}.

The first published genome of a parasitic nematode in the Spirurina
was the draft genome of \textit{Brugia malayi}
\cite{ghedin_draft_2007}. As a second gneome in the Spirurina recently
the genome of \textit{Ascaris suum} \cite{pmid22031327}.

Also in the remaining clades of the nematoda genome sequencing
folorished: For the animal-prasite \textit{Trichinella spiralis} from
clade I \cite{pmid21336279}, the plant parasites \textit{Meloidogyne
  incognita} \cite{pmid18660804} and \textit{Meloidogyne hapla}
\cite{pmid18809916} as well as the the pinewood nematode
\textit{Bursaphelenchus xylophilus} \cite{pmid21909270} (a plant
parasite using a beetle as an vector) from clade IV have recently
genome sequences have been pulished.

The current revolution in sequencing methodology (see
\ref{sec:ad-seq}) brings into sight many more sequenced nematode
genomes (including that of \textit{A. crassus}). The 959 nematode
genomes initiative promotes such sequencing of nematode genomes and
makes working-drafts of genome-assemblies available for analytical
purposes in a \texttt{blast}-server \cite{pmid22058131} .

Before the advent of NGS the lack of genomic information in many
species of nematodes promoted the use of ESTs as a tool for
gene-discovery. Partial genomes \textit{sensu}
\cite{parkinson_partigene--constructing_2004} were successfully
interrogated for a large array of genes interesting for various
scientific communities. In nematode parasites of vertebrates,
pathogenic factors were described as potential vaccine candidates
\cite{pmid11406138}.

Cystein-proteinase inhibitors (cystatins) and serin protenase
inhibitors (serpins) are thought to interact with the antigen
presentation in vertebrat hosts \cite{pmid11406138}. Homologues of
mammalian cytokines were identified, which are believed to interact
with mammalian cytocine receptors to divert the immune response to a
TH2-type response \cite{maizels_helminth_2004} (an anti-inflammatory,
rather cellular response, thought to be non-effective against
helmiths). Further molecules involved in host-parasite interaction,
which have been identified in transcriptome-projects include abundant
larval transcripts of \textit{B. malay} (Bm-ALT)
\cite{gomez-escobar_abundant_2002} and venom like allergens (Bm-VLA)
\cite{pmid11704277}.

In some of these studies secreted proteins were in the center of
interest. They could potentially be excreted by the nematode to allow
movement and food-uptake bot also to interact with the host's immune
system. The detection of signal-peptides for secretion using
\textit{in silico} analysis of ESTs has been used to highlight
candidate genes for example in \textit{Nippostrongylus brasiliensis}
\cite{harcus_signal_2004}, and across all nematode ESTs
\cite{nagaraj_needles_2008}. Proteomic analysis in \textit{Brugia
  malayi} \cite{pmid19352421,pmid18958170}, \textit{Heligmosomoides
  polygyrus} \cite{pmid21722761} and \textit{Haemonchus contortus}
\cite{pmid12576473} was able to find evidence for excretion for some
of the protein-products and to highlight additional candidate genes.

Obviousely NGS also leaves it's marks currently in nematode
transcriptomics \cite{pmid22044053}.


!!! FIT: 
That positive or diversifying selection on parasite proteins from the
host-parasite interface can lead to a overabundance of non-synonymous
changes (altering the protein sequence) over synonymous polymorphisms
e.g. in \textit{Plasmodium} \cite{pmid7630387}.

\section{Advances in sequencing technology}
\label{sec:ad-seq}

Advances in sequencing technology (often termed ``Next Generation
Sequencing''; NGS), provide the opprotunity for rapid and
cost-effective generation of genome-scale data. 

The technologies portrayed here and used in the work underlying this
thesis is, that - like sanger sequencing - can't work on single
molecules and thus target molecules have to be amplified. This
amplifiction has to produce spatially separated templates and
immobilistation on a solid surface to archive this clonal amplifiction
is used in preparation of both pyrosequencing and for the
illumina-platform \cite{pmid19997069}. The implemetnation of this in
each technology will be explaind in the corresponding subchapter.

One cumbersome aspect of the need for amplifiction is the high amount
of DNA starting-material (3–20 $\mu$g) required
\cite{pmid19997069}. Other disatvatages include, that mutations during
clonal amplificateion in templates can disguise error as sequence
variants. Nucleotide compositon of the target may also introduce
amplification bias and thus biased product yield
\cite{pmid19327155}. This in turn leads to underrepresentation of
cerain molecules, most detrimental in quantitative applications, such
as RNA–seq \cite{pmid19015660}. However, while alternative single
molecule approaches exist (\cite{pmid21431759, pmid21572978} and can
be applied to address the above stated the problems
\cite{pmid21431761, pmid21957044}), to date thes technologies are in
throughput and relieblity not competitive for most real life
applications.

\figuremacro{sequencing_costs}{Falling seqeuncing costs}{Sequencing
  costs falling due to advances in Solexa-sequencing: Due to improved
  read-length and data-volume on this plattform per base
  sequencing-prices for many applications thumble into free fall. Data
  provided by \href{http://www.genome.gov/sequencingcosts/}{National
    Human Genome Research Institute, NHGRI}.}

The sequencing reaction itself differes between platforms, but the
technologies described as NGS have in common that they use a different
chemistry comapared to the Sanger-method. Up to date all practicable
method produce much more, but shorter reads than classical sanger
sequencing.

This fostered use and development of new methods to assemble
large-scale shotgun sequences, as higher coverage but shorter
read-length (and also lower accuracy) are increasing the computational
complexity of the assembly-problem (reviewed in \cite{pmid20211242}).

\subsection{Pyro-sequencing}
\label{sec:pyro-seq}

\figuremacro{454}{Schematic representation of pyrosequencing
  reaction}{ (a) DNA (genomic or transcriptomic) is isolated,
  fragmented, ligated to adapters and denaturated into single
  strands. (b) Under conditions that favor one fragment per bead
  fragments are bound to beads. These beads are isolated and
  compartmentalized in the droplets of an emulsion and PCR (a mixture
  of reagents in oil). Whitin each droplet DNA is amplified, and beads
  are obtained which carrying millions of copies of a unique DNA
  template. (c) After denaturation of DNA, beads are deposited into
  wells of a fiber-optic slide (called picolitre plate). (d)
  Immobilised enzymes carried on smaller beads are added to each well
  and a solid phase pyrophosphate sequencing reaction is
  initiated. (e) A portion of a fiber-optic slide, in a scanning
  electron micrograph (prior to bead deposition) (f) Major subsystems
  of the 454 sequencing instrument: a fluidic assembly holding
  nucleotides seperately (object i), the well-containing
  picoliter-plate in a flow cell (object ii), a CCD camera assembly
  and the user interface for instrument control (object iii)
  \cite{pmid18846085}} 


Pyrosequencing (or 454-sequencign; named by the company makin it
commercially available) uses emulsion PCR to apmlify single DNA
molecules attached to beads after fragmentation by mechanical shearing
or ultrasound \cite{pmid20298868} (see figure \ref{454}). This covers
each bead with multiple copies of one target molecule. The beads are
then distributed over the wells of a fiber-optic slide, the so called
picolitre plate. A single bead per well is covered with enzymes on the
surface of smaller beads. These enzymes are used in the actual
pyrosequencing reaction originally developed by P\r{a}l Nyr\'{e}n in
the 1990s \cite{pmid17185753}. The release of inorganic PPi as a
result of nucleotide incorporation by polymerase starts a cascade of
enzymatic reactions. The released PPi is converted to ATP by ATP
sulfurylase, providing energy for luciferase to oxidize luciferin and
to generate light. The added nucleotide is known as nucletoides are
flushed over the plate one at a time. A high resolution camera records
the emission of light. The intensity of emmited light is proprotional
to the number of nucleotides incorporated. The ability to distinguish
length of homopolymeric runs of the same nucleotide decreases with the
length of such homopolymer runs \cite{pmid21685085}. Current
``Titanium chemistry'' is producing read of ~400 bases length, ``FLX
chemistry'' (used up to 2009) was able to produce reads of ~250 bases
length \cite{pmid21514329}.

This longer read length of 454-sequencing \cite{pmid16056220} compared
to other NGS technologies (see \ref{sec:ill-seq}), allows \textit{de
  novo} assembly of Expressed Sequence Tags (ESTs) in organisms
lacking previouse genomic or transcriptomic data (for a comprehensive
list of studies using this approach before Oct 2010 see
\cite{pmid20950480}).

\subsection{Illumina-Solexa sequencing}
\label{sec:ill-seq}

Solexa illumina technology is to date (Dec. 2011) the most competitive
commercial sequencing platforms enabeling a broad spectrum of
applications.

\figuremacro{ilmn_all}{Schematic representation of illumina
  sequencing}{(a) DNA (genomic or transcriptomic) is isolated,
  fragmented and ligated to adapters. (b) Single strandedfragments are
  bound to a glass-slide. (c-d) Solid-phase bridge amplificatin using
  unlabled nucleotides, primers (binding the adapters) and polymerase
  leaves clusters of double stranded DNA distributed over the slide.
  (e) four lableled reversible terminators, primers (binding the
  adapters) and polymerase are added. Laser excitation an image of the
  emmited fluorescence is taken . Step (e) is repeated multiple times
  (=length of sequence)}

The Illumina-Solexa platform uses bridge amplification to produce
copies of single DNA molecules in clusters on a glass slide. 





These
clusters are then sequenced usign a sequencing by synthesis technique:
``removable termitator'' nucleatodes emitting a base specific
fluorescence are flushed over the class slide transient incorproation
is detected using a high resolution camera. This leads to an error
model different from 454 sequencing: Homoploymer runs are
non-problematic, but due to the decreasing propensity of terminators
for removal, sequencing quality decrases in from 5' to 3' direction.





Recent increases in read length (from 35 bases in 2008 to over 100
bases in 2011 ) are beginning to allow \textit{de novo} sequencing of
large genomes !!! CITE (panda) and transcriptomes !!!CITE. In the same
periode throughput also increased from ~6,000,000 reads in 2008 to
~20.000.000 reads in 2011 per lane of the instrument.

The high throughput of the Illumina-Solexa platform makes it also
first choice for gene expression analyis \cite{pmid21627854}:

RNA-seq \cite{pmid19015660} 

SuperSAGE \cite{pmid20967605} using expression-tags provides the
benefit of classical SAGE-analysis \cite{pmid7570003} with those of
the ulta hight throughput of Illumina-Solexa
sequencing. normalisations



\subsection{Computational methods in DNA-sequence analysis}
\label{sec:comp-meth-dna}

In this context a common characteristic of all DNA-sequencing methods
has to be emphasized: Read-length is usually shorter than the length
of the target molecule to be sequenced. This potential problem is
solved by oversampling the target molecule, producing overlapping
sequence. The amount of redundancy of the overlap is termed coverage
(e.g. 10-fold coverage means a base is sequenced 10 times redundantly)
the method as such is referred to as shotgun-sequencing and has -
shortly after sequencing chemestry - been described by Sanger
\cite{pmid6260957}. Soon copmuter programs were necessary to align
sequences, to compute overlaps and consensus sequences
\cite{pmid461197} and the process of computationally reconstructing
the target molecule was termed sequence-assembly \cite{pmid6251542}.

The first step in this overlap-consensus approach is to detect
overlapping sequence in a series of pairwise alignments. Two classical
approaches exist, the first being local ``Smith-Waterman'' alignment
\cite{pmid7265238} the second ``Needleman-Wunsch'' global alignment
\cite{pmid7334527}.

Of course these alignment methods have usages outside of sequence
assembly in general sequence comparison, including protein
sequence. The program \texttt{Blast}, for example, enables large scale
comparison of sequences against databases. It is based on a heuristic
approximation of Smith-Watemam alignments: After a seeding step, in
which small regions of similarity (protein) or perfect matches
(nucleotide) are found, it useses local-alignments to extend regions
of similarity to form high-scoring segment pairs (HSPs). Using a
sophisticated statistical procedure it reports two measurments used to
asses the significance of matches: The e-value reports the number of
hits as good or better than the present hit expected against the
current database by chance. It is usually used to order hits from a
search. The bit-score in contrast is normalized with respect to the
scoring system and database and can thus be used to compare hits from
different searches.

Whith the advent of next generation sequencing (see \ref{sec:ad-seq})
even the heuristic approach of \texttt{Blast} or it's mapping
equivalent \texttt{Blat} \cite{pmid11932250} was not ideally suited
for the massive amouts of data. New kinds of alignment methods were
needed to handle data volume, error structure and short read-length.

\texttt{Ssaha2} \cite{pmid11591649} is able to speed up searches by
orders of magnitude building a hash table indexing k-tuples (k
contiguous bases, implicitly also done in the seeding step of
\texttt{Blast/Blat}). Then sorting of matching indices gives regions
of high similarity without an alignment. These are then aligned using
a banded Smith-Waterman algorithm.

\texttt{Burrows-Wheeler Aligner} (\texttt{BWA}) \cite{pmid20080505}
builds a suffix array holding the starting positions of suffixes of a
lexicographically ordered stiring. Then exact as well as inexact
matches can be found and gapped alignment can be generated.

The assembly problem assebmly problem 



\subsection{Applications of NGS in ecology and evolution}
\label{sec:appl-ngs}

A study on trout in Lake Superior \cite{pmid20331779} used an approach
similar to the appoach in the work presented here: Fish, which show
two different phenotypes were raised in a common environment,
demonstrating the genetic fixation of the phenotypic trait. 454
sequencing was then used to measure the gene expression levels and
successfully indentied 40 genes from two biochemical pathways being
differently expressed. However, in addition to showing divergent
evolution of gene-expression, this sutdy highlighted the limitations
of 454 sequencing for gene-expression analysis.

NGS technologies are are increasinly used in studies on organisms with
ecological and evolutionary significance. Such ecological and
evolutionary ``model organisms'' often lack reference genomes to guide
the assembly-process.


\section{Gene-expression and evolutionary divergence}

Today, both theoretical arguments as well as field and laboratory data
suggest that evolution, including divergence of populations, can occur
very rapidly given the right selective pressure. Such situations
provide us with the opportunity of examining how divergence and even
speciation work at the molecular genetic level
\cite{via_ecological_2002} .


In \textit{Drosophila} variation of gene-expression within a single
species can be attributed more to trans-regulatory elements, while
expression divergent between species is dominated by cis-regulatory
differences \cite{pmid20354124}. Furthermore sterility of hybrid
between species of this genus has been shown to result from
incompatibilities in gene-regulatory networks \cite{pmid16757655}.


% ----------------------------------------------------------------------

%%% Local Variables: ***
%%% mode:latex ***
%%% TeX-master: "../thesis.tex"  ***
%%% tex-main-file: "../thesis.tex" ***
%%% End: ***
     