
% this file is called up by thesis.tex
% content in this file will be fed into the main document

%: ----------------------- introduction file header -----------------------
\chapter{Introduction}
\label{chap:intro}
% the code below specifies where the figures are stored
\ifpdf
    \graphicspath{{1_introduction/figures/PNG/}{1_introduction/figures/PDF/}{1_introduction/figures/}}
\else
    \graphicspath{{1_introduction/figures/EPS/}{1_introduction/figures/}}
\fi

% ----------------------------------------------------------------------
%: ----------------------- introduction content ----------------------- 
% ----------------------------------------------------------------------

\section{The study organism: \textit{Anguillicola crassus}} 
\label{stud-org}

\subsection{Ecological significance} 
\label{eco-sig}

\textit{Anguillicola crassus} Kuwahara, Niimi and Ithakagi 1974
\cite{kuwahara_Niimi_Itagaki_1974} is a swimbladder nematode naturally
parasitising the Japanese eel (\textit{Anguilla japonica}) indigenous
to East-Asia. In the last 30 years anthropogenic expansions of its
geographic- and host-range to new continents and host-species
attracted interest of limnologists and ecologists. The newly acquired
hosts are, like the native host, freshwater eels of the genus
\textit{Anguilla}, and the use of the definitive host seems to be
limited to this genus \cite{sures_development_1999}. However, the
nematode displays a high versatility and plasticity in most other
aspects of it's life, and this has been proposed one of the reasons
for its success invading new continents
\cite{taraschewski_hosts_2007}.

\textit{A. crassus} colonized Europe in the early 1980ies and spread
through almost all populations of the European eel (\textit{Anguilla
  anguilla}) during the following decades (reviewed in
\cite{kirk_impact_2003}). This spread includes populations of the
European eel in North Africa
\cite{gargouri_ben_abdallah_spatio-temporal_2006,
  loukili_dynamics_2007} and at the present day \textit{A. crassus} is
found in all but the northernmost populations of the European eel in
Iceland \cite{kristmundsson_parasite_2007}. It has to be noted
however, that low water temperature \cite{knopf_impact_1998} and
salinity \cite{kirk_effect_2000} limit the dispersal of
\textit{A. crassus} larvae and thus high epidemiological parameters
are rather expected in freshwater and in southern latitudes.

Wielgoss et al. \cite{wielgoss_population_2008} studied the population
structure of \textit{A. crassus} using microsattelite markers and
inferred details about the colonization process and history. Their
data are in good agreement with previous knowledge about the history
of introduction and dispersal. Therefore this process of introduction
and spread can be considered very well illuminated:

\figuremacro{world-ac}{Transcontinental dispersal of
  \textit{A. crassus}}{Invasions of different continents by different
  source-populations are illustrated using arrows. Red color indicates
  the range of the eel species targeted by the invasion. Modified form
  \cite{mnderle_kologische_2005}, based on data reviewed in
  \cite{kirk_impact_2003} and newer findings in
  \cite{wielgoss_population_2008} and \cite{sasal_parasite_2008}.}

\textit{A. crassus} was first recorded in 1882 in North-West Germany,
and this record was published in a German fishery magazine in 1985
\cite{fischer_teichwirt}. The import of Japanese Eels from Taiwan to
the harbor of Bremerhaven in 1980, was soon identified as most likely
source of introduction
\cite{koops_anguillicola-infestations_1989}. Taiwan as the most likely
geographical source of the introduction was in turn also inferred from
population structure using the mentioned microsatellites. Furthermore,
from the fact that genetic diversity is highest in northern regions of
Germany and gradually declines to the south, Wielgoss et al.
concluded a single introduction event to Germany as source for all
populations of \textit{A. crassus} in the comprehensive set of
investigated populations of the European eel. This signal was
persistent together with a punctual signal for anthropogenic mixing of
eels and parasite populations due to restocking
\cite{pmid20646147}. However a recent study found additional
haplotypes for Cytochrome C oxidase subunit I (COXI) in Turkey, and a
second introduction to the Eastern Mediterranean seems possible
\cite{dl_py}. These Turkish haplotypes cluster with Taiwanese
haplotypes and the introduction source would be similar to the main
introduction (see also figure \ref{mCOXI-phylo}).

A second colonization of \textit{A. crassus}, succeeded in
North-America: Since the 1990s populations of the American eel
(\textit{Anguilla rostrata}) have been invaded as novel hosts
\cite{fries_notes:_1996,barse_exotic_1999,
  barse_swimbladder_2001}. Wielgoss et al. identified Japan as the
most likely source of this American population of \textit{A. crassus}
using microsatellite data. Laetsch et al. \cite{dl_py} showed that all
source-populations for different introductions (even the introduction
to the US from Japan) are from one of two separated clades of
\textit{A. crassus} endemic all over East Asia (see also figure
\ref{mCOXI-phylo}).

Finally \textit{A. crassus} has been detected in three indigenous
species of freshwater eels on the island of Reunion near Madagascar
\cite{sasal_parasite_2008}.

Copepods and ostracods serve as intermediate hosts of
\textit{A. crassus} in Asia, as well as in the introduced ranges
\cite{moravec_first_2005}. In these hosts L2 larvae develop to L3
larvae infective for the final host. Once ingested by an eel they
migrate through the intestinal wall and via the body cavity into the
swimbladder wall \cite{haenen_effects_1996}, i.a. using a trypsin-like
proteinase\cite{polzer_identification_1993}. In the swimbladder wall
L3 larvae hatch to L4 larvae. After a final moult from the L4 stage to
adults (via a short pre-adult stage) the parasites inhabit the lumen of
the swimbladder, where they eventually mate. Eggs containing L2 larvae
are released via the eel's \textit{ductus pneumaticus} into it's
intestine and finally into the water
\cite{de_charleroy_life_1990}. The time needed for the completion of a
typical life-cycle from egg to reproducing female is interesting to
determine the number of generations European populations of
\textit{A. crassus} have spent in their newly acquired
environment. Based on laboratory infections it can be estimated to vary
between 70 and 120 days at water temperatures around 20$^{\circ}$.
Such an estimate is leading to 2 generations completed per year in
Europe and a total of circa 60 generations since introduction.


\figuremacro{l_cycle}{Life-cycle of \textit{A. crassus}}{Adult females
  deposit already hatched L2 in the lumen of the swimbladder. Larvae
  migrate through the \textit{ductus pneumaticus} and the intestine
  into the open water. Copepods serve as intermediate host where
  infective L3-larvae develop. These can be transported and accumulated
  in paratenic hosts or directly ingested by an eel. They migrate
  through the eel's intestinal wall into the swimbladder wall. After
  the final molt to adults worms arrive in the lumen of the
  swimbladder, feed on blood and reproduce. Modified from
  \cite{mnderle_kologische_2005}.}

High prevalence of the parasite of above 70\%
\cite{wrtz_distribution_1998,thomas1992population}, as well as high
intensities of infections were reported, throughout the newly
colonized area \cite{lefebvre_anguillicolosis:_2004}. In the natural
host in Asia prevalence and intensities are lower than in Europe
\cite{mnderle_occurrence_2006}.

One of the possible differences between Asian and European population
of \textit{A. crassus} could be the widespread use of paratenic hosts
in European waters \cite{thomas_paratenic_1992,
  pietrock_dynamics_2002}. Such a use of paratenic hosts has not been
reported from the Asian range of the parasite and there is speculation
that the use and availability of paratenic hosts could be a factor
explaining the success of invasion or even the higher epidemiological
parameters in Europe compared to Asia. However, the lack of evidence
for the use of paratenic host in Asia is rater likely to be a a result
of the lack of appropriate studies in Asian water systems, given the
broad spectrum of paratenic hosts use by \textit{A. crassus}
\cite{pietrock_dynamics_2002,rolbiecki_acndab_2004,szkely_dynamics_1995},
including even amphibians and larvae of aquatic insects
\cite{moravec_amphibians_1998}.

Also the abundance of the final hosts \textit{An. anguilla} and
\textit{An. japonica} itself could have an effect on epidemiological
parameters \cite{schabuss_dynamics_2005}. This host-density however is
thought to be similar for each of two host-species in its endemic area
\cite{tesch1983aal}, and more explicitly it is in in parallel rapidly
declining for the last decades both in Asia and Europe
\cite{pmid12713741}.

These factors are thus unlikely to explain the differences in
epidemiological parameters and the differences in abundance and
intensity of \textit{A. crassus} infections in East Asia compared to
Europe are commonly attributed to the different host-parasite
relations in the definitive eel-host permitting a differential
survival of the larval and the adult parasites
\cite{knopf_differences_2004, knopf_swimbladder_2006}.

The impact of \textit{A. crassus} on the European eel has been a major
focus of research during the past decades. Pathogenic effects on the
eels can lead to mortality of eels, when combined with co-stressors
\cite{gollock_physiological_2005}. Responses in \textit{An. anguilla}
show hallmarks of pathology, including thickening
\cite{wurtz_tara_2000} and inflammation
\cite{beregi_radiodiagnostic_1998} of the swimbladder wall,
infiltration with white blood cells and dilated blood vessels.

Especially these changes in the tissue of the swimbladder wall have
been shown to influence swimming behavior and it has been speculated
that eel may fail to complete their spawning migration
\cite{palstra_swimming_2007}. While nobody would claim Anguillicolosis
(the condition caused by \textit{Anguillicola}) to be the main reason
for the decline of eel stocks, it could very well be a cofactor
\cite{sures_science_letter} adding to the tragic main factor of
overfishing of glass-eels \cite{pmid12713741}.

Data from experimental infections of \textit{An. anguilla} with
\textit{A. crassus} suggest that in this host the parasite undergoes
(under experimental conditions) a density-dependent regulation keeping
the number of worms within a certain (high) range
\cite{fazio_regulation_2008}.

In contrast to the European eel, the Japanese eel is capable of
killing larvae of the parasite after vaccination
\cite{knopf_vaccination_2008} or under high infection pressure
\cite{heitlinger_massive_2009}: A high mortality of
\textit{A. crassus} larvae has been reported in the swimbladder wall
of \textit{An. japonica} \cite{mnderle_occurrence_2006} and under high
infection pressure even more pronounced in the intestinal wall
\cite{heitlinger_massive_2009}.

Furthermore it has been shown that the establishment of encapsulated
larvae inside the intestinal wall is related to killing of larvae in
the swimbladder wall: significant numbers of encapsulated larvae in
the intestinal wall were not observed when capsules in the
swimbladder-wall were absent. No capsules in the intestinal wall have
been found in single, non-repeated experimental infections of Japanese
eels, while larvae are killed in the swimbladder wall. These
observation shows that larvae are first encapsulated in the
swimbladder wall and encapsulation inside the intestinal wall follows
only repeated heavy infections. These features suggest a major role of
acquired or infection induced immunity in the formation of capsules
\cite{heitlinger_massive_2009} and thus a prominent role of
host-immunity in the natural host \textit{An. japonica}.

Interestingly the differences in the two host also affect the size and
life-history of the worm: In European eels the nematodes are bigger
and develop and reproduce faster \cite{knopf_differences_2004}.

\figuremacro{worm_diff}{Difference between worms in the swimbladder of
  the European eel and the Japanese eel}{Note the bigger size and
  higher number of worm in a typically infected European eel. In
  comparison in the Japanese eel worms are smaller and intensities of
  infection are much lower. The dark brown matter is ingested
  eel-blood visible through the transparent nematode body- and
  intestinal wall, the white matter are developing eggs and larvae in
  ovaries of female \textit{A. crassus}.}


\subsection{Evolutionary significance}
\label{ev-sig}

\subsubsection{The eel-host}
\label{sec:eel-host}

With a view on the potential co-evolution and especially adaptation of
\textit{Anguilla} spp. to \textit{A. crassus} the catadromous
reproduction of freshwater eels might play an important
role. Individuals of both Atlantic species \textit{An. anguilla} and
\textit{An. rostrata}) migrate thousands of kilometers to reproduce in
the area of the Sargasso sea \cite{pmid19779192}. The Japanese eel in
its endemic area migrates to the west of the southern West Mariana
Ridge \cite{pmid20735676}. Eel larvae then migrate to their freshwater
habitats with the help of oceanic currents. While hybrids between the
two Atlantic eel species have only been reported from Iceland
\cite{pmid21299662}, European eels as a species are considered
panmictic \cite{pmid20735687}: Signals for population structure,
interpreted as evidence against panmixia first \cite{pmid11234011},
have been shown to be an artifact of temporal variation between
cohorts of juvenile eels \cite{pmid19417764, pmid21299662,
  pmid16024374}. Such panmixia reduces the effectiveness of
selection. Uninfected populations participating in reproduction make
rapid local adaptation to a prasite less likely.

Interestingly it has been shown, that individual genetic
heterozygosity in \textit{An. anguilla} is no predictor for
\textit{A. crassus} infestation \cite{pmid19840264}. This is
remarkable, as in a diverse spectrum of organisms such as plants,
marine bivalves, fish or mammals correlations between heterozygosity
and fitness-related traits and especially with parasite-infestation
have been observed \cite{pmid16262866,pmid18398424}. Variation at
highly polymorphic loci is one of the cornerstones of host-adaptation
\cite{pmid20078764}. Once variation is present in a population,
overdominance (or heterozygote superiority) can favor heterozygous
individuals \cite{pmid19129114,pmid17603099}. Matching parasite
antigens and allowing to present them as an epitope, the MHC class II
molecule for example has been demonstrated to be under diversifying
selection in many vertebrate species. Stickleback display variable
copy-numbers of a class IIb MHC gene and \textit{A. crassus}, using it
a paratenic-host, has been shown to select for variability and
heterozygosity at these loci \cite{wegner_parasite_2003}. Vice verca
the vertebrate immune system and especially its memory component are
thought to be driving positive selection on antigens of microorganisms
\cite{conway_measuring_2002}.

Morphological and functional differences between the immune systems of
teleost fishes and other vertebrates (especially mammals) are
prevalent \cite{press1999morphology}. The immune system of eels
especially differs in many details: It lacks all but the M-class of
antibodies and response to macro-parasites is carried out mainly by
neutrophile rather than eosinophile granulocytes
\cite{nielsen_eel_2006}. However, the immune systems of mammals and
fish also show some genetic, molecular and cellular similarity. While
for example the Atlantic cod has lost genes for MHC II
\cite{pmid21832995}, this gene shows conservation in the adaptive
immune system of jawed vertebrates \cite{pmid21078341} and its
presence has been confirmed in transcriptome data for
\textit{An. anguilla} \cite{pmid17666525}.

A decline of epidemiological parameters for European populations of
\textit{A. crassus} has been hypothesized based on data published over
two decades. This decline however, has not been confirmed in an
explicit meta-analysis. If it would be present, possible explanations
would include lower population density of the eel (likely
\cite{schabuss_dynamics_2005}), an evolution of the eel host towards
better resistance (rather unlikely; see above), and an evolution of
\textit{A. crassus} towards lower or at least altered virulence (part
of the present investigation).

\subsubsection{Interest in \textit{A. crassus} based on its
    phylogeny}
\label{phyl-int}

The genus \textit{Anguillicola} comprises five morphospecies
\cite{taraschewski_revision_1988}: In East Asia, in addition to
\textit{A. crassus}, \textit{A. globiceps} Yamaguti, 1935
\cite{yamaguti_globiceps} parasitises \textit{An.
  japonica}. \textit{A. novaezelandiae} is endemic to New Zealand and
South-Eastern Australia in \textit{Anguilla australis} and
\textit{A. australiensis} Johnston et Mawson, 1940
\cite{johnston1940some} parasitises the long-fin eel \textit{Anguilla
  reinhardtii} in North-Eastern Australia. Finally
\textit{A. papernai} is known from the African longfin eel
\textit{Anguilla mossambica} in Southern Africa and Madagascar.

\figuremacro{nLSU-phylo}{Phylogeny of the genus \textit{Anguillicola}
  based nLSU}{Phylogram inferred from nuclear large ribosomal subunit
  (nLSU) of \textit{Anguillicola} and outgroups using Bayesian
  Inference. Labels on internal branches indicate Bayesian posterior
  probabilities. From \cite{dl_py}}

In 2006 F. Moravec promoted the the former subgenus
\textit{Anguillicoloides}, comprising all species of
swimbladder-nematodes but \textit{A. globiceps}, to the rank of a
genus \cite{moravec_anguillicoloides}. In the meantime this
subdivision of the Anguillicolidae in two genera was revised based on
the rejection of monophyly of the new genus \textit{Anguillicoloides}
and \textit{``Anguillicoloides crassus''} was restored to
\textit{Anguillicola crassus} \cite{dl_py}. In this study,
\textit{A. crassus} was identified as the basal species in the genus,
analyzing the nuclear genes small ribosomal subunit (nSSU) and large
ribosomal subunit (nLSU, see figure \ref{nLSU-phylo}). An alternative
phylogenetic hypothesis derived from mitochondrial cytochrome c
oxidase subunit I (COX I) sequences places \textit{A. crassus} in a
clade with the oceanic species and \textit{A. globiceps} and
\textit{A. papernai} in a sister clade (see figure \ref{mCOXI-phylo}).

Neither of these phylogenetic hypotheses is compatible with the
phylogeny of the eel-hosts without host-switching: Assuming the
establishment of \textit{Anguillicola} in an ancestral Indo-pacific
host at least three host-switch events are needed, even to explain
classical (non-recent, i.e. non-anthropogenic) host-parasite
associations. Two of these host-capture events must have spanned the
major splits in the eel phylogeny \cite{minegishi_molecular_2005}:
Oceanic \textit{Anguillicola} must have captured hosts transitioning
between the clade of \textit{An. reinhardtii} and
\textit{An. japonica} to the clade in which \textit{An. australis} is
found. Also the basal species of freshwater eels
\textit{An. mossambica} must have been captured in an host-capture
event involving a phylogenetically distant host-species.

\figuremacroW{mCOXI-phylo}{Phylogeny of the genus
  \textit{Anguillicola} based on COXI}{Phylogram inferred for
  \textit{Anguillicola} and outgroups based on mitochondrial
  Cytochrome C oxidase subunit I (COXI) using Bayesian
  Inference. Labels on internal branches indicate Bayesian posterior
  probabilities. From \cite{dl_py}}{0.95}

The recent anthropogenic host-switches of \textit{A. crassus} from
\textit{An. japonica} to \textit{An. anguilla} and
\textit{An. rostrata} constitute additional acquisitions of
phylogenetically well separated hosts. This affinity for
host-switching may be an evolutionary relict found only in one of the
two clades (putative cryptic species) in which \textit{A. crassus} can
be devided \cite{dl_py}.

The to date most likely phylogenetic hypothesis places the genus
\textit{Anguillicola} (the only genus in the family Anguillicolidae)
at a basal position in the Spirurina (clade III \textit{sensu}
\cite{blaxter_molecular_1998}), one of 5 major clades of nematodes
\cite{nadler_molecular_2007, wijov_evolutionary_2006}. The Spirurina
exclusively exhibit a animal-parasitic lifestyle and comprise
important human pathogens as well as prominent parasites of livestock
(e.g. the Filaroidea and Ascarididae). The finer subdivision of the
Spirurina into Spirurina A, and the Sister clades Spirurina B and C
from \cite{dl_py} can be seen in figure \ref{clade-phylo}.

\figuremacroW{clade-phylo}{Phylogeny of nematode clade III based on
  nuclear small ribosomal subunit}{Phylogram inferred from nuclear
  small ribosomal subunit for Spirurina using Bayesian
  Inference. Branches are collapsed to highlight major groups. Labels
  on internal branches indicate Bayesian posterior probabilities. From
  \cite{dl_py}}{0.95}

Within the Spirurina B an enormous phylogenetic diversity of the
definitive hosts can be observed, ranging from fresh-water fish as
hosts for the Anguillicolidae to cartilaginous fish for
Echinocephalus, mammals parasitised by Gnathostoma and Linstowinema to
reptiles as hosts for Tanqua. In addition to this diversity, a common
characteristic of Spirurina B and C is a complex life-cycle involving
freshwater or marine intermediate hosts. Application of parsimony
principles thus favors a complex life history as the ancestral state
for the Spirurina.

This phylogenetic position makes the Anguillicolidae an interesting
system as outgroup taxon to understand the evolution of parasitic
phenotypes in the Spirurina. In addition the recent anthropogenic
expansion of especially \textit{A. crassus} to new host species
provides the opportunity to observe phenotypic modifications as well
as early genetic divergence making it an ideal satellite-model.

\subsubsection{A taxonomy of common garden experiments and the
  divergence of \textit{A. crassus} populations}
\label{div-ac}

Common-garden and transplant experiments are a method to separate
genetic components (G) of phenotypic differences from environmental
(E) influences, used for almost as long as scientists investigate
evolution \cite{kerner_classic_common_garden,
  bonnier_classic_common_garden}.

The goal of a classical common garden experiment is the exclusion of
environmental factors: By carefully choosing an universal environment
(the garden) genetic differences between potentially diverged
population of a species should be isolated and elucidated. This
approach is equivalent to one-factorial design investigating only the
genetic factor (G). However, an experimental design aiming to exclude
environmental effects bears the risk of overlooking main effects of
the genotype component blurred by genotype by environment (GxE)
interactions. In other words: there are situations in which the
differences in genotypes could be visible only under special
environmental conditions.

This limitations of the common garden approach are addressed in
transplant experiments. Representatives of each population are raised
in the other population's natural environment. Explicitly including
the environmental component this represents a two-factorial design in
which interactions between genotype and environment (GxE) can be
incorporated into an analytical model.

In situations where host-parasite interactions should be studied the
experimental design is complicated by one further genetic factor.
When a common garden scenario is applied to different parasites
infecting one hosts-species (or vice versa) such an experiment can be
best described as ``inoculation experiment under common garden
conditions''. Often only one of the interacting species can be
regarded as the focal species. In the presented
\textit{A. crassus}/\textit{Anguilla} spp. project it is the parasite,
as definitive genetic differences between the host-species are not in
the focus. However using only one host-species the experiment would be
equivalent to the analysis of the focal genotype, missing GxG
interactions. This is addressed by a ``reciprocal cross-inoculation
experiment under common garden conditions'' \cite{kaltz_shykoff_rev}.
The infection of both host-species with both parasite populations
allows the incorporation of genotype by genotype (GxG) effects into an
analytical model. This approach is chosen in the experiments presented
in this thesis.

In a recent study using this method (and inspiring the experimental
design for our project) both European and Japanese eels were infected
under laboratory conditions with worms from three geographic origins:
Southern Germany, Poland and Taiwan.

\figuremacro{ula_stages}{Differences in developmental speed}{Three
  populations of \textit{A. crassus} (panels in columns) were raised
  in two different hosts (panels in rows). Eels were dissected at 4
  different time points post infection (dpi). Bars represent means of
  recovered individuals from three different life-cycle stages
  indicated by color. Differences between parasite-populations are
  pointed out in the main text. Data courtesy of Urszula Weclawski.}

In these experiments differences between the two European populations
and and the Taiwanese population of worms manifested. These
differences were especially (but not solely) visible in the early
stages of the life-cycle:

In the European eel the number of L3 larvae from the Taiwanese
population of worms was higher than from European worms. From the
Taiwanese population less L4 larvae were observed at 25 dpi and the
levels of this larval stage were stable during the infection, in
contrast the numbers of L4 for the European populations decreased with
time. Additionally up to 50 dpi there were less living adults observed
for worm from the Taiwanese population and fewer dead adult worms were
recorded for the Taiwanese population beginning from 50 dpi.

In the Japanese eel fewer L3 larvae at 25dpi were observed from the
Taiwanese population compared to the European population of
worms. Additionally more L4 larvae at this point in time and fewer
living adults at 25 and 150 dpi, as well as fewer adults beginning
from 50 dpi from worms of Taiwanese origin could be recovered compared
to worms of European origin (Weclawski et al. unpublished; see figure
\ref{ula_stages}).

These findings can be consolidated to the interpretation that an
increase in the speed of development was observed in the European
populations of \textit{A. crassus} compared to the Taiwanese source
population.

Measurements at different time-points are not easy to integrate into a
more general interpretation of observed recovery of worms as
fitness-components. Such fitness-components are usually thought to be
a approximation to fitness (with life-time reproductive success as one
of the closest approximations). Life history traits generally possess
lower heritability and are under stronger selection
\cite{pmid3316130}. The inferred faster development of the European
population of \textit{A. crassus} can thus be regarded highly
interesting as candidate-phenotype for adaptation. However, the
slightly delayed development of the Taiwanese population even in the
natural host \textit{An. japonica} would constitute an maladaptation
\cite{pmid21708731} in one possible interpretation of these results.

The differences however are small in \textit{An. japonica} and could
possibly have a second explanation: GxG interactions could be hidden
in \textit{An. japonica} by GxGxE interactions. Such triple
interactions could lead to superior fitness-components of the natural
host-parasite genotype combination e.g. only at elevated water
temperature or under other (even additional biotic) environmental
conditions. An optimal experimental approach would thus be able to
disentangle even GxGxE interactions and a design would be advantageous
as it would explicitly include potential heterogeneity in the
environment shaping GxGxE interaction as predicted ty the theory of
the geographic mosaic of coevolution
\cite{thompson2005geographic}. Such an experimental design a
``reciprocal cross-inoculation under reciprocal transplant
conditions'' \cite{pmid18419564} is however impossible to implement in
a mobile host-parasite system threatening biosafety as artificial
secondary introductions are required for a transplant.

Nevertheless, the present experimental results provide a solid
foundation for further research. They demonstrate divergence of the
European population of \textit{A. crassus}. Furthermore the loss of
genetic diversity in the European population
\cite{wielgoss_population_2008} seems to not have led to a decrease of
fitness.

Interpretation of morphological characters in these studies proved
difficult: Size of the worms seems to be mainly determined by the
uptake of host-blood and is thus largely object to phenotypic
modification, with a genetic component hard to detect. The approach
taken in the study underlying this thesis builds on the above design
but uses gene-expression levels as the phenotypic entity studied. This
approach is enabled by recent advances in DNA-sequencing technology.

\section{DNA sequencing}

\subsection{Two out of three: DNA sequencing and the central dogma of
  molecular biology}
\label{sec:dm}


\figuremacroN{C_D}{Major macromolecules bearing biological sequence
  information:}{A schematic view on the flow of genetic information
  in a cellular life: Enzymes (red font) process macromolecules
  carrying genetic information from DNA to RNA, from RNA to
  protein. Picture from
  \href{http://en.wikipedia.org/wiki/Central_dogma_of_molecular_biology}{wikipedia}.}

Two kinds of macromolecules carry all the information evolution shaped
over the course of the last 3.5 billion years from generation to
generation: DNA and only in some viruses RNA. Proteins as the building
blocks and functional molecules of life are transient manifestation of
this information \cite{crick1958biological}. In all cellular life
genetic information flows from the replicating DNA to RNA in a process
called transcription and from RNA to Protein in a process called
translation \cite{pmid5422595} (see figure \ref{C_D}).

The relatively inert DNA is adapted to carry information over
generations and to limit the number of mutation (also by evolving low
error in polymerase) \cite{pmid21821597}. The single stranded, more
reactive RNA on the other hand can create secondary structures by
base-pairing with itself or other macromolecules and is involved in
numerous cellular processes making use of this reactivity
\cite{pmid21850044}: microRNAs (miRNAs) regulate translation by
binding mRNA, initiate degradation and thus decrease it's levels
\cite{pmid20703300,pmid11679654}, small nuclear RNAs (snRNAs) are
(among other functions) part of the spliceosome (see below), small
nucleolar RNAs (snoRNAs) direct a machinery to perform site-specific
rRNA modification \cite{pmid19446021}. In addition a variety of poorly
understood other non-protein coding RNA (ncRNA) families exist
\cite{pmid16344563}. Together with proteins ribosomal RNAs (rRNAs) are
building blocks of the ribosome, where translation takes
place. Transfer RNAs (tRNAs) carry amino acids to the ribosome
specific to their anti-codon sequence. There, at the ribosome, amino
acids are incorporated into the polypeptide chain according to codon
recognized in coding sequence (CDS) of a messenger RNA (mRNA) molecule
and a protein is synthesized \cite{pmid4887876}.

These mRNAs (like the untranslated RNAs above) have been transcribed
from genomic DNA (see figure \ref{RNA_pro}). Eukaryotic mRNAs have a
special structure to prevent them from and regulate degradation and to
allow interaction with non-coding RNA and with the ribosome during
translation: The 5' CAP-structure and the 3' poly-A tail are added
directly during transcription.

\figuremacro{RNA_pro}{The structure of a protein coding gene and it's
  mRNA}{A schematic view of posttranscriptional modifications in an
  eukaryotic gene. Introns are spliced, 5' and 3' structures are added
  and the mRNA molecule is exported into the cytoplasm. Note that the
  double stranded nature of the genomic DNA (gray) is not indicated in
  this comic and no indication of the enzymes unwinding genomic DNA
  for transcription is given.}

Other post- or co-transcriptional modifications often include the
excision of introns, non-coding regions found in genomic DNA
interspersed in coding regions. This excision is directed by the
spliceosome containing snRNAs and proteins. In this splicing step
alternative exons can be joined, skipped or even introns can be
retained, increasing transcriptome and proteome diversification
\cite{pmid17158149}. Only after the processing of pre-mRNA to mature
mRNA, the molecule is released into the cytoplasm where it eventually
can be translated (see above).

The complete set of transcripts in a cell is called the
transcriptome. One of the major goals of transcriptomics (the analysis
of the transcriptome) is to asses quantity of transcripts for a
specific treatment, genetic background, developmental stage or
physiological condition. Intermediate goals in this process are the
categorization of transcript into one of the diverse families above
(mRNAs or ncRNAs and small RNAs) and the determination of the
transcriptional and translational structure of genes (mRNA): Finding
start sites for both transcription from the genome and for translation
into protein, 5' and 3' ends, splicing patterns and other
post-transcriptional modifications \cite{pmid19015660}.

Transcriptome-projects and transcriptomic data have been invaluable to
determine the structure of the genome (information gained from the
transcriptme informs about genomic features) but they are also in the
center of one of the major challenges in biology linking genotypes to
phenotypes. The ``expression'' of the gene in an literal sense would
be the phenotype visible for natural selection. It is known that
posttranslational modification, the degradation and turnover of both
mRNA and proteins have a strong influence on this gene-expression, and
in this sense the global measurement protein expression (proteomics)
would be one step closer towards a phenotype. Indeed increasingly
proteomic information is used to complement genomics and
transcriptomics \cite{pmid20121477}. However, overall levels of mRNA
abundance correlate well with protein abundance
\cite{pmid21593866}. Measurements of protein levels are methodically
more demanding than measurements of mRNA levels (see \ref{his-seq})
and thus all estimates of gene-expression in this thesis are based on
measurements of RNA-abundance and the term gene-expression is even
used as a synonym for RNA-abundance. All mentions of protein sequence
in the results of this document are derived from computational
prediction based on the nucleotide sequence of mRNA.

All sequencing technologies for nucleic acid outlined below have in
common, that they work on DNA not on RNA. Therefore transcriptome
sequencing involves a step in which (more or less specifically) mRNA
is reverse transcribed into complementary DNA (cDNA). The
RNA-dependent DNA-polymerase (reverse transcriptase) used for this
process is originally found in retroviruses.

\subsection{The history and methods of high-throughput DNA-sequencing}
\label{his-seq}

For almost three decades the method developed by Frederick Sanger
\cite{pmid271968} was the only practical choice for determining the
sequence of nucleic acid. Starting from denatured DNA, the method uses
four different dideoxynucleotides (ddATP, ddCTP, ddGTP, ddTTPs) to
terminate synthesis throughout the reaction (along the whole molecule)
at the respective incorporation sites. The method first used
radioactive labels attached to primers in four separate reactions for
each of the ddNTP. The length of the partial DNA-sequences then had to
be determined on a single-base resolution agarose gel. Later
fluorescent labeling of ddNTPs allowed all four reactions to be
performed together. Additionally modern machines use the
chain-termination method combined with capillary gel electrophoresis
\cite{pmid2326186} in a highly parallelized way.

Due to these advancements it was possible to tackle sequencing of
bigger genomes, after phages in the first years of DNA sequencing
\cite{pmid1264203}: The bacterium \textit{Escherichia coli} in 1997
\cite{pmid9278503}, the baker's yeast \textit{Saccharomyces
  cerevisiae} in 1996 \cite{pmid8849441}, the nematode
\textit{Caenorhabditis elegans} in 1998 \cite{pmid9851916}, the fruit
fly \textit{Drosophila melanogaster} in 2000 \cite{adams2000genome}
and the mouse \textit{Mus musculus} in 2002 \cite{pmid12466850} were
the first organisms with sequenced genomes. For these laboratory
model-organisms multi-national consortia financed and coordinated
sequencing in multi-million dollar projects. This ``first generation
of genomics'' culminated in the publication of the human genome in
2001 \cite{pmid11181995}.

In parallel to the mentioned genome-projects transcriptome projects
were conducted. Single pass Sanger-sequencing reads called expressed
sequence tags (ESTs) were mapped to genomic sequenc identifying coding
regions \cite{pmid2047873}. First estimate of the number of genes in
the human genome for example are based on extrapolation of the number
of genes found with this method in early sequenced regions of the
gneome \cite{pmid7920649}.

Costs and labor constrained genome-sequencing to the well established
laboratory-model organisms mentioned above. In addition to the
sequencing reaction itself, it was the need for cloning into DNA
vectors for separation and amplification of DNA-fragments, that made
costs and labor associated with this method prohibitive for a large
scale application in non-model organisms.

\subsection{DNA-sequencing in Nematodes}
\label{sec:dna-sequ-nemat}

In 1998 \textit{Caenorhabditis elegans} had become the first
multicellular organism with a sequenced genome
\cite{pmid9851916}. Soon it was noted, that in addition to it's use as
a general model system for the metazoa and beyond, knowledge gained in
this species has the potential to be even more valuable within the
phylum nematoda \cite{blaxter_caenorhabditis_1998}. The breadth and
detail of genomic information available for \textit{C. elegans} to
date is illustrated by a recent publication, using transcriptomics to
provide detailed annotation of the diverse functional genomic elements
and their interactions at single base resolution
\cite{pmid21177976}. With this amount of data digested into usable
information \textit{C. elegans} continues be an invaluable resource in
nematode genomics: 21,000 protein coding genes, over 5,000 RNA genes
and 100.2 megabases (Mb) of overall sequence still provide the most
thoroughly investigated comparative basis for new genome or
transcriptome projects started in the Nematoda.

The genome sequence of \textit{Caenorhabditis elegans} was soon
complemented by the genome of \textit{Caenorhabditis briggsae}
\cite{stein_genome_2003}, a second nematode from the genus
\textit{Caenorhabditis} sequenced a satellite-system for comparative
genomics inside this genus. As a second satellite-model in clade V the
necromenic \textit{Pristionchus pacificus} (living in close
association with beetles) has a published draft genome
\cite{pmid18806794}.

The first published genome of a parasitic nematode in the Spirurina
was the genome of \textit{Brugia malayi} \cite{ghedin_draft_2007} and
as a second genome in the Spirurina recently the genome of
\textit{Ascaris suum} has been published \cite{pmid22031327}.

Also in the remaining clades of the nematoda genome sequencing
flourished: For the animal-parasite \textit{Trichinella spiralis} from
clade I \cite{pmid21336279}, the plant parasites \textit{Meloidogyne
  incognita} \cite{pmid18660804} and \textit{Meloidogyne hapla}
\cite{pmid18809916} as well as the the pinewood nematode
\textit{Bursaphelenchus xylophilus} \cite{pmid21909270} (a plant
parasite using a beetle as an vector) from clade IV recently genome
sequences have have been analyzed and published.

The current revolution in sequencing methodology (see
\ref{sec:ad-seq}) brings into sight many more sequenced nematode
genomes (including that of \textit{A. crassus}). The 959 nematode
genomes initiative promotes such sequencing of nematode genomes and
makes working-drafts of genome-assemblies available for analytical
purposes on a \texttt{Blast}-server \cite{pmid22058131}.

Before the advent of next generation sequencing (NGS; see
\ref{sec:ad-seq}) the lack of genomic information in many species of
nematodes promoted the use of ESTs as a tool for
gene-discovery. Partial genomes \textit{sensu}
\cite{parkinson_partigene--constructing_2004} were successfully
interrogated for a large array of genes interesting for various
scientific communities. In nematode parasites of vertebrates,
pathogenic factors were described as potential vaccine candidates
\cite{pmid11406138}. Change in expression of these molecules
constitutes an \textit{a priori} hypothesis to be tested for different
populations and host-environments in \textit{A. crassus}:

Cystein-proteinase inhibitors (cystatins) and serin proteinase
inhibitors (serpins) are thought to interact with the antigen
presentation in vertebrate hosts \cite{pmid11406138}. Homologues of
mammalian cytokines were identified, which are believed to interact
with mammalian cytokine receptors to divert the immune response to a
TH2-type response \cite{maizels_helminth_2004} (an anti-inflammatory,
cellular response, thought to be non-effective against
helmiths). Further molecules involved in host-parasite interaction
identified in transcriptome-projects include abundant larval
transcripts of \textit{B. malayi} (Bm-ALT)
\cite{gomez-escobar_abundant_2002} and venom like allergens (Bm-VLA)
\cite{pmid11704277}.

In some of these studies secreted proteins were in the center of
interest. They could potentially be excreted by the nematode to allow
movement and food-uptake but also to interact with the host's immune
system. The detection of signal-peptides for secretion using
\textit{in silico} analysis of ESTs has been used to highlight
candidate genes for example in \textit{Nippostrongylus brasiliensis}
\cite{harcus_signal_2004}, and across all nematode ESTs
\cite{nagaraj_needles_2008}.

Over the years sequence information derived from EST-data and whole
genome sequencing has been collected and updated into the nembase
transcriptome databases
\cite{parkinson_nembase:resource_2004,wasmuth_extent_2008}. The recent
compendium nembase4 describes clustering of 679,480 raw ESTs in
233,295 clusters from 62 species \cite{pmid21550347}. This database
provides an invaluable collection of confirmed infromation for
comparison, validation and hypothesis generation when new
transcriptomes are analysed as in the present project.

Obviously NGS also leaves it's marks currently in nematode
transcriptomics: NGS analysis on the transcriptomes of
\textit{Ancylostoma caninum} \cite{pmid20470405}, \textit{Pristionchus
  pacificus} \cite{pmid20237107}, \textit{Litomosoides sigmodontis}
\cite{pmid20950480} and \textit{Ascaris suum} \cite{pmid21685128} have
been published and a recent review \cite{pmid22044053} lists 8 further
datasets for other species already available in public
repositories. Additionally for \textit{Haemonchus contortus} a
pyrsequencing-transcriptome has been published \cite{pmid20420710}
unnoticed by the above review, illustrating the explosive expansion of
data and publications.

\subsection{Advances in sequencing technology}
\label{sec:ad-seq}

Advances in sequencing technology (often termed ``Next Generation
Sequencing''; NGS), provide the opportunity for rapid and
cost-effective generation of genome-scale DNA-sequence data. Labor and
costs associated with DNA-sequences were drastically reduced during
the last 5 years (see figure \ref{sequencing_costs}).


\figuremacro{sequencing_costs}{Falling sequencing costs}{Sequencing
  costs falling due to advances in Solexa-sequencing: Due to improved
  read-length and data-volume on this platform per base
  sequencing-prices for many applications tumble into free fall. Data
  provided by \href{http://www.genome.gov/sequencingcosts/}{National
    Human Genome Research Institute, NHGRI}.}

The technologies portrayed here and used in the work underlying this
thesis can't work on single molecules and thus target molecules have
to be amplified like in Sanger-sequencing. This amplification has to
produce spatially separated templates. Especially new methods to
address this methodological need are in the heart of the new
technologies. Immobilization on a solid surface to archive clonal
amplification is used in preparation of both pyrosequencing and for
the Illumina-platform \cite{pmid19997069}. The detailed implementation
of this solid-state amplification in each technology differs and will
be explained in the corresponding sub-chapter.

% One cumbersome aspect of the need for amplification is the high amount
% of DNA starting-material ($3 -– 20 \mu$g) required
% \cite{pmid19997069}. Other disadvantages include, that mutations during
% clonal amplification in templates can disguise error as sequence
% variants. Nucleotide composition of the target may also introduce
% amplification bias and thus biased product yield
% \cite{pmid19327155}. This in turn leads to underrepresentation of
% certain molecules. The last point can be detrimental in quantitative
% applications, such as RNA-seq \cite{pmid19015660}. However, while
% alternative single molecule approaches exist (eg. \cite{pmid21431759,
%   pmid21572978}) and can be applied to address the above stated the
% problems \cite{pmid21431761, pmid21957044}, to date these technologies
% are in throughput and reliability not competitive for most real life
% applications.


\subsubsection{Pyro-sequencing}
\label{sec:pyro-seq}

\figuremacro{454}{Schematic representation of pyrosequencing}{ (a) DNA
  (genomic or transcriptomic) is isolated, fragmented, ligated to
  adapters and denatured into single strands (b) Under conditions that
  favor one fragment per bead fragments are bound to beads. These
  beads are isolated and compartmentalized in the droplets of an
  emulsion and PCR (a mixture of reagents in oil). Within each droplet
  DNA is amplified, and beads are obtained carrying millions of copies
  of a unique DNA template. (c) After denaturation of DNA, beads are
  deposited into wells of a fiber-optic slide (called picolitre
  plate). (d) Immobilized enzymes carried on smaller beads are added
  to each well and a solid phase pyrophosphate sequencing reaction is
  initiated. (e) A portion of a fiber-optic slide, in a scanning
  electron micrograph (prior to bead deposition) (f) Major subsystems
  of the 454 sequencing instrument: a fluidic assembly holding
  nucleotides separately (object i), the well-containing
  picolitre-plate in a flow cell (object ii), a CCD camera assembly
  and the user interface for instrument control (object iii)
  \cite{pmid18846085}}

Prior to pyrosequencing (or 454-sequencing; named by the company making
it commercially available) an emulsion PCR is used to clonally amplify
DNA molecules attached to beads (figure \ref{454}): After
fragmentation by mechanical shearing or ultrasound \cite{pmid20298868}
(see figure \ref{454}), DNA is ligated to adapters, denatured and
single stranded molecules are attached to complementary sequence on a
bead. Emulsion of beads in oil together with enzymes under conditions
that favor one bead per water/enzyme droplet allows PCR in
micro-scale reactions. This covers each bead with multiple copies of
one target molecule. The beads are then distributed over the wells of
a fiber-optic slide, the so called picolitre plate. A single bead per
well is covered with enzymes on the surface of smaller beads. These
enzymes are used in the actual pyrosequencing reaction originally
developed by P\r{a}l Nyr\'{e}n in the 1990s \cite{pmid17185753}. The
release of inorganic PPi as a result of nucleotide incorporation by
polymerase starts a cascade of enzymatic reactions. The released PPi
is converted to ATP by ATP sulfurylase, providing energy for
luciferase to oxidize luciferin and to generate light. The added
nucleotide is known as nucleotides are flushed over the plate one at a
time. A high resolution camera records the emission of light. The
intensity of emitted light is proportional to the number of
nucleotides incorporated.

The ability to distinguish length of homopolymeric runs of the same
nucleotide decreases with the length of such homopolymer runs
\cite{pmid21685085}. Current ``Titanium chemistry'' is producing reads
of $>$ 350 bases length, ``FLX chemistry'' (used up to 2009) was able
to produce reads of roughly 250 bases length \cite{pmid21514329}.

This longer read length of 454-sequencing \cite{pmid16056220} compared
to other NGS technologies (see \ref{sec:ill-seq}), allows \textit{de
  novo} assembly of transcripts in organisms lacking previous genomic
or transcriptomic data (for a comprehensive list of studies using this
approach before Oct 2010 see \cite{pmid20950480}).

\subsubsection{Illumina-Solexa sequencing}
\label{sec:ill-seq}

Illumina-Solexa technology is to date (Dec. 2011) the most competitive
commercial sequencing platforms enabling a broad spectrum of
applications.

\figuremacro{ilmn_all}{Schematic representation of Illumina-
  sequencing}{(a) DNA (genomic or transcriptomic) is isolated,
  fragmented and ligated to adapters. (b) Single stranded fragments
  are bound to a glass-slide. (c-d) Solid-phase bridge amplification
  using unlabeled nucleotides, primers (binding the adapters) and
  polymerase leaves clusters of double stranded DNA distributed over
  the slide.  (e) four labeled reversible terminators, primers
  (binding the adapters) and polymerase are added. An image of the
  emitted fluorescence under laser excitation is taken . Step (e) is
  repeated multiple times for the length of the DNA-sequence. Modified
  from
  \href{http://seqanswers.com/forums/showthread.php?t=21}{Seqanswers-forum}}

The Illumina-Solexa platform uses bridge-amplification to produce
clonal copies of DNA molecules in clusters on a glass slide (figure
\ref{ilmn_all}): Fragmented, double-stranded DNA is therefore ligated to
a pair of oligonucleotide-adapters in a forked configuration (the
adapter-ends have non-complementary sequence). Two primers are used in
an initial amplification and a double-stranded molecule with a
different adapter on either end is produced. Denatured single-strands
are then annealed to complementary adapters on the surface of a glass
slide. Using the 3' end of the surface-bound oligonucleotide as a
primer, a new strand is synthesized. Subsequently the adapter sequence
at the 3' end of newly synthesized copied strand is bound to another
surface-bound complementary oligonucleotide. This results in a
bridge-structure and generation of a new priming-site for synthesis
after denaturation. Multiple cycles of this kind of solid-state PCR
result in growth of clusters on the surface of the glass-slide
\cite{pmid18987734}.

In the actual sequencing reaction these clusters are sequenced using a
sequencing by synthesis technique: polymerase and all four nucleotides
simultaneously are flushed over the glass slide in successive
cycles. To avoid incorporation of multiple nucleotides, ``removable
terminator''-nucleotides are used, which allow only incorporation of
one nucleotide per strand pre cycle. These nucleotides are labeled
each with a different removable fluorophore. Transient incorporation
of a fluorophore along with a nucleotide is detected using a high
resolution camera after laser-induced excitation. The fluorophore is
removed and next cycle is initiated \cite{pmid18987734}.

This leads to an error model different from 454 sequencing: Runs of
homopolymeric sequence are not problematic, but due to the decreasing
propensity of terminators for removal, sequencing quality decreases in
from 5' to 3' direction.

An slight alternation of the above method, which is extremely useful
to inform assembly, is paired-end sequencing: After the first
sequencing (as above), the original template strand is used to
regenerate the complementary strand. This complementary strand then
acts as a template for the second sequencing reaction producing
complementary sequence from the other end of the molecule. Using
template molecules of a certain size range, sequence information can
be obtained spanning 200-500 bases (the possible span of a nucleotide
bridge in bridge-amplification) \cite{pmid18987734}.

Additionally recent increases in read length (from 35 bases in 2008 to
over 100 bases in 2011) are beginning to allow \textit{de novo}
sequencing and assembly of large eukaryotic genomes (e.g. that of the
giant panda \cite{pmid20010809}) and transcriptomes
\cite{pmid21679424} (but see also \ref{sec:comp-meth-dna} for
methodical challenges). In the same period throughput also increased
from roughly 6,000,000 reads in 2008 to roughly 20,000,000 reads in
2011 on one of six lanes (compartments of the glass-slied) of the
instrument.

The high throughput of the Illumina-Solexa platform makes it also
first choice for gene expression analysis \cite{pmid21627854}: RNA-seq
has revolutionized transcriptomics both in model and non-model
organisms \cite{pmid19015660}. SuperSAGE \cite{pmid20967605} using
expression-tags provides the benefit of classical SAGE-analysis
\cite{pmid7570003} with those of the ultra heigh throughput of
Illumina-Solexa sequencing.

\subsection{Computational methods in DNA-sequence analysis}
\label{sec:comp-meth-dna}


Although the sequencing reaction itself differs between platforms, the
technologies described above have in common that up to date they
produce much more, but shorter reads than classical Sanger-sequencing.

This fostered use and development of new methods to assemble
large-scale shotgun sequences, as higher coverage but shorter
read-length (and also lower accuracy) are increasing the computational
complexity of the assembly-problem (reviewed in \cite{pmid20211242}).

In the context of computational tools this common characteristic of
all DNA-sequencing methods has to be emphasized: Read-length is
usually shorter than the length of the target molecule to be
sequenced. This potential problem is solved by oversampling the target
molecule, producing overlapping sequence. The amount of redundancy of
the overlap is termed coverage (e.g. 10-fold coverage means a base is
sequenced 10 times redundantly) the method as such is referred to as
shotgun-sequencing and has - shortly after sequencing chemistry - been
described by F. Sanger \cite{pmid6260957}. Soon computer programs were
necessary to align sequences, to compute overlaps and consensus
sequences \cite{pmid461197} and the process of computationally
reconstructing the target molecule was termed sequence-assembly
\cite{pmid6251542}. This reconstructed target molecules are termed
contigs, derived from contiguous sequence. In an (hardly achieved)
optimal genome-assembly a contig would thus represent a chromosome, in
an optimal transcriptome assembly there would be a contig for every
transcript in the cell.

The first step in the overlap-consensus approach is to detect
overlapping sequence in a series of pairwise alignments. Two classical
approaches exist, the first being local ``Smith-Waterman'' alignment
\cite{pmid7265238} the second ``Needleman-Wunsch'' global alignment
\cite{pmid7334527}. Of course these alignment methods have usages
outside of sequence assembly in general sequence comparison, including
protein sequence.

The program \texttt{Blast} \cite{pmid2231712}, for example, enables
large scale comparison of sequences against databases. It is based on
a heuristic approximation of Smith-Waterman alignments: After a
seeding step, in which small regions of similarity (protein) or
perfect matches (nucleotide) are found, it uses local-alignments to
extend regions of similarity and to form high-scoring segment pairs
(HSPs). Using a sophisticated statistical procedure it reports two
measurements used to asses the significance of matches: The e-value
reports the number of hits as good or better than the present hit
expected against the current database by chance. It is usually used to
order hits from a search. The bit-score in contrast is normalized with
respect to the scoring system and database and can thus be used to
compare hits from different searches.

With the advent of next generation sequencing (see \ref{sec:ad-seq})
even the heuristic approach of \texttt{Blast} or it's mapping
equivalent \texttt{Blat} \cite{pmid11932250} was not ideally suited
for the massive amounts of data. New kinds of alignment methods were
needed to handle data volume, error structure and short read-length.
Mapping describes a subset of the assembly problem and mapping
programs confine themselfs to this sub-problem. In mapping only the
positions (and the quality) of a match relative to an already
sequenced longer contig are interrogated. \texttt{Ssaha2}
\cite{pmid11591649} is able to speed up such sequence searches by
orders of magnitude. It builds a hash table indexing k-tuples (k
contiguous bases, an approch implicitly also used in the seeding step
of \texttt{Blast/Blat}). Then sorting of matching indices gives
regions of high similarity without an alignment, but these region can
then be aligned using a banded Smith-Waterman
algorithm. \texttt{Burrows-Wheeler Aligner} (\texttt{BWA})
\cite{pmid20080505} builds a suffix array holding the starting
positions of suffixes of a lexicographically ordered string. Then
exact as well as inexact matches can be found and a gapped alignment
can be generated.

For \textit{de novo} assembly of genomes new algorithmic approaches
involve consturction of a de Bruijn-graph. In most formulations of
this new approach instead of nodes in the graph (sequences) edges
(overlaps) are traversed. This way problematic repeats are joined and
sub-sequences reused. The method uses a splitting of sequences in
k-mers of defined length (edges in the de Bruijn-graph) and is thus
optimal for very short reads \cite{pmid18349386}.

On top of this complexity found in \textit{de novo} assembly of
genomes, transcriptome assembly has to deal with additional challenges
resulting from the biology of the transcriptome (see \ref{sec:dm}):
(a) The depths of reads obtained from cDNA for different transcripts
differs dramatically, additionally target molecules may be covered
uneven across their length. (b) In highly expressed transcripts more
erronous bases are found in total. (c) Transcripts from adjacent loci
can overlap and can be erronously fused to form chimeric
transcripts. (d) Multiple real transcripts can exist per genomic
locus, due to alternative splicing. (e) Additionally sequences that
are repeated in different genes (domains) introduce ambiguity
\cite{pmid21572440}.

Using pyrosequencing instead of the solexa-platform problems (a) and
(b) are less pronounced because of the overall lower
coverage. Problems (c) and (e) can be better resolved because of the
longer read-length. For the same reason the power for the resolution
of alternate splicing isoforms (d) is enhanced (at least for
high-coverage transcripts). Recent versions of \texttt{gsAssembler}
(also called \texttt{Newbler}; Roche/454) provide an opportunity to
asses alternative splicing \cite{pmid21138572}.

The project presented here takes the approach of first using
pyrosequencing to define a reference transcriptome and then mapping
reads from the solexa-platform to this reference.

But also downsteam of the sequence assembly translation of the highly
complex, potentially biased, multidimesional data into biological
relevant knowledge provides computational challenges.

Inferrence of single nucleotide polymorphism (SNPs) requires
statistical categorization in true polymorphisms and sequencing
errors. Tools like \texttt{VarScan} \cite{pmid19542151} or
\texttt{VCFtools} \cite{pmid21653522} combine alignment depth, quality
of the base call in each sequence, quality of mapping to the reference
and the base composition in the region into a statistical
framework. \texttt{GigaBayes} \cite{pmid18204455} uses additionally an
\textit{a priori} expected polymorphism rate. Less attention is
usually paid to indels (insertions or deletions), genomic
rearrangements, copy number polymorphisms caused by local duplication
and other structural variations. While these are common types of
variation between genomes, they can be harder to detect
\cite{pmid22084086}.

Assesment of statistical sigificance of differences in read counts
(from transcirptomic data; also called ``digital transciptomics''),
needs some special treatment in comparison to the well established
methods for microarray-data \cite{smyth2005limma}. While both kinds of
data need normalisation relative to overall transcript abundance
measured (fluorescence or counts), sequencing derived read counts
follow a negative binomial distribution \cite{pmid17728317} instead of
a normal distribution for microarray data. To allow testing for low
numbers of replicates sofware commonly uses global estimates of
variance to restrain and partly replace individual variance. State of
the art methods using these approaches are implemented in the
R-packages \texttt{DESeq} \cite{pmid20979621}, \texttt{edgeR}
\cite{pmid19910308} and \texttt{baySeq} \cite{pmid20698981}.

The functional interpretation of results (from SNP-calling or digital
transcriptomics) linking them to biological meaningful annotation
needs a standardized vocabulary in a datastucture across species and
databases. Gene ontolgy (GO) provides such an vocabulary of controlled
terms.  The terms are organized in an directed, acyclic graph. This
means, that a hierarchical stucture links lower level ``child''-terms
(more specific) to higher level ``parent''-terms (less specific)
through a standardized set of directional relations. Back-links
forming circles are not allowed
\cite{pmid10802651,pmid22123736}. E.g. ``endopeptidase activity'' ``is
a'' ``peptidase activity'', not the other way round. The ``is a`` in
the previous sentence is such a directional realtion and other
possible links would be e.g. ``part of'' or ``regulates''.


\subsection{Applications of NGS in ecology and evolution and
  gene-expression divergence}

\label{sec:appl-ngs}

% Today, both theoretical arguments as well as field and laboratory data
% suggest that evolution, including divergence of populations, can occur
% very rapidly given the right selective pressure. Such situations
% provide us with the opportunity of examining how divergence and even
% speciation work at the molecular genetic level
% \cite{via_ecological_2002}.

Especially pyrosequencing has been used to study the transcriptomes of
organisms with ecological and evolutionary significance.

% Such ecological and evolutionary model organisms often lack reference
% genomes to guide the assembly-process.

% In \textit{Drosophila} variation of gene-expression within a single
% species can be attributed more to trans-regulatory elements, while
% expression divergent between species is dominated by cis-regulatory
% differences \cite{pmid20354124}. Furthermore sterility of hybrids
% between species of this genus has been shown to result from
% incompatibilities in gene-regulatory networks \cite{pmid16757655}.

A study on trout in Lake Superior \cite{pmid20331779} used an approach
similar to the approach in the work presented here: Fish, which show
two different phenotypes were raised in a common environment,
demonstrating the genetic fixation of the phenotypic trait. 454
sequencing was then used to measure the gene expression levels and
successfully identified 40 genes from two biochemical pathways being
differently expressed. However, in addition to showing divergent
evolution of gene-expression, this study highlighted the limitations
of 454 sequencing for gene-expression analysis.

In the seagrass \textit{Zostera marina} northern and southern
populations were subjected to heat stress for a short time-periode.
Analysis of the transcripome using pyrosequencing datad revealed
different patterns of resilience of expression patterns after a heat
wave.

A study on two phylogenetically distant mangrove species found
convergent evolution of gene expression. From the fact, thatff closer
relatives of the studied species living in different ecological niches
do not show the same similarities the study condluded an adaptationf
to the similar environment \cite{dassanayake2009shedding}.

% Positive or diversifying selection on parasite proteins from the
% host-parasite interface can lead to a overabundance of non-synonymous
% changes (altering the protein sequence) over synonymous polymorphisms
% e.g. in \textit{Plasmodium} \cite{pmid7630387}.

% An additional feature of parasite gene-expression is the theoretically
% deduced need to express only a single allele of a polymorphic parasite
% infection locus: In parasites gene-expression is thought to evolve
% towards avoidance of co-expression: For polymorphism to be positively
% selected it requires the evolution of a regulator locus or the
% evolution of polymorphism is followed by the evolution of a regulator
% locus is \cite{pmid15913420}.

% Two virulence factor LbGAP in venom-producing tissues that the major
% virulence factor in the wasp \textit{Leptopilina boulardi} differs
% only quantitatively. The regulation of gene expression might thus be
% major mechanism at the origin of intraspecific variation of virulence
% \cite{pmid21124871}.


% ----------------------------------------------------------------------

%%% Local Variables: ***
%%% mode:latex ***
%%% TeX-master: "../thesis.tex"  ***
%%% tex-main-file: "../thesis.tex" ***
%%% End: ***
     