% this file is called up by thesis.tex
% content in this file will be fed into the main document

%: ----------------------- name of chapter  -------------------------
\chapter{Pyrosequencing of the \textit{A. crassus}
  transcriptome} % top level followed by section, subsection
\label{cha:pyro}

%: ----------------------- paths to graphics ------------------------

% change according to folder and file names
\ifpdf
    \graphicspath{{5_pyro/figures/PNG/}{5_pyro/figures/PDF/}{5_pyro/figures/}}
\else
    \graphicspath{{5_pyro/figures/EPS/}{5_pyro/figures/}}
\fi

%: ----------------------- contents from here ------------------------

\section{Overview}
\label{sec:454-overv}

In this chapter the transcriptome assembly of \textit{A. crassus} is
analysed in its biological context. It constitutes a basis for
molecular research on this important species and furthermore provides
unique insights into the evolution of parasitism in the Spirurina.

After extensive screening of 756,363 raw pyrosequencing reads, I
assembled 353,055 into 11,371 contigs spanning 6,575,121 bases and
additionally obtained 21,147 singleton and lower quality contigs
spanning 6,157,974 bases. I obtained annotations for ca. 60\% of the
contigs and 40\% of the tentatively unique genes (TUGs) confirming the
high quality of especially the contigs. I identified 5,112 high
quality single nucleotide polymorphisms (SNPs) and suggest 199 of them
as most suitable markers for population-genetic studies. Correlation
between different analyses provided further insights and confirmed
biologically relevant expectations: I found an overabundance of
predicted signal peptide cleavage sites in sequence conserved in
Nematoda and novel in \textit{A. crassus}, correlations between coding
polymorphism and differential expression, between coding polymorphism
and peptide cleavage sites and between conservation and presence of
orthologs with lethal RNAi-phenotypes in \textit{C. elegans}. GO-term
analysis identified an enrichment of peptidases and subunits of the
respiratory chain for transcripts under positive selection. Enzymes
for energy metabolism were also found enriched in genes differentially
expressed between European and Asian \textit{A. crassus}.

\section{Sampling \textit{A. crassus}}

One female worm and one male worm were sampled from an aquaculture
with height infection loads in Taiwan. An additional female worm was
sampled from a stream with low infection pressure adjacent to the
aquaculture. All these worms were parasitising endemic
\textit{An. japonica}. A female worm and pool of L2 larval stages were
sampled from \textit{An. anguilla} in the river Rhine, one female worm
from a lake in Poland. All adult worms were filled with large amounts
of host-blood, therefore I anticipated abundant host-contamination in
sequencing data and decided to sequence a liver sample of an
uninfected \textit{An. japonica} for screening.

\section{Sequencing, trimming and pre-assembly screening}

A total of 756,363 raw sequencing reads were generated for
\textit{A. crassus} (see table \ref{screening-lib}). These were
trimmed for base call quality, and filtered by length to give 585,949
high-quality reads (spanning 169,863,104 bases). In the eel dataset from
159,370 raw reads 135,072 were assembled after basic quality screening.

I then screened the \textit{A. crassus} reads for contamination by
host (30,071 matched previously sequenced eel genes or my own
\textit{An. japonica} 454 transcriptome, which had been assembled into
10,639 mRNA contigs. (181,783 reads matched large or small subunit
nuclear or mitochondrial ribosomal RNA sequences of
\textit{A. crassus}) . In addition to fish mRNAs, I identified (and
removed) 5,286 reads in the library derived from the L2 nematodes that
had significant similarity to cercozoan (likely parasite) ribosomal
RNA genes (see table \ref{screening-lib}).

\begin{table}[!h]
\begin{tabular}{rllllll}
   \hline
library & E1 & E2 & L2 & M & T1 & T2 \\ 
   \hline
life.st & adult f & adult f & L2 larvae & adult m & adult f & adult f \\ 
  source.p & Europe R & Europe P & Europe R & Asia C & Asia C & Asia W \\ 
  raw.reads & 209325 & 111746 & 112718 & 106726 & 99482 & 116366 \\ 
  lowqal & 92744 & 10903 & 15653 & 15484 & 7947 & 27683 \\ 
  AcrRNA & 76403 & 11213 & 30654 & 31351 & 24929 & 7233 \\ 
  eelmRNA & 4835 & 3613 & 1220 & 1187 & 7475 & 11741 \\ 
  eelrRNA & 13112 & 69 & 1603 & 418 & 514 & 38 \\ 
  Cercozoa & 0 & 0 & 5286 & 0 & 0 & 0 \\ 
  valid & 22231 & 85948 & 58302 & 58286 & 58617 & 69671 \\ 
  valid.span & 7167338 & 24046225 & 16661548 & 17424408 & 14443123 & 20749177 \\ 
  mapping.unique & 12023 & 65398 & 39690 & 36782 & 42529 & 55966 \\ 
  mapping.Ac &  8359 & 61070 & 12917 & 31656 & 37158 & 50018 \\ 
  mapping.MN &  5883 & 48006 &  8475 & 18986 & 28823 & 41545 \\ 
  over.32 &  3528 & 34051 & 10444 & 21219 & 22435 & 1602 \\ 
  \hline
\end{tabular}
\caption[Pyrosequencing library statistics]{
  \textbf{Pyrosequencing library statistics} - 
  For two sequencing libraries from European eels (E1 and E2)
  one form L2-larvae (L2), one from male (M) and two from Eels in Taiwan
  (T1 and T2) the following statistics are given. life.st = lifecycle
  stage: f for female m for male. source.p = source population: R for
  Rhine, P for Poland, C for cultured, W for wild. raw.reads = raw
  number of sequencing reads obtained. lowqal = number of reads
  discarded due to low quality or length in \texttt{Seqclean}
  \cite{tgicl_pertea}. AcrRNA = number of reads hitting
  \textit{A. crassus}-rRNA (screened). eelmRNA = number of reads hitting
  eel transcriptome-sequences (screened). eelrRNA = number of reads
  hitting eel-rRNA genes (screened). Cercozoa = number of reads hitting
  cercozoan rRNA (screened). valid = number of reads valid after
  screening (assembled). valid.span = number of bases valid (assembled).
  mapping.unique = number of reads mapping uniquely to the
  assembly. mapping.Ac = number of reads mapping to the part of the
  assembly considered \textit{A. crassus} origin (see post-assembly
  screening). mapping.MN = number of reads mapping to the highCA-derived
  part of the assembly (and also \textit{A. crassus} origin). over.32 =
  number of reads mapping to contigs with overall coverage of more than
  32 reads (considered in gene-expression analysis).}
  \label{screening-lib}
\end{table}

\afterpage{\clearpage}

\section{Assembly (see also chapter \ref{chap:eval-ass})}

I assembled the remaining 353,055 reads (spanning 100,491,819 bases)
using the combined assembler strategy \cite{pmid20950480} and Roche
454 GSassembler (\texttt{Newbler} version 2.6) and \texttt{Mira}
(version 3.21) \cite{miraEST}. From this I derived 13,851 contigs that
were supported by both assembly algorithms, 3,745 contigs only
supported by one of the assembly algorithms and 22,591 singletons that
were not assembled by either approach (see table \ref{ass-stat}). When
scored by matches to known genes, the contigs supported by both
assemblers are of the highest credibility, and this set is thus termed
the high credibility assembly (highCA). Those with evidence from only
one assembler and the singletons are of lower credibility
(lowCA). These datasets are the most parsimonious (having the smallest
size) for their quality (covering the largest amount of sequence in
reference transcriptomes). In the highCA parsimony and low redundancy
is prioritised, while in the complete assembly (highCA plus lowCA)
completeness is prioritised. The 40,187 sequences (contig consensuses
and singletons) in the complete assembly are referred to below as
tentatively unique genes (TUGs).

\begin{table}[!h]
  \begin{center}
  \begin{tabular}{rrrr}
  \hline
 & lowCA & highCA & combined \\ 
  \hline
total.contigs & 26336 & 13851 & 40187 \\ 
  rRNA.contigs & 835 & 60 & 895 \\ 
  fish.contigs & 2419 & 1022 & 3441 \\ 
  xeno.contigs & 1935 & 1398 & 3333 \\ 
  remaining.contigs & 21147 & 11371 & 32518 \\ 
  remaining.span & 6157974 & 6575121 & 12733095 \\ 
  non.u.cov & 14.665 & 10.979 & 12.840 \\ 
  cov & 2.443 & 6.838 & 4.624 \\ 
  p4e.BLAST-similarity & 4356 & 5663 & 10019 \\ 
  p4e.ESTScan & 8324 & 3597 & 11921 \\ 
  p4e.LongestORF & 8347 & 2085 & 10432 \\ 
  p4e.no-prediction & 93 & 14 & 107 \\ 
  full.3p & 5906 & 2714 & 8620 \\ 
  full.5p & 1484 & 1270 & 2754 \\ 
  full.l & 104 & 185 & 289 \\ 
  GO & 2635 & 3874 & 6509 \\ 
  EC & 966 & 1492 & 2458 \\ 
  KEGG & 1608 & 2236 & 3844 \\ 
  IPR & 0 & 7557 & 7557 \\ 
  nem.blast & 4868 & 5820 & 10688 \\ 
  any.blast & 5106 & 6007 & 11113 \\ 
   \hline
\end{tabular}
\caption[Assembly classification and contig
statistics]{\textbf{Assembly classification and contig statistics} -
  Summary statistics for contigs from different assembly-categories
  given in columns as highCA = high credibility assembly; lowCA = low
  credibility assembly, combined = complete assembly. Rows indicate
  summary statistics: total.contigs = numbers of total contigs,
  fish.contigs = number of contigs hitting eel-mRNA or Chordata in
  NCBI-nr or NCBI-nt (screened out), xeno.contigs = number of contigs
  with best hit (NCBI-nr and NCBI-nt) to non-eukaryote (screened out),
  remaining.contigs = number of contigs remaining after this
  screening, remaining.span = total length of remaining contigs,
  non.u.cov = non-unique mean base coverage of contigs, cov = unique
  mean base coverage of contigs, p4e.``X'' = number protein
  predictions derived in p4e, where ``X'' describes the method of
  prediction (see \ref{pro4est}), full.3p = number of contigs complete
  at 3', full.5p = number of contigs complete at 5', GO = number of
  contigs with GO-annotation, KEGG = number of contigs with
  KEGG-annotation, EC = number of contigs with EC-annotation,
  nem.blast = number of contigs with \texttt{BLAST}-hit to nematode in
  nr, any.blast = number of contigs with \texttt{BLAST}-hit to
  non-nematode (eukaryote non chordate) sequence in NCBI-nr.}
\label{ass-stat}
  \end{center}
\end{table}

\afterpage{\clearpage}

I screened the complete assembly for residual host contamination, and
identified 3,441 TUGs that had higher, significant similarity to eel
(and chordate) sequences (my 454 ESTs and EMBLBank Chordata proteins)
than to nematode sequences \cite{pmid21550347}.

Given my prior identification of cercozoan ribosomal RNAs, I also
screened the complete assembly for contamination with other
transcriptomes.

1,153 TUGs were found mapping to Eukaryota outside of the kingdoms
Metazoa, Fungi and Viridiplantae. These hits included a wide range of
Protists ranging from Apicomplexa (mainly Sarcocystidae, 28 hits and
Cryptosporidiidae 10 hits) over Bacillariophyta (diatoms, mainly
Phaeodactylaceae, 41 hits) and Phaeophyceae (brown algae, mainly
Ectocarpaceae, 180 hits) and Stramenopiles (Albuginaceae, 63 hits) to
Kinetoplasitda (Trypanosomatidae, 26 hits) and Heterolobosea
(Vahlkampfiidae, 38 hits).

Additionally I found 298 TUGs with hits to fungi (e.g
Ajellomycetaceae, 53 hits) and 585 TUGs with hits to plants.

Hits outside the Eukaryota were mainly to Bacteria (825 hits) and
within those mostly to members of the Proteobacteria (484 hits). No
hits were found to Wolbachia or related Bacteria known as symbionts of
nematodes and arthropods. 9 TUGs were hitting sequence from Viruses
and 8 from Archaea.

I excluded all TUGs with best hits outside Metazoa and my assembly
thus has 32,518 TUGs, spanning 12,733,095 bases (of which 11,371 are
highCA-derived, and span 6,575,121 bases) that are likely to derive
from \textit{A. crassus}.

\section{Protein prediction}

For 32,411 TUGs a protein was predicted using \texttt{prot4EST}
\cite{wasmuth_prot4est:_2004} (see table \ref{ass-stat}). The full
open reading frame was obtained in 353 TUGs, while while for 2,683 the
5' end and for 8,283 the 3' end was complete. In 13,379 TUGs the
corrected sequence with the imputed ORF was slightly changed compared
to the raw sequence.

\section{Annotation}
\label{454-annot}

I obtained basic annotations with orthologous sequences from
\textit{C. elegans} for 9,554 TUGs, from \textit{B. malayi} for 9,662
TUGs, from nempep \cite{parkinson_nembase:resource_2004, pmid21550347}
for 11,617 TUGs and with uniprot proteins for 11,113 TUGs.

I used \texttt{annot8r} \cite{schmid_annot8r:_2008} to assign gene
ontology (GO) terms for 6,509 TUGs, Enzyme Commission (EC) numbers for
2,458 TUGs and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway
annotations for 3,844 TUGs (see table \ref{ass-stat}). Additionally
5,125 highCA derived contigs were annotated with GO terms through
\texttt{InterProScan} \cite{pmid11590104}. Nearly one third (6,987) of
the \textit{A. crassus} TUGs were annotated with at least one
identifier, and 1,829 had GO, EC and KEGG annotations (see figure
\ref{ann_venn_man}).

\figuremacroW{ann_venn_man}{Annotation using different identifiers} {Number
  of annotations obtained for Gene Ontology (GO), Enzyme Commission
  (EC) and Kyoto Encyclopedia of Genes and Genomes (KEGG) terms
  through \texttt{Annot8r} \cite{schmid_annot8r:_2008} for all TUGs
  (a) and for highCA derived contigs (b). The latter includes
  additional domain-based annotations obtained with
  \texttt{InterProScan} \cite{pmid11590104}.}{0.65}

I compared my \textit{A. crassus} GO annotations for high-level
GO-slim terms to the annotations (obtained the same way) for the
complete proteome of the filarial nematode \textit{B. malayi} and the
complete proteome of \textit{C. elegans} (see figure \ref{slim_com}).

Correlation shows the occurrence of terms for the partial
transcriptome of \textit{A. crassus} to be more similar to the
proteome of \textit{B. malayi} (0.95; Spearman correlation
coefficient) than to the proteome of \textit{C. elegans} (0.9). Also
the tow model-nematode compared to each other (0.91) are less similar
in the occurrence of terms than the two parasites.

\figuremacro{slim_com}{Cross-taxa comparison of annotation}{For Gene
  Ontology (GO) categories molecular function, cellular compartment
  and biological process the proportion (for each ontology-category
  and species) of terms in high level GO-slim categories is given as
  obtained through \texttt{Annot8r} \cite{schmid_annot8r:_2008}.}

I inferred presence of signal peptide cleavage sites in the predicted
protein sequence using \texttt{SignalP} \cite{pmid21959131}. I
predicted 920 signal peptide cleavage sites and 65 signal peptides
with a transmembrane signature. Again these predictions are more
similar to predictions using the same methods for the proteome
\textit{B. malayi} (742 signal peptide cleavage sites and 41 with
transmembrane anchor) than for the proteome of \textit{C. elegans}
(4273 signal peptide cleavage sites and 154 with transmembrane
anchor).

I inferred the presence of a lethal RNAi phenotype in the orthologous
annotation of \textit{C. elegans}. For 257 TUGs a non-lethal phenotype
was inferred for 6029 TUGs a lethal phenotype.

\section{Evolutionary conservation}

\textit{A. crassus} TUGs were classified as conserved, conserved in
Metazoa, conserved in Nematoda, conserved in Spirurina or novel to
\textit{A. crassus} by comparing them to public databases and using
two \texttt{BLAST} bit-score cutoffs to define relatedness (see table
\ref{evol-con}).

\begin{table}[!h]
\begin{tabular}{rrrrrr}
  \hline
 & conserved & novel.in.m & novel.in.n & novel.in.cl3 & novel.in.Ac \\ 
  \hline
bit.50.all & 5604 & 1713 & 2173 & 1485 & 21543 \\ 
  bit.80.all & 3506 & 1382 & 2014 & 1525 & 24091 \\ 
  bit.50.highCA & 3479 & 875 & 1010 & 601 & 5406 \\ 
  bit.80.highCA & 2457 & 832 & 1084 & 716 & 6282 \\ 
   \hline
\end{tabular}
\caption[Evolutionary conservation and novelty]{\textbf{Evolutionary
    conservation and novelty} - The kingdom Metazoa (novel.in.m),
  the phylum Nematoda (novel.in.n) and clade III (Spirurina;
  novel.in.cl3) were assessed for occurrences of
  \texttt{BLAST}-hits at two different bitscore thresholds (50 =
  bit.50 and 80 = bit.80). TUGs without any hit at a given threshold 
  were categorised as novel in \textit{A. crassus} (novel.in.Ac).
  Both novelty and conservation can be
  derived from this (numbers for conservation would be the cumulative
  sum of lower-level novelty).}
\label{evol-con}
\end{table}

\afterpage{\clearpage}

Roughly a third and a quarter of the highCA derived contigs were
categorised as conserved across kingdoms at a bitscore threshold of 50
and 80, respectively. Roughly half or 3/5 of the these contigs were
identified as novel in \textit{A. crassus}.

The remaining highCA contigs spread across intermediate
relatedness-levels. More sequences were categorised as novel at the
phylum level (Nematoda) compared to kingdom and clade III level and
the number of contigs at intermediate relatedness-levels was roughly
consistent for the two bitscore thresholds.

The latter points about intermediate conservation levels were also
true, when all TUGs were analysed. The numbers of TUGs categorised at
these intermediate levels roughly doubled. In contrast, the proportion
of additional conserved lowCA TUGs is small compared to additional
TUGs categorised as novel in \textit{A. crassus}, mirroring the higher
amount of erroneous sequence.

Proteins predicted to be novel to Nematoda and novel in
\textit{A. crassus} were significantly enriched in signal peptide
annotation compared to conserved proteins, proteins novel in Metazoa
and novel in clade III (Fisher's exact test p$<$0.001 ;
\ref{nov_sig}).

\figuremacro{nov_sig}{Enrichment of signal-positives for categories of
  evolutionary conservation}{Proportions of
  \texttt{SignalP}-predictions for each category of evolutionary
  conservation. Generally - across bit-score thresholds - TUGS novel
  in nematodes and in \textit{A. crassus} have the highest proportion
  of signal-positives. sigP = signalIP-prediction; Yes-noTM, cleavage
  site predicted; Yes-TM, transmembrane-anchor predicted.}

The proportion of lethal RNAi phenotypes was significantly higher for
orthologs of conserved TUGs (97.23\%) than for orthologs of TUGs not
conserved (94.65\%) across kingdoms (p$<$0.001, Fisher's exact test).

\section{Identification of single nucleotide polymorphisms}

I called single nucleotide polymorphisms (SNPs) on the 1,099,419 bases
of the TUGs that had coverage of more then 8-fold available using
\texttt{VARScan} \cite{pmid19542151}. I excluded SNPs predicted to
have more than 2 alleles or that mapped to an undetermined (N) base in
the reference, and retained 10,458 SNPs. The ratio of transitions (ti;
6,890) to transversion (tv; 3568) in this set was 1.93. Using the
prot4EST predictions and the corrected sequences, 7,153 of the SNPs
were predicted to be inside an ORF, with 2,310 at codon first
positions, 1,819 at second positions and 3,024 at third positions. As
expected ti/tv inside ORFs (2.41) was higher than outside ORFs
(1.25). The ratio of synonymous polymorphisms per synonymous site to
non-synonymous polymorphisms per non-synonymous site (dn/ds) was
0.42. I filtered these SNPs to exclude those that might be associated
with analytic bias. As Roche 454 sequences have well-known
systematic errors associated with homopolymeric nucleotide sequences
\cite{pmid21685085}, I analysed the effect of exclusion of SNPs in, or
close to, homopolymer regions. I observed changes in ti/tv and in
dn/ds when SNPs were discarded using different size thresholds for
homopolymer runs and proximity thresholds (see figure \ref{snp_hom}).

\figuremacro{snp_hom}{Homopolymer screening for SNP-calling}{When SNPs
  in or adjacent to homopolymeric regions are removed changes in ti/tv
  (a) and dn/ds (b) are observed: As the overall number of SNPs is
  reduced both ratios change to more plausible values. Note the
  reversed axis for dn/ds to plot these lower values to the right. For
  homopolymer length $>$ 3 a linear trend for the total number of SNPs
  and the two measurements is observed. A width of 11 for the
  screening window provides most plausible values (suggesting
  specificity) while still incorporating a high number of SNPs
  (sensitivity).}

Based on this I decided to exclude SNPs with a homopolymer-run as long
as or longer than 4 bases inside a window of 11 bases (5 to bases to
the right, 5 to the left) around the SNP. I also observed a
relationship between TUG dn/ds and TUG coverage, associated with the
presence of sites with low abundance minority alleles (less than 7\%
of the allele calls), suggesting that some of these may be errors.
Removing low abundance minority allele SNPs from the set removed this
effect (see figure \ref{snp_final}). My filtered SNP dataset
includes 5,112 SNPs. I retained 4.65 SNPs per kb of contig sequence,
with 8.37 synonymous SNPs per 1,000 synonymous bases and 2.4
non-synonymous SNPs per 1,000 non-synonymous bases. A mean dn/ds of
0.231 was calculated for the 859 TUGs (762 highCA-derived contigs)
containing at least one synonymous SNP.

\figuremacro{snp_final}{SNP-calling and SNP categories}{Overabundance
  of SNPs at (a) codon-position two and of (c) non-synonymous SNPs for
  low percentages of the minority allele. (b) Significant positive
  correlation of coverage and dn/ds before removing these SNPs at a
  threshold of 7\% ($p<$ 0.001, $R^2=$ 0.015) and (d) no significant
  correlation afterwards ($R^2<$0.001, $p=$0.211).}

\section{Polymorphisms associated with biological processes}

I consolidated my annotation and polymorphism analyses by examining
correlations between nonsynonymous variability and particular
classifications.

Signal peptide containing proteins have been shown to have higher
rates of evolution than cytosolic proteins in a number of nematode
species. In \textit{A. crassus}, TUGs predicted to contain signal
peptide cleavage sites in SignalP showed a trend towards higher dn/ds
values than TUGs without signal peptide cleavage sites (p = 0.074; two
sided Mann-Whitney-test).

Positive selection can be inferred from dn/ds analyses, and I defined
TUGs with a dn/ds higher than 0.5 as positively selected. I identified
over-represented GO ontology terms associated with these putatively
positively selected genes (see table \ref{go-pos} and additional
figures \ref{tGO_DN_DS_BP_classic_10_all},
\ref{tGO_DN_DS_CC_classic_10_all} and
\ref{tGO_DN_DS_MF_classic_10_all}).

\begin{longtable}{lp{4.5cm}rrrl}
  \hline
  GO.ID & Term & Annotated & Significant & Expected & p-value \\ 
  \endfirsthead
  \multicolumn{6}{c}%
  {{\bfseries \tablename\ \thetable{} -- continued from previous page}} \\
  \hline
  GO.ID & Term & Annotated & Significant & Expected & p-value \\ 
  \hline 
  \endhead
  \hline
  \multicolumn{6}{|r|}{{Continued on next page}} \\ 
  \hline
  \endfoot
  \endlastfoot
  \hline
  \multicolumn{6}{l}{Molecular function} \\ 
  GO:0008233 & peptidase activity &  43 &  12 & 5.26 & 0.0028 \\ 
  GO:0015179 & L-amino acid transmembrane transporter activity &   2 &   2 & 0.24 & 0.0147 \\ 
  GO:0016787 & hydrolase activity & 110 &  20 & 13.45 & 0.0262 \\ 
  GO:0043021 & ribonucleoprotein binding &   6 &   3 & 0.73 & 0.0266 \\ 
  GO:0005102 & receptor binding &  26 &   7 & 3.18 & 0.0288 \\ 
  GO:0046982 & protein heterodimerization activity &  16 &   5 & 1.96 & 0.0348 \\ 
  GO:0004129 & cytochrome-c oxidase activity &   3 &   2 & 0.37 & 0.0407 \\ 
  GO:0004540 & ribonuclease activity &   3 &   2 & 0.37 & 0.0407 \\ 
  GO:0005275 & amine transmembrane transporter activity &   3 &   2 & 0.37 & 0.0407 \\ 
  GO:0005342 & organic acid transmembrane transporter activity &   3 &   2 & 0.37 & 0.0407 \\ 
  GO:0005275 & amine transmembrane transporter activity &   3 &   2 & 0.37 & 0.0407 \\ 
  GO:0005342 & organic acid transmembrane transporter activity &   3 &   2 & 0.37 & 0.0407 \\ 
  GO:0015002 & heme-copper terminal oxidase activity &   3 &   2 & 0.37 & 0.0407 \\ 
  GO:0015171 & amino acid transmembrane transporter activity &   3 &   2 & 0.37 & 0.0407 \\ 
  GO:0016675 & oxidoreductase activity, acting on a heme group of donors &   3 &   2 & 0.37 & 0.0407 \\ 
  GO:0016676 & oxidoreductase activity, acting on a heme group of donors, oxygen as acceptor &   3 &   2 & 0.37 & 0.0407 \\ 
  GO:0046943 & carboxylic acid transmembrane transporter activity &   3 &   2 & 0.37 & 0.0407 \\ 
  GO:0047035 & testosterone dehydrogenase (NAD+) activity &   3 &   2 & 0.37 & 0.0407 \\ 
  GO:0015077 & monovalent inorganic cation transmembrane transporter activity &  12 &   4 & 1.47 & 0.0471 \\ 
  \hline
   \multicolumn{6}{l}{Biological process}  \\ 
   GO:0009081 & branched chain family amino acid metabolic process &   3 &   3 & 0.36 & 0.0017 \\ 
  GO:0042594 & response to starvation &  15 &   6 & 1.82 & 0.0052 \\ 
  GO:0006914 & autophagy &  12 &   5 & 1.45 & 0.0090 \\ 
  GO:0006520 & cellular amino acid metabolic process &  44 &  11 & 5.33 & 0.0102 \\ 
  GO:0007281 & germ cell development &  17 &   6 & 2.06 & 0.0105 \\ 
  GO:0090068 & positive regulation of cell cycle process &  17 &   6 & 2.06 & 0.0105 \\ 
  GO:0009308 & amine metabolic process &  57 &  13 & 6.90 & 0.0118 \\ 
  GO:0051325 & interphase &  23 &   7 & 2.79 & 0.0139 \\ 
  GO:0051329 & interphase of mitotic cell cycle &  23 &   7 & 2.79 & 0.0139 \\ 
    GO:0010564 & regulation of cell cycle process &  34 &   9 & 4.12 & 0.0140 \\ 
  GO:0051726 & regulation of cell cycle &  52 &  12 & 6.30 & 0.0143 \\ 
  GO:0005997 & xylulose metabolic process &   2 &   2 & 0.24 & 0.0145 \\ 
  GO:0006739 & NADP metabolic process &   2 &   2 & 0.24 & 0.0145 \\ 
  GO:0009744 & response to sucrose stimulus &   2 &   2 & 0.24 & 0.0145 \\ 
  GO:0010172 & embryonic body morphogenesis &   2 &   2 & 0.24 & 0.0145 \\ 
  GO:0015807 & L-amino acid transport &   2 &   2 & 0.24 & 0.0145 \\ 
  GO:0019321 & pentose metabolic process &   2 &   2 & 0.24 & 0.0145 \\ 
  GO:0034285 & response to disaccharide stimulus &   2 &   2 & 0.24 & 0.0145 \\ 
  GO:0050885 & neuromuscular process controlling balance &   2 &   2 & 0.24 & 0.0145 \\ 
  GO:0006915 & apoptosis &  78 &  16 & 9.45 & 0.0147 \\ 
  GO:0009056 & catabolic process & 149 &  26 & 18.04 & 0.0148 \\ 
  GO:0031571 & mitotic cell cycle G1/S transition DNA damage checkpoint &  14 &   5 & 1.70 & 0.0187 \\ 
  GO:0044106 & cellular amine metabolic process &  55 &  12 & 6.66 & 0.0224 \\ 
  GO:0009063 & cellular amino acid catabolic process &  10 &   4 & 1.21 & 0.0234 \\ 
  GO:0000082 & G1/S transition of mitotic cell cycle &  15 &   5 & 1.82 & 0.0255 \\ 
  GO:0030330 & DNA damage response, signal transduction by p53 class mediator &  15 &   5 & 1.82 & 0.0255 \\ 
  GO:0033238 & regulation of cellular amine metabolic process &  15 &   5 & 1.82 & 0.0255 \\ 
  GO:0042770 & signal transduction in response to DNA damage &  15 &   5 & 1.82 & 0.0255 \\ 
  GO:0072331 & signal transduction by p53 class mediator &  15 &   5 & 1.82 & 0.0255 \\ 
  GO:0006401 & RNA catabolic process &   6 &   3 & 0.73 & 0.0259 \\ 
  GO:0010638 & positive regulation of organelle organization &   6 &   3 & 0.73 & 0.0259 \\ 
  GO:0042981 & regulation of apoptosis &  64 &  13 & 7.75 & 0.0312 \\ 
  GO:0043067 & regulation of programmed cell death &  64 &  13 & 7.75 & 0.0312 \\ 
  GO:0009310 & amine catabolic process &  11 &   4 & 1.33 & 0.0335 \\ 
  GO:0051084 & 'de novo' posttranslational protein folding &  11 &   4 & 1.33 & 0.0335 \\ 
  GO:0008219 & cell death &  93 &  17 & 11.26 & 0.0370 \\ 
  GO:0016265 & death &  93 &  17 & 11.26 & 0.0370 \\ 
  GO:0012501 & programmed cell death &  86 &  16 & 10.41 & 0.0371 \\ 
  GO:0010941 & regulation of cell death &  66 &  13 & 7.99 & 0.0396 \\ 
  GO:0000393 & spliceosomal conformational changes to generate catalytic conformation &   3 &   2 & 0.36 & 0.0400 \\ 
  GO:0006123 & mitochondrial electron transport, cytochrome c to oxygen &   3 &   2 & 0.36 & 0.0400 \\ 
  GO:0006865 & amino acid transport &   3 &   2 & 0.36 & 0.0400 \\ 
  GO:0009313 & oligosaccharide catabolic process &   3 &   2 & 0.36 & 0.0400 \\ 
  GO:0031023 & microtubule organizing center organization &   3 &   2 & 0.36 & 0.0400 \\ 
  GO:0045292 & nuclear mRNA cis splicing, via spliceosome &   3 &   2 & 0.36 & 0.0400 \\ 
  GO:0045840 & positive regulation of mitosis &   3 &   2 & 0.36 & 0.0400 \\ 
  GO:0051262 & protein tetramerization &   3 &   2 & 0.36 & 0.0400 \\ 
  GO:0051289 & protein homotetramerization &   3 &   2 & 0.36 & 0.0400 \\ 
  GO:0051297 & centrosome organization &   3 &   2 & 0.36 & 0.0400 \\ 
  GO:0051785 & positive regulation of nuclear division &   3 &   2 & 0.36 & 0.0400 \\ 
  GO:2000242 & negative regulation of reproductive process &   3 &   2 & 0.36 & 0.0400 \\ 
  GO:0007286 & spermatid development &   7 &   3 & 0.85 & 0.0415 \\ 
  GO:0009267 & cellular response to starvation &   7 &   3 & 0.85 & 0.0415 \\ 
  GO:0048515 & spermatid differentiation &   7 &   3 & 0.85 & 0.0415 \\ 
  GO:0016071 & mRNA metabolic process &  47 &  10 & 5.69 & 0.0437 \\ 
  GO:0006458 & 'de novo' protein folding &  12 &   4 & 1.45 & 0.0457 \\ 
  GO:0022607 & cellular component assembly & 103 &  18 & 12.47 & 0.0484 \\ 
  \hline
  \multicolumn{6}{l}{Cellular compartment}  \\ 
  GO:0030532 & small nuclear ribonucleoprotein complex &   7 &   4 & 0.84 & 0.005 \\ 
  GO:0005682 & U5 snRNP &   2 &   2 & 0.24 & 0.014 \\ 
  GO:0015030 & Cajal body &   2 &   2 & 0.24 & 0.014 \\ 
  GO:0046540 & U4/U6 x U5 tri-snRNP complex &   2 &   2 & 0.24 & 0.014 \\ 
  GO:0016607 & nuclear speck &   6 &   3 & 0.72 & 0.025 \\ 
  GO:0005739 & mitochondrion & 136 &  23 & 16.35 & 0.031 \\ 
  GO:0005604 & basement membrane &   3 &   2 & 0.36 & 0.039 \\ 
  GO:0060198 & clathrin sculpted vesicle &   3 &   2 & 0.36 & 0.039 \\ 
\hline\\
\caption[Over-representation of GO-terms in positively
selected]{\textbf{Over-representation of GO-terms in positively
    selected} - GO-terms over-represented in contigs putatively under
  positive selection. Horizontal lines separate categories of the
  GO-ontology. First category is molecular function, second biological
  process, last cellular compartment. P values (pval) for over-
  representation (Fishters exact test) are given along with the number
  of positively selected contigs (Count; dn/ds $>$ 0.5) and the number
  of contigs with this annotation for which a dn/ds was obtained
  (Size) and the description of the GO-term (Term) see also additional
  figures \ref{tGO_DN_DS_BP_classic_10_all},
  \ref{tGO_DN_DS_CC_classic_10_all} and
  \ref{tGO_DN_DS_MF_classic_10_all}.}\\
\label{go-pos}
\end{longtable}

Within the molecular function category, ``peptidase activity'' was the
most significantly overrepresented term and had twelve TUGs supporting the
overrepresentation. The highlighted twelve peptidases annotated with
eleven unique orthologs in \textit{C. elegans} and \textit{B. malayi}.
Other overrepresented terms abundant over categories pointed to
subunits of the respiratory chain e.g. ``heme-copper terminal oxidase
activity'' and ``cytochrome-c oxidase activity'' in molecular function
and ``mitochondrion'' in cellular compartment and to amino and fatty
acid catabolic processes.

At both bitscore thresholds contigs novel in clade III and novel in
\textit{A. crassus} had a significantly higher dn/ds than other
contigs (novel.in.metazoa - novel.in.Ac, 0.005 and 0.015;
novel.in.nematoda - novel.in.Ac, 0.005 and 0.002; novel.in.nematoda -
novel.in.clade3, 0.207 and 0.045; comparison, p-value from bitscore of
50 and p-value from bitscore of 80, Nemenyi-Damico-Wolfe-Dunn test,
given only for significant comparisons; figure \ref{dn_ds_con}).

\figuremacro{dn_ds_con}{Positive selection and evolutionary
  conservation}{Box-plots for dn/ds in TUGs according to different
  categories of evolutionary conservation. Significant comparisons are
  novel.in.metazoa - novel.in.Ac (0.005 and 0.015), novel.in.nematoda
  - novel.in.Ac (0.005 and 0.002), novel.in.nematoda - novel.in.clade3
  (0.207 and 0.045; p-value for bitscore of 50 and 80,
  Nemenyi-Damico-Wolfe-Dunn test).}

Orthologs of \textit{C. elegans} transcripts with lethal RNAi
phenotype are expected to evolve under stronger selective
constraints. Indeed the values of dn/ds showed a non-significant trend
towards lower values in TUGs with orthologs with a lethal phenotype
compared to a non-lethal phenotypes (p=0.138, two-sided U-test).

\section{SNP markers for single worms}
\label{sing-w}

I used \texttt{Samtools}\cite{journals/bioinformatics/LiHWFRHMAD09}
and \texttt{Vcftools}\cite{pmid21653522} to call genotypes in single
worms (adult sequencing libraries). This resulted in 199 informative
sites in 152 contigs, where two alleles were found in at least one
assured genotype at least in one of the worms.

\begin{table}[ht]
\begin{center}
\begin{tabular}{rrrrr}
  \hline
 & rel.het & int.rel & ho.loci & std.het \\ 
  \hline
T2 & 0.45 & -0.73 & 0.59 & 1.00 \\ 
  T1 & 0.93 & -0.95 & 0.34 & 1.62 \\ 
  M & 0.37 & -0.73 & 0.66 & 0.84 \\ 
  E1 & 0.38 & -0.83 & 0.60 & 0.91 \\ 
  E2 & 0.18 & -0.35 & 0.82 & 0.50 \\ 
   \hline
\end{tabular}
\caption[Measurements of multi-locus heterozygosity for single
worms]{\textbf{Measurements of multi-locus heterozygosity for single
    worms} - Genotyping for a set of 199 SNPs, different measurements
  were obtained to asses genome-wide heterozygosity.  Measurements for
  relative heterozygosity (rel.het; number of homozygous sites/ number
  of heterozygous sites), internal relatedness (int.rel;
  \cite{pmid11571049}), homozygosity by loci (ho.loci;
  \cite{pmid17107491}) and standardised heterozygosity (std.het;
  \cite{coltman81j}) are given.  All these measurements are pointing
  to sample T1 (Taiwanese worm from a wild population) as the most
  heterozygous and sample E2 (the European worm from Poland) as the
  least heterozygous individual. Heterozygote-heterozygote correlation
  \cite{pmid21565077} confirmed the genome-wide significance of these
  markers.}
\label{snp-sing}
\end{center}
\end{table}

Internal relatedness \cite{pmid11571049}, homozygosity by loci
\cite{pmid17107491} and standardised heterozygosity \cite{coltman81j}
were all highlighting the Taiwanese worm from the wild population
(sample T1) as the most and the European worm from Poland (sample E2)
as the least heterozygous individual. The other worms had intermediate
values between these two extremes (see table \ref{snp-sing}).

I confirmed the genome-wide significance of these estimates using
heterozygosity-heterozygosity correlation \cite{pmid21565077}. These
tests confirmed the representativeness of the 199 SNP-markers for the
whole genome in population genetic studies ($\mu$ = 0.78,
$ci_l$=0.444; $\mu$ = 0.86 and $ci_l$ = 0.596; $\mu$ = 0.87 and
$ci_l$= 0.632; mean and lower bound of 95\% confidence intervals from
1000 bootstrap replicates for internal relatedness, homozygosity by
loci and standardised heterozygosity). Using a higher number of
genotyped individuals these markers would allow to asses the amount of
inbreeding in populations of \textit{A. crassus}.


\section{Differential expression}

I also analysed gene-expression inferred from mapping. Of the 353,055
reads 252,388 (71.49\%) mapped uniquely (with their best hit) to the
fullest assembly (including the all assembled contigs as a ``filter''
later removing screened out sequences for analysis). The number of
reads mapping is given for each library in table \ref{screening-lib},
to get unbiased estimates of expression I removed also all contigs
with a coverage lower than 32 reads overall and thus analysed 658
contigs.

Using the statistics of of Audic and Claverie \cite{pmid9331369} and
filtering for relevant contrasts, 54 contigs showed an expression
predominantly in the male library, 56 contigs in the female
library. 56 contigs were primarily expressed in the libraries from
Taiwan, 22 contigs in the European library.

Overrepresentation of of GO-terms differentially expressed between the
male and female libraries highlighted especially ribosomal proteins,
oxidoreductases and collagen processing enzymes as enriched (table
\ref{over-de-mf-454} and additional figures
\ref{tGO_SEX_EXP_BP_classic_10_all},
\ref{tGO_SEX_EXP_CC_classic_10_all} and
\ref{tGO_SEX_EXP_MF_classic_10_all}). These ribosomal proteins were
all overexpressed in the male library, oxidoreductases and collagen
processing enzymes were all overexpressed female libraries.


\begin{longtable}{lp{4.5cm}rrrl}
  \hline
  GO.ID & Term & Annotated & Significant & Expected & p-value \\ 
  \endfirsthead
  \multicolumn{6}{c}%
  {{\bfseries \tablename\ \thetable{} -- continued from previous page}} \\
  \hline
  GO.ID & Term & Annotated & Significant & Expected & p-value \\ 
  \hline 
  \endhead
  \hline
  \multicolumn{6}{|r|}{{Continued on next page}} \\ 
  \hline
  \endfoot
  \endlastfoot
  \hline
  \multicolumn{6}{l}{Molecular function} \\ 
  GO:0005198 & structural molecule activity &  51 &  18 & 8.28 & 0.00019 \\ 
  GO:0016706 & oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxyge... &   3 &   3 & 0.49 & 0.00407 \\ 
  GO:0004656 & procollagen-proline 4-dioxygenase activity &   2 &   2 & 0.32 & 0.02595 \\ 
  GO:0031543 & peptidyl-proline dioxygenase activity &   2 &   2 & 0.32 & 0.02595 \\ 
  GO:0034641 & cellular nitrogen compound metabolic process & 159 &  37 & 25.03 & 0.00020 \\ 
   \hline
   \multicolumn{6}{l}{Biological process}  \\ 
GO:0048731 & system development & 146 &  35 & 22.98 & 0.00020 \\ 
  GO:0034621 & cellular macromolecular complex subunit organization &  73 &  22 & 11.49 & 0.00026 \\ 
  GO:0006807 & nitrogen compound metabolic process & 162 &  37 & 25.50 & 0.00034 \\ 
  GO:0032774 & RNA biosynthetic process &  70 &  21 & 11.02 & 0.00043 \\ 
  GO:0071822 & protein complex subunit organization &  71 &  21 & 11.18 & 0.00055 \\ 
  GO:0043933 & macromolecular complex subunit organization &  82 &  23 & 12.91 & 0.00063 \\ 
  GO:0000022 & mitotic spindle elongation &  19 &   9 & 2.99 & 0.00080 \\ 
  GO:0051231 & spindle elongation &  19 &   9 & 2.99 & 0.00080 \\ 
  GO:0044281 & small molecule metabolic process & 188 &  40 & 29.59 & 0.00082 \\ 
  GO:0006139 & nucleobase-containing compound metabolic process & 139 &  32 & 21.88 & 0.00157 \\ 
  GO:0048856 & anatomical structure development & 188 &  39 & 29.59 & 0.00241 \\ 
  GO:0071841 & cellular component organization or biogenesis at cellular level & 139 &  31 & 21.88 & 0.00408 \\ 
  GO:0090304 & nucleic acid metabolic process & 105 &  25 & 16.53 & 0.00546 \\ 
  GO:0071842 & cellular component organization at cellular level & 135 &  30 & 21.25 & 0.00559 \\ 
  GO:0016070 & RNA metabolic process &  96 &  23 & 15.11 & 0.00797 \\ 
  GO:0040007 & growth & 138 &  30 & 21.72 & 0.00847 \\ 
  GO:0050789 & regulation of biological process & 198 &  39 & 31.17 & 0.00952 \\ 
  GO:0042274 & ribosomal small subunit biogenesis &  10 &   5 & 1.57 & 0.01084 \\ 
  GO:0009791 & post-embryonic development & 116 &  26 & 18.26 & 0.01151 \\ 
  GO:0007275 & multicellular organismal development & 221 &  42 & 34.79 & 0.01156 \\ 
  GO:0022414 & reproductive process & 105 &  24 & 16.53 & 0.01280 \\ 
  GO:0042157 & lipoprotein metabolic process &   7 &   4 & 1.10 & 0.01335 \\ 
  GO:0007051 & spindle organization &  27 &   9 & 4.25 & 0.01435 \\ 
  GO:0007052 & mitotic spindle organization &  27 &   9 & 4.25 & 0.01435 \\ 
  GO:0040009 & regulation of growth rate &  62 &  16 & 9.76 & 0.01599 \\ 
  GO:0040010 & positive regulation of growth rate &  62 &  16 & 9.76 & 0.01599 \\ 
  GO:0018988 & molting cycle, protein-based cuticle &  23 &   8 & 3.62 & 0.01616 \\ 
  GO:0010467 & gene expression & 114 &  25 & 17.94 & 0.01935 \\ 
  GO:0042303 & molting cycle &  24 &   8 & 3.78 & 0.02127 \\ 
  GO:0071840 & cellular component organization or biogenesis & 171 &  34 & 26.92 & 0.02143 \\ 
  GO:0032501 & multicellular organismal process & 241 &  44 & 37.94 & 0.02183 \\ 
  GO:0009416 & response to light stimulus &   8 &   4 & 1.26 & 0.02360 \\ 
  GO:0032502 & developmental process & 227 &  42 & 35.73 & 0.02409 \\ 
  GO:0008543 & fibroblast growth factor receptor signaling pathway &   2 &   2 & 0.31 & 0.02437 \\ 
  GO:0018401 & peptidyl-proline hydroxylation to 4-hydroxy-L-proline &   2 &   2 & 0.31 & 0.02437 \\ 
  GO:0019471 & 4-hydroxyproline metabolic process &   2 &   2 & 0.31 & 0.02437 \\ 
  GO:0019511 & peptidyl-proline hydroxylation &   2 &   2 & 0.31 & 0.02437 \\ 
  GO:0046887 & positive regulation of hormone secretion &   2 &   2 & 0.31 & 0.02437 \\ 
  GO:0071570 & cement gland development &   2 &   2 & 0.31 & 0.02437 \\ 
  GO:0000279 & M phase &  44 &  12 & 6.93 & 0.02555 \\ 
  GO:0009792 & embryo development ending in birth or egg hatching & 123 &  26 & 19.36 & 0.02787 \\ 
  GO:0016043 & cellular component organization & 167 &  33 & 26.29 & 0.02838 \\ 
  GO:0009152 & purine ribonucleotide biosynthetic process &   5 &   3 & 0.79 & 0.02925 \\ 
  GO:0009260 & ribonucleotide biosynthetic process &   5 &   3 & 0.79 & 0.02925 \\ 
  GO:0002164 & larval development & 106 &  23 & 16.69 & 0.03108 \\ 
  GO:0042254 & ribosome biogenesis &  21 &   7 & 3.31 & 0.03144 \\ 
  GO:0000003 & reproduction & 137 &  28 & 21.56 & 0.03399 \\ 
  GO:0022613 & ribonucleoprotein complex biogenesis &  26 &   8 & 4.09 & 0.03482 \\ 
  GO:0065007 & biological regulation & 217 &  40 & 34.16 & 0.03874 \\ 
  GO:0007010 & cytoskeleton organization &  57 &  14 & 8.97 & 0.03908 \\ 
  GO:0045927 & positive regulation of growth &  68 &  16 & 10.70 & 0.03978 \\ 
  GO:0071843 & cellular component biogenesis at cellular level &  27 &   8 & 4.25 & 0.04344 \\ 
  GO:0048518 & positive regulation of biological process & 127 &  26 & 19.99 & 0.04357 \\ 
  GO:0034645 & cellular macromolecule biosynthetic process & 103 &  22 & 16.21 & 0.04358 \\ 
  GO:0000226 & microtubule cytoskeleton organization &  32 &   9 & 5.04 & 0.04471 \\ 
  GO:0007017 & microtubule-based process &  32 &   9 & 5.04 & 0.04471 \\ 
  GO:0006364 & rRNA processing &  18 &   6 & 2.83 & 0.04643 \\ 
  GO:0044267 & cellular protein metabolic process & 134 &  27 & 21.09 & 0.04769 \\ 
  GO:0002119 & nematode larval development & 104 &  22 & 16.37 & 0.04876 \\ 
  GO:0009059 & macromolecule biosynthetic process & 104 &  22 & 16.37 & 0.04876 \\ 
  GO:0030529 & ribonucleoprotein complex &  62 &  20 & 9.84 & 0.00022 \\ 
  GO:0043228 & non-membrane-bounded organelle & 115 &  28 & 18.25 & 0.00178 \\ 
  GO:0043232 & intracellular non-membrane-bounded organelle & 115 &  28 & 18.25 & 0.00178 \\ 
  GO:0044444 & cytoplasmic part & 258 &  48 & 40.95 & 0.00181 \\ 
  GO:0043227 & membrane-bounded organelle & 251 &  47 & 39.84 & 0.00274 \\ 
  GO:0043231 & intracellular membrane-bounded organelle & 251 &  47 & 39.84 & 0.00274 \\ 
  GO:0005829 & cytosol & 149 &  33 & 23.65 & 0.00306 \\ 
  GO:0031981 & nuclear lumen &  66 &  18 & 10.48 & 0.00538 \\ 
  GO:0005618 & cell wall &  17 &   7 & 2.70 & 0.00922 \\ 
  GO:0070013 & intracellular organelle lumen &  92 &  22 & 14.60 & 0.01115 \\ 
  GO:0043226 & organelle & 270 &  48 & 42.86 & 0.01309 \\ 
  GO:0043229 & intracellular organelle & 270 &  48 & 42.86 & 0.01309 \\ 
  GO:0030312 & external encapsulating structure &  18 &   7 & 2.86 & 0.01324 \\ 
  GO:0044446 & intracellular organelle part & 193 &  38 & 30.63 & 0.01332 \\ 
  GO:0009536 & plastid &  27 &   9 & 4.29 & 0.01507 \\ 
  GO:0044422 & organelle part & 195 &  38 & 30.95 & 0.01703 \\ 
  GO:0043233 & organelle lumen &  95 &  22 & 15.08 & 0.01721 \\ 
  GO:0022627 & cytosolic small ribosomal subunit &  15 &   6 & 2.38 & 0.01909 \\ 
  GO:0031974 & membrane-enclosed lumen &  97 &  22 & 15.40 & 0.02257 \\ 
   \hline
   \multicolumn{6}{l}{Cellular compartment}  \\ 
GO:0045169 & fusome &   2 &   2 & 0.32 & 0.02477 \\ 
  GO:0070732 & spindle envelope &   2 &   2 & 0.32 & 0.02477 \\ 
  GO:0015935 & small ribosomal subunit &  16 &   6 & 2.54 & 0.02684 \\ 
  GO:0005737 & cytoplasm & 275 &  48 & 43.65 & 0.02798 \\ 
  GO:0009507 & chloroplast &  25 &   8 & 3.97 & 0.02868 \\ 
  GO:0005791 & rough endoplasmic reticulum &   5 &   3 & 0.79 & 0.02991 \\ 
  GO:0005811 & lipid particle &  30 &   9 & 4.76 & 0.03102 \\ 
  GO:0005773 & vacuole &  46 &  12 & 7.30 & 0.03833 \\ 
\hline\\
\caption[Over-representation of GO-terms in positively
selected]{\textbf{Over-representation of GO-terms in differentially
    expressed between male and female worms} - Significance level
  (p.value) for over-representation are given along with the number of
  differentially expressed contigs (Significant) and the number of
  contigs with this annotation analysed (Annotated) and the
  description of the GO-term (Term). For a graph of induced GO-terms
  see also additional figures \ref{tGO_SEX_EXP_BP_classic_10_all},
  \ref{tGO_SEX_EXP_CC_classic_10_all} and
  \ref{tGO_SEX_EXP_MF_classic_10_all}.}\\
\label{over-de-mf-454}
\end{longtable}


Overrepresentation of of GO-terms differentially expressed between
libraries from worms of European and Asian origin highlighted catalytic
activity especially related to energy metabolism
(table \ref{over-de-ae-454} and additional figures
\ref{tGO_EEL_EXP_BP_classic_10_all},
\ref{tGO_EEL_EXP_CC_classic_10_all} and
\ref{tGO_EEL_EXP_MF_classic_10_all}).  Acyltransferase contigs were
all upregulated in the European libraries. However, the expression
patterns for other contigs connected to metabolism did not show
concerted up or down-regulation (e.g. for ``steroid biosynthetic
process'' 2 contigs were downregulated in the European library, 3
contigs upregulated).


\begin{longtable}{lp{4.5cm}rrrl}
  \hline
  GO.ID & Term & Annotated & Significant & Expected & p-value \\ 
  \endfirsthead
  \multicolumn{6}{c}%
  {{\bfseries \tablename\ \thetable{} -- continued from previous page}} \\
  \hline
  GO.ID & Term & Annotated & Significant & Expected & p-value \\ 
  \hline 
  \endhead
  \hline
  \multicolumn{6}{|r|}{{Continued on next page}} \\ 
  \hline
  \endfoot
  \endlastfoot
  \hline
  \multicolumn{6}{l}{Molecular function} \\ 
GO:0016408 & C-acyltransferase activity &   3 &   3 & 0.37 & 0.0018 \\ 
  GO:0016747 & transferase activity, transferring acyl groups other than amino-acyl groups &   4 &   3 & 0.50 & 0.0065 \\ 
  GO:0003824 & catalytic activity & 158 &  27 & 19.62 & 0.0088 \\ 
  GO:0016746 & transferase activity, transferring acyl groups &   8 &   4 & 0.99 & 0.0099 \\ 
  GO:0001871 & pattern binding &   2 &   2 & 0.25 & 0.0151 \\ 
  GO:0003682 & chromatin binding &   2 &   2 & 0.25 & 0.0151 \\ 
  GO:0003985 & acetyl-CoA C-acetyltransferase activity &   2 &   2 & 0.25 & 0.0151 \\ 
  GO:0008061 & chitin binding &   2 &   2 & 0.25 & 0.0151 \\ 
  GO:0030247 & polysaccharide binding &   2 &   2 & 0.25 & 0.0151 \\ 
  GO:0003713 & transcription coactivator activity &   6 &   3 & 0.75 & 0.0273 \\ 
  GO:0005543 & phospholipid binding &   6 &   3 & 0.75 & 0.0273 \\ 
  GO:0004090 & carbonyl reductase (NADPH) activity &   3 &   2 & 0.37 & 0.0417 \\ 
  GO:0008289 & lipid binding &  12 &   4 & 1.49 & 0.0483 \\ 
  GO:0016853 & isomerase activity &  12 &   4 & 1.49 & 0.0483 \\ 
   \hline
   \multicolumn{6}{l}{Biological process}  \\ 
GO:0016126 & sterol biosynthetic process &   5 &   4 & 0.60 & 0.00083 \\ 
  GO:0048732 & gland development &   9 &   5 & 1.08 & 0.00173 \\ 
  GO:0016125 & sterol metabolic process &   6 &   4 & 0.72 & 0.00228 \\ 
  GO:0006694 & steroid biosynthetic process &  10 &   5 & 1.20 & 0.00316 \\ 
  GO:0006338 & chromatin remodeling &   4 &   3 & 0.48 & 0.00596 \\ 
  GO:0006695 & cholesterol biosynthetic process &   4 &   3 & 0.48 & 0.00596 \\ 
  GO:0044281 & small molecule metabolic process & 188 &  30 & 22.63 & 0.00748 \\ 
  GO:0008202 & steroid metabolic process &  12 &   5 & 1.44 & 0.00825 \\ 
  GO:0042180 & cellular ketone metabolic process &  57 &  13 & 6.86 & 0.00845 \\ 
  GO:0023051 & regulation of signaling &  28 &   8 & 3.37 & 0.01087 \\ 
  GO:0019219 & regulation of nucleobase-containing compound metabolic process &  41 &  10 & 4.94 & 0.01412 \\ 
  GO:0001655 & urogenital system development &   2 &   2 & 0.24 & 0.01416 \\ 
  GO:0001822 & kidney development &   2 &   2 & 0.24 & 0.01416 \\ 
  GO:0006611 & protein export from nucleus &   2 &   2 & 0.24 & 0.01416 \\ 
  GO:0007528 & neuromuscular junction development &   2 &   2 & 0.24 & 0.01416 \\ 
  GO:0009953 & dorsal/ventral pattern formation &   2 &   2 & 0.24 & 0.01416 \\ 
  GO:0048581 & negative regulation of post-embryonic development &   2 &   2 & 0.24 & 0.01416 \\ 
  GO:0048741 & skeletal muscle fiber development &   2 &   2 & 0.24 & 0.01416 \\ 
  GO:0051124 & synaptic growth at neuromuscular junction &   2 &   2 & 0.24 & 0.01416 \\ 
  GO:0070050 & neuron homeostasis &   2 &   2 & 0.24 & 0.01416 \\ 
  GO:0072001 & renal system development &   2 &   2 & 0.24 & 0.01416 \\ 
  GO:0006082 & organic acid metabolic process &  54 &  12 & 6.50 & 0.01489 \\ 
  GO:0019752 & carboxylic acid metabolic process &  54 &  12 & 6.50 & 0.01489 \\ 
  GO:0043436 & oxoacid metabolic process &  54 &  12 & 6.50 & 0.01489 \\ 
  GO:0008152 & metabolic process & 266 &  37 & 32.02 & 0.01526 \\ 
  GO:0006355 & regulation of transcription, DNA-dependent &  30 &   8 & 3.61 & 0.01697 \\ 
  GO:0019953 & sexual reproduction &  44 &  10 & 5.30 & 0.02361 \\ 
  GO:0048747 & muscle fiber development &   6 &   3 & 0.72 & 0.02503 \\ 
  GO:0051171 & regulation of nitrogen compound metabolic process &  51 &  11 & 6.14 & 0.02556 \\ 
  GO:0009966 & regulation of signal transduction &  21 &   6 & 2.53 & 0.02842 \\ 
  GO:0032787 & monocarboxylic acid metabolic process &  21 &   6 & 2.53 & 0.02842 \\ 
  GO:0051252 & regulation of RNA metabolic process &  33 &   8 & 3.97 & 0.03036 \\ 
  GO:0048545 & response to steroid hormone stimulus &  16 &   5 & 1.93 & 0.03141 \\ 
  GO:0065008 & regulation of biological quality &  81 &  15 & 9.75 & 0.03399 \\ 
  GO:0050794 & regulation of cellular process & 151 &  24 & 18.18 & 0.03420 \\ 
  GO:0010033 & response to organic substance &  60 &  12 & 7.22 & 0.03487 \\ 
  GO:0048609 & multicellular organismal reproductive process &  60 &  12 & 7.22 & 0.03487 \\ 
  GO:0002026 & regulation of the force of heart contraction &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0007416 & synapse assembly &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0007431 & salivary gland development &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0007435 & salivary gland morphogenesis &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0007559 & histolysis &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0007595 & lactation &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0016271 & tissue death &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0022612 & gland morphogenesis &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0030518 & steroid hormone receptor signaling pathway &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0030522 & intracellular receptor mediated signaling pathway &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0030879 & mammary gland development &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0034612 & response to tumor necrosis factor &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0035070 & salivary gland histolysis &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0035071 & salivary gland cell autophagic cell death &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0035220 & wing disc development &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0035272 & exocrine system development &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0043628 & ncRNA 3'-end processing &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0045540 & regulation of cholesterol biosynthetic process &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0050808 & synapse organization &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0051091 & positive regulation of sequence-specific DNA binding transcription factor activity &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0051262 & protein tetramerization &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0051289 & protein homotetramerization &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0090181 & regulation of cholesterol metabolic process &   3 &   2 & 0.36 & 0.03923 \\ 
  GO:0032504 & multicellular organism reproduction &  61 &  12 & 7.34 & 0.03954 \\ 
  GO:0002165 & instar larval or pupal development &   7 &   3 & 0.84 & 0.04016 \\ 
  GO:0003015 & heart process &   7 &   3 & 0.84 & 0.04016 \\ 
  GO:0007589 & body fluid secretion &   7 &   3 & 0.84 & 0.04016 \\ 
  GO:0048872 & homeostasis of number of cells &   7 &   3 & 0.84 & 0.04016 \\ 
  GO:0060047 & heart contraction &   7 &   3 & 0.84 & 0.04016 \\ 
  GO:0006351 & transcription, DNA-dependent &  41 &   9 & 4.94 & 0.04017 \\ 
  GO:0009308 & amine metabolic process &  41 &   9 & 4.94 & 0.04017 \\ 
  GO:0006066 & alcohol metabolic process &  35 &   8 & 4.21 & 0.04262 \\ 
  GO:0006357 & regulation of transcription from RNA polymerase II promoter &  12 &   4 & 1.44 & 0.04362 \\ 
  GO:0009968 & negative regulation of signal transduction &  12 &   4 & 1.44 & 0.04362 \\ 
  GO:0010648 & negative regulation of cell communication &  12 &   4 & 1.44 & 0.04362 \\ 
  GO:0023057 & negative regulation of signaling &  12 &   4 & 1.44 & 0.04362 \\ 
  GO:0007165 & signal transduction &  69 &  13 & 8.31 & 0.04443 \\ 
  GO:0007276 & gamete generation &  42 &   9 & 5.06 & 0.04652 \\ 
  GO:0009888 & tissue development &  42 &   9 & 5.06 & 0.04652 \\ 
  GO:0044237 & cellular metabolic process & 255 &  35 & 30.69 & 0.04950 \\ 
   \hline
   \multicolumn{6}{l}{Cellular compartment}  \\ 
GO:0031967 & organelle envelope &  47 &  12 & 5.52 & 0.0033 \\ 
  GO:0031975 & envelope &  48 &  12 & 5.64 & 0.0040 \\ 
  GO:0005740 & mitochondrial envelope &  29 &   8 & 3.41 & 0.0116 \\ 
  GO:0005643 & nuclear pore &   2 &   2 & 0.23 & 0.0135 \\ 
  GO:0046930 & pore complex &   2 &   2 & 0.23 & 0.0135 \\ 
  GO:0005739 & mitochondrion &  93 &  17 & 10.92 & 0.0184 \\ 
  GO:0031966 & mitochondrial membrane &  28 &   7 & 3.29 & 0.0322 \\ 
  GO:0005902 & microvillus &   3 &   2 & 0.35 & 0.0374 \\ 
  GO:0044429 & mitochondrial part &  36 &   8 & 4.23 & 0.0432 \\ 
   \hline\\
   \caption[Over-representation of GO-terms in differentially
   expressed between worms from Asia and Europe
   ]{\textbf{Over-representation of GO-terms in differentially
       expressed between worms from Asia and Europe} - Significance
     level (p.value) for over-representation are given along with the
     number of differentially expressed contigs (Significant) and the
     number of contigs with this annotation analysed (Annotated) and
     the description of the GO-term (Term). For a graph of incuced
     GO-terms see also additional figures
     \ref{tGO_EEL_EXP_BP_classic_10_all},
     \ref{tGO_EEL_EXP_CC_classic_10_all} and
     \ref{tGO_EEL_EXP_MF_classic_10_all}.}\\
   \label{over-de-ae-454}
\end{longtable}

Enrichment of signal-positives was not found in any category of
overexpressed genes. Differntially expressed genes also showed no
pattern of enrichement in conservation categories and no enrichment of
\textit{C. elegans} orthologs with lethal/non-lethal RNAi-phenotypes.

Significantly elevated dn/ds was found for contigs differentially
expressed according to worm-origin (Fisher's exact test p=0.005; also
both up- or downregulated were significant). Contigs overexpressed in
the female libraries showed elevated levels of dn/ds (Fisher's exact
test p=0.035). In contrast male overexpressed genes showed decreased
levels of dn/ds (Fisher's exact test p=0.015). Within these groups
there was no correlation between dn/ds and log-fold-change values for
gene-expression.



%%% Local Variables: ***
%%% mode:latex ***
%%% TeX-master: "../thesis.tex"  ***
%%% tex-main-file: "../thesis.tex" ***
%%% End: ***
