% this file is called up by thesis.tex
% content in this file will be fed into the main document

%: ----------------------- name of chapter  -------------------------
\chapter{Pyrosequencing of the \textit{A. crassus}
  transcriptome} % top level followed by section, subsection
\label{cha:pyro}


%: ----------------------- paths to graphics ------------------------

% change according to folder and file names
\ifpdf
    \graphicspath{{5_pyro/figures/PNG/}{5_pyro/figures/PDF/}{5_pyro/figures/}}
\else
    \graphicspath{{5_pyro/figures/EPS/}{5_pyro/figures/}}
\fi

%: ----------------------- contents from here ------------------------

\section{Overview}
\label{sec:454-overv}

In this chater the transcriptome assembly of \textit{A. crassus} is
analysed in its biological context. It constitues a basis for
molecular research on this important species and furthermore provides
unique insights into the evolution of parasitism in the Spirurina.

After extensive screening of 756.363 raw pyrosequencing reads, I
assembled 353.055 into 11.371 contigs spanning 7.971.550 bases and
additionally obtained 21.147 singleton and lower quality contigs
spanning 8.095.986 bases. I obtained annotations for ca. 60\% of the
contigs and 40\% of the tentatively unique genes (TUGs) confirming the
high quality of especially the contigs. I found an overabundance of
predicted signal peptide cleavage sites in sequence conserved in
Nematoda and novel in \textit{A. crassus}. I identified 5112 high
quality Single nucleotide polymorphisms (SNPs) and suggest 199 of them
as most suitable markers for population-genetic studies. The GO-term
proteinase was identified as overrepresented in contigs under positive
selection. Comparing male and female as well as Asian and European
\textit{A. crassus} I developed a method for future work with this
transcriptome as a reference in mapping experiments.

\section{Sampling \textit{A. crassus}}

One female worm and one male worm were sampled from an aquaculture
with height infection loads in Taiwan. An additional female worm was
sampled from a stream with low infection pressure adjacent to the
aquaculture. All these worms were parasitising endemic
\textit{An. japonica}. A female worm and pool of L2 larval stages were
sampled from \textit{An. anguilla} in the river Rhine, one female worm
from a lake in Poland. All adult worms were filled with large amounts
of host-blood, therefore I anticipated abundant host-contamination in
sequencing data and decided to sequence a liver sample of an
uninfected \textit{An. japonica} for screening.

\section{Sequencing, trimming and pre-assembly screening}

A total of 756363 raw sequencing reads were generated for
\textit{A. crassus} (see table \ref{screening-lib}). These were
trimmed for base call quality, and filtered by length to give 585949
high-quality reads (spanning 169863104 bases). In the eel dataset from
159370 raw reads 135072 were assembled after basic quality screening.

I then screened the \textit{A. crassus} reads for contamination by
host (30071 matched previously sequenced eel genes or our own
\textit{An. japonica} 454 transcriptome, which had been assembled into
10639 mRNA contigs. (181783 reads matched large or small subunit
nuclear or mitochondrial ribosomal RNA sequences of
\textit{A. crassus}) . In addition to fish mRNAs, I identified (and
removed) 5286 reads in the library derived from the L2 nematodes that
had significant similarity to cercozoan (likely parasite) ribosomal
RNA genes (see table \ref{screening-lib}). 

% latex table generated in R 2.13.0 by xtable 1.5-6 package
% Wed Nov 23 10:32:00 2011
\begin{table}[!h]
\begin{tabular}{rllllll}
   \hline
library & E1 & E2 & L2 & M & T1 & T2 \\ 
   \hline
life.st & adult f & adult f & L2 lavae & adult m & adult f & adult f \\ 
  source.p & Europe R & Europe P & Europe R & Asia C & Asia C & Asia W \\ 
  raw.reads & 209325 & 111746 & 112718 & 106726 & 99482 & 116366 \\ 
  lowqal & 92744 & 10903 & 15653 & 15484 & 7947 & 27683 \\ 
  AcrRNA & 76403 & 11213 & 30654 & 31351 & 24929 & 7233 \\ 
  eelmRNA & 4835 & 3613 & 1220 & 1187 & 7475 & 11741 \\ 
  eelrRNA & 13112 & 69 & 1603 & 418 & 514 & 38 \\ 
  Cercozoa & 0 & 0 & 5286 & 0 & 0 & 0 \\ 
  valid & 22231 & 85948 & 58302 & 58286 & 58617 & 69671 \\ 
  valid.span & 7167338 & 24046225 & 16661548 & 17424408 & 14443123 & 20749177 \\ 
  mapping.unique & 12023 & 65398 & 39690 & 36782 & 42529 & 55966 \\ 
  mapping.Ac &  8359 & 61070 & 12917 & 31656 & 37158 & 50018 \\ 
  mapping.MN &  5883 & 48006 &  8475 & 18986 & 28823 & 41545 \\ 
  \hline
\end{tabular}
  \caption[Pyro-sequencing library statistics]{\textbf{Statistics for different 
      libraries} 
    For two sequencing libraries from European eels (E1 and E2)
    one form L2-larvae (L2), one from male (M) and two from Eels in Taiwan
    (T1 and T2) the following statistics are given. life.st = lifecycle
    stage: f for female m for male. source.p = source population: R for
    Rhine, P for Poland, C for cultured, W for wild. raw.reads = raw
    number of sequencing reads obtained. lowqal = number of reads
    discarded due to low quality or length in \textit{Seqclean}
    \cite{tgicl_pertea}. AcrRNA = number of reads hitting
    \textit{A. crassus}-rRNA (screened). eelmRNA = number of reads hitting
    eel transcriptome-sequences (screened). eelrRNA = number of reads
    hitting eel-rRNA genes (screened). Cercozoa = number of reads hitting
    cercozoan rRNA (screened). valid = number of reads valid after
    screening (assembled). valid.span = number of bases valid (assembled).
    mapping.unique = number of reads mapping uniquely to the
    assembly. mapping.Ac = number of reads mapping to the part of the
    assembly considered \textit{A. crassus} origin (see post-assembly
    screening). mapping.MN = number of reads mapping to the highCA-derived
    part of the assembly (and also \textit{A. crassus} origin).}
  \label{screening-lib}
\end{table}

\afterpage{\clearpage}

\section{Assembly}


I assembled the remaining 353055 reads (spanning 100491819 bases)
using the combined assembler strategy \cite{pmid20950480} and Roche
454 GSassembler (version 2.6) and MIRA (version 3.21)
\cite{miraEST}. From this I derived 13851 contigs that were supported
by both assembly algorithms, 3745 contigs only supported by one of the
assembly algorithms and 22591 singletons that were not assembled by
either approach (see table \ref{ass-stat}). When scored by matches to
known genes, the contigs supported by both assemblers are of the
highest credibility, and this set is thus termed the high credibility
assembly (highCA). Those with evidence from only one assembler and the
singletons are of lower credibility (lowCA). These datasets are the
most parsimonious (having the smallest size) for their quality
(covering the largest amount of sequence in reference
transcriptomes). In the highCA parsimony and low redundancy is
prioritized, while in the complete assembly (highCA plus lowCA)
completeness is prioritized. The 40187 sequences (contig consensuses
and singletons) in the complete assembly are referred to below as
tentatively unique genes (TUGs).

\begin{table}[!h]
\begin{tabular}{rrrr}
  \hline
 & lowCA & highCA & combined \\ 
  \hline
total.contigs & 26336 & 13851 & 40187 \\ 
  rRNA.contigs & 835 & 60 & 895 \\ 
  fish.contigs & 2419 & 1022 & 3441 \\ 
  xeno.contigs & 1935 & 1398 & 3333 \\ 
  remaining.contigs & 21147 & 11371 & 32518 \\ 
  remaining.span & 8095986 & 7971550 & 16067536 \\ 
  non.u.cov & 14.665 & 10.979 & 12.840 \\ 
  cov & 2.443 & 6.838 & 4.624 \\ 
  p4e.BLAST-similarity & 4356 & 5663 & 10019 \\ 
  p4e.ESTScan & 8324 & 3597 & 11921 \\ 
  p4e.LongestORF & 8347 & 2085 & 10432 \\ 
  p4e.no-prediction & 93 & 14 & 107 \\ 
  full.3p & 5906 & 2714 & 8620 \\ 
  full.5p & 1484 & 1270 & 2754 \\ 
  full.l & 104 & 185 & 289 \\ 
  GO & 2635 & 3874 & 6509 \\ 
  EC & 966 & 1492 & 2458 \\ 
  KEGG & 1608 & 2236 & 3844 \\ 
  IPR & 0 & 7557 & 7557 \\ 
  nem.blast & 4868 & 5820 & 10688 \\ 
  any.blast & 5106 & 6007 & 11113 \\ 
   \hline
\end{tabular}
\caption[Assembly classification and contig
statistics]{\textbf{Assembly classification and contig statistics -}
  Summary statistics for contigs from different assembly-categories
  given in columns as highCA = high credibility assembly; lowCA = low
  credibility assembly, combined = complete assembly. Rows indicate
  summary statistics: total.contigs = numbers of total contigs,
  fish.contigs = number of contigs hitting eel-mRNA or Chordata in
  NCBI-nr or NCBI-nt (screened out), xeno.contigs = number of contigs
  with best hit (NCBI-nr and NCBI-nt) to non-eukaryote (screened out),
  remaining.contigs = number of contigs remaining after this
  screening, remaining.span = total length of remaining contigs,
  non.u.cov = non-unique mean base coverage of contigs, cov = unique
  mean base coverage of contigs, p4e.``X'' = number protein
  predictions derived in p4e, where ``X'' describes the method of
  prediction (see Methods), full.3p = number of contigs complete at
  3', full.5p = number of contigs complete at 5', GO = number of
  contigs with GO-annotation, KEGG = number of contigs with
  KEGG-annotation, EC = number of contigs with EC-annotation,
  nem.blast = number of contigs with \texttt{BLAST}-hit to nematode in
  nr, any.blast = number of contigs with \texttt{BLAST}-hit to
  non-nematode (eukaryote non chordate) sequence in NCBI-nr.}
\label{ass-stat}
\end{table}

\afterpage{\clearpage}

I screened the complete assembly for residual host contamination, and
identified 3441 TUGs that had higher, significant similarity to
eel (and chordate) sequences (our 454 ESTs and EMBLBank Chordata
proteins) than to nematode sequences \cite{pmid21550347}.

Given our prior identification of cercozoan ribosomal RNAs, I also
screened the complete assembly for contamination with other
transcriptomes.

1153 TUGs were found mapping to Eukaryota outside of the kingdoms
Metazoa, Fungi and Viridiplantae. These hits included a wide range of
Protists ranging from Apicomplexa (mainly Sarcocystidae, 28 hits and
Cryptosporidiidae 10 hits) over Bacillariophyta (diatoms, mainly
Phaeodactylaceae, 41 hits) and Phaeophyceae (brown algae, mainly
Ectocarpaceae, 180 hits) and Stramenopiles (Albuginaceae, 63 hits) to
Kinetoplasitda (Trypanosomatidae, 26 hits) and Heterolobosea
(Vahlkampfiidae, 38 hits).

Additionally I found 298 TUGs with hits to fungi (e.g
Ajellomycetaceae, 53 hits) and 585 TUGs with hits to plants.

Hits outside the Eukaryota were mainly to Bacteria (825 hits) and
within those mostly to members of the Proteobacteria (484 hits). No
hits were found to Wolbachia or related Bacteria known as symbionts of
nematodes and arthropods. 9 TUGs were hitting sequence from Viruses
and 8 from Archaea.

I excluded all TUGs with best hits outside Metazoa and our assembly
thus has 32518 TUGs, spanning 154052 bases (of which 11371 are
highCA-derived, and span 154052 bases) that are likely to derive from
of \textit{A. crassus}.

\section{Protein prediction}

For 32411 TUGs a protein was predicted using prot4EST
\cite{wasmuth_prot4est:_2004} (see table \ref{ass-stat}). The full
open reading frame was obtained in 353 TUGs, while while for 2683 the
5' end and for 8283 the 3' end was complete. In 13379 TUGs the
corrected sequence with the imputed ORF was slightly changed compared
to the raw sequence.

\section{Annotation}
\label{454-annot}

I obtained basic annotations with orthologous sequences from
\textit{C. elegans} for 9554 TUGs, from \textit{B. malayi} for 9662
TUGs, from nempep \cite{parkinson_nembase:resource_2004, pmid21550347}
for 11617 TUGs and with uniprot proteins for 11113 TUGs.

I used annot8r \cite{schmid_annot8r:_2008} to assign gene ontology
(GO) terms for 6509 TUGs, Enzyme Commission (EC) numbers for 2458 TUGs
and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway annotations
for 3844 TUGs (see table \ref{ass-stat}). Additionally 5125 highCA
derived contigs were annotated with GO terms through
\texttt{InterProScan} \cite{pmid11590104}. Nearly one third (6987) of
the \textit{A. crassus} TUGs were annotated with at least one
identifier, and 1829 had GO, EC and KEGG annotations (see figure
\ref{ann_venn_man}).

\figuremacroW{ann_venn_man}{Annotation using different identifiers} {Number
  of annotations obtained for Gene Ontology (GO), Enzyme Commission
  (EC) and Kyoto Encyclopedia of Genes and Genomes (KEGG) terms
  through \texttt{Annot8r} \cite{schmid_annot8r:_2008} for all TUGs
  (a) and for higCA derived contigs (b). The latter includes
  additional domain-based annotations obtained with
  \texttt{InterProScan} \cite{pmid11590104}.}{0.65}

I compared our \textit{A. crassus} GO annotations for high-level
GO-slim terms to the annotations (obtained the same way) for the
complete proteome of the filarial nematode \textit{B. malayi} and the
complete proteome of \textit{C. elegans} (see figure \ref{slim_com}).

Correlation shows the occurrence of terms for the partial
transcriptome of \textit{A. crassus} to be more similar to the
proteome of \textit{B. malayi} (0.95; Spearman correlation
coefficient) than to the proteome of \textit{C. elegans} (0.9). Also
the tow model-nematode compared to each other (0.91) are less similar
in the occurrence of terms than the two parasites.

\figuremacro{slim_com}{Cross taxa comparison of annotation}{For Gene
  Ontology (GO) categories molecular function, cellular compartment
  and biological process the number of terms in high level GO-slim
  categories is given as obtained through \texttt{Annot8r}
  \cite{schmid_annot8r:_2008}.}

I inferred presence of signal peptide cleavage sites in the predicted
protein sequence using \texttt{SignalP} \cite{pmid21959131}. I
predicted 920 signal peptide cleavage sites and 65 signal peptides
with a transmembrane signature. Again these predictions are more
similar to predictions using the same methods for the proteome
\textit{B. malayi} (742 signal peptide cleavage sites and 41 with
transmembrane anchor) than for the proteome of \textit{C. elegans}
(4273 signal peptide cleavage sites and 154 with transmembrane
anchor).

I inferred the presence of a lethal rnai phenotype in the orthologous
annotation of \textit{C. elegans}. For 257 TUGs a non-lethal phenotype
was inferred for 6029 TUGs a lethal phenotype.

\section{Evolutionary conservation}

\textit{A. crassus} TUGs were classified as conserved, conserved in
Metazoa, conserved in Nematoda, conserved in Spirurina or novel to
\textit{A. crassus} by comparing them to public databases and using
two \texttt{BLAST} bit-score cutoffs to define relatedness (see table
\ref{evol-con}).

\begin{table}[!h]
\begin{tabular}{rrrrrr}
  \hline
 & conserved & novel.in.m & novel.in.n & novel.in.cl3 & novel.in.Ac \\ 
  \hline
bit.50.all & 5604 & 1713 & 2173 & 1485 & 21543 \\ 
  bit.80.all & 3506 & 1382 & 2014 & 1525 & 24091 \\ 
  bit.50.highCA & 3479 & 875 & 1010 & 601 & 5406 \\ 
  bit.80.highCA & 2457 & 832 & 1084 & 716 & 6282 \\ 
   \hline
\end{tabular}
\caption[Evolutionary conservation and novelty]{\textbf{Evolutionary
    conservation and novelty -} The kingdom Metazoa (novel.in.m),
  the phylum Nematoda (novel.in.n) and clade III (Spirurina;
  novel.in.cl3) were assessed for occurrences of
  \texttt{BLAST}-hits at two different bitscore thresholds (50 =
  bit.50 and 80 = bit.80). TUGs without any hit at a given threshold 
  were categorized as novel in \textit{A. crssus} (novel.in.Ac).
  Both novelty and conservation can be
  derived from this (numbers for conservation would be the cumulative
  sum of lower-level novelty).}
\label{evol-con}
\end{table}

\afterpage{\clearpage}

Roughly a third and a quarter of the higCA derived contigs were
categorized as conserved across kingdoms at a bitscore threshold of 50
and 80, respectively. Roughly half or 3/5 of the these contigs were
identified as novel in \textit{A. crassus}.

The remaining higCA contigs spread across intermediate
relatedness-levels. More sequences were categorised as novel at the
phylum level (Nematoda) compared to kingdom and clade III level and
the number of contigs at intermediate relatedness-levels was roughly
consistent for the two bitscore thresholds.

The latter points about intermediate conservation levels were also
true, when all TUGs were analysed. The numbers of TUGs categorised at
these intermediate levels roughly doubled. In contrast, the proportion
of additional conserved lowCA TUGs is small compared to additional
TUGs categorised as novel in \textit{A. crassus}, mirroring the higher
amount of erroneous sequence.

Proteins predicted to be novel to Nematoda and novel in
\textit{A. crassus} were significantly enriched in signal peptide
annotation compared to conserved proteins, proteins novel in Metazoa
and novel in clade III (Fisher's exact test p$<$0.001 ;
\ref{nov_sig}).

\figuremacro{nov_sig}{Enrichment of Signal-positives for categories of
  evolutionary conservations}{Proportions of
  \texttt{SignalP}-predictions for each category of evolutionary
  conservation. Generally - across bit-score thresholds - TUGS novel
  in nematodes and in \textit{A. crassus} have the highest proportion
  of signal-positives.}

The proportion of lethal rnai phenotypes was significantly higher for
orthologs of conserved TUGs (97.23\%) than for orthologs of TUGs not
conserved (94.65\%) across kingdoms (p$<$0.001, Fisher's exact test).

\section{Identification of single nucleotide polymorphisms}

I called single nucleotide polymorphisms (SNPs) on the 1099419 bases
of the TUGs that had coverage of more then 8-fold available using
\texttt{VARScan} \cite{pmid19542151}. I excluded SNPs predicted to
have more than 2 alleles or that mapped to an undetermined (N) base in
the reference, and retained 10458 SNPs. The ratio of transitions (ti;
6890) to transversion (tv; 3568) in this set was 1.93 . Using the
prot4EST predictions and the corrected sequences, 7153 of the SNPs
were predicted to be inside an ORF, with 2310 at codon first
positions, 1819 at second positions and 3024 at third positions. As
expected ti/tv inside ORFs (2.41) was higher than outside ORFs
(1.25). The ratio of synonymous polymorphisms per synonymous site to
non-synonymous polymorphisms per non-synonymous site (dn/ds) was
0.42. I filtered these SNPs to exclude those that might be associated
with analytical bias. As Roche 454 sequences have well-known
systematic errors associated with homopolymeric nucleotide sequences
\cite{pmid21685085}, I analysed the effect of exclusion of SNPs in,
or close to, homopolymer regions. I observed changes in ti/tv and in
dn/ds when SNPs were discarded using different size thresholds for
homopolymer runs and proximity thresholds (see figure \ref{snp_hom}).

\figuremacro{snp_hom}{Homoploymer screening for SNP-calling}{When SNPs
  in or adjacent to homopolymeric regions are removed changes in ti/tv
  and dn/ds are observed: As the overall number of SNPs is reduced
  both ratios change to more plausible values. Note the reversed axis
  for dn/ds to plot these lower values to the right. For homopolymer
  length $>$ 3 a linear trend for the total number of SNPs and the two
  measurements is observed. A width of 11 for the screening window
  provides most plausible values (suggesting specificity) while still
  incorporating a high number of SNPs (sensitivity).}

Based on this I decided to exclude SNPs with a homopolymer-run as
long as or longer than 4 bases inside a window of 11 bases (5 to bases
to the right, 5 to the left) around the SNP. I also observed a
relationship between TUG dn/ds and TUG coverage, associated with the
presence of sites with low abundance minority alleles (less than 7\%
of the allele calls), suggesting that some of these may be errors.
Removing low abundance minority allele SNPs from the set removed this
effect (see figure \ref{snp_final}).  Our filtered SNP dataset
includes 5112 SNPs. I retained 4.65 SNPs per kb of contig sequence,
with 8.37 synonymous SNPs per 1000 synonymous bases and 2.4
non-synonymous SNPs per 1000 non-synonymous bases. A mean dn/ds of
0.231 was calculated for the 859 TUGs (762 highCA-derived contigs)
containing at least one synonymous SNP.

\figuremacro{snp_final}{SNP calling and SNP categories}{Overabundance
  of SNPs at (a) codon-position two and of (c) non-synonymous SNPs for
  low percentages of the minority allele. (b) Significant positive
  correlation of coverage and dn/ds before removing these SNPs at a
  threshold of 7\% ($p<$ 0.001, $R^2=$ 0.015) and (d) afterwards
  ($R^2<$0.001, $p=$0.211).}

\section{Polymorphisms associated with biological processes}

I consolidated our annotation and polymorphism analyses by examining
correlations between nonsynonymous variability and particular
classifications.

Signal peptide containing proteins have been shown to have higher
rates of evolution than cytosolic proteins in a number of nematode
species. In \textit{A. crassus}, TUGs predicted to contain signal
peptide cleavage sites in SignalP showed a trend towards higher dn/ds
values than TUGs without signal peptide cleavage sites (p = 0.074; two
sided Mann-Whitney-test).

Positive selection can be inferred from dn/ds analyses, and I defined
TUGs with a dn/ds higher than 0.5 as positively selected. I
identified over- and under-represented GO ontology terms associated
with these putatively positively selected genes (see table
\ref{go-pos}). Within the molecular function category, ``peptidase
activity'' was the most significantly overrepresented term and had 13
TUGs supporting the overrepresentation. The highlighted 13 peptidases
annotated with eleven unique orthologs in \textit{C. elegans} and
\textit{B. malayi}.  The term ``structural constituent of ribosome''
was underrepresented.

% latex table generated in R 2.13.0 by xtable 1.5-6 package
% Wed Nov 23 10:32:01 2011
\begin{longtable}{rrrp{7cm}r}
 Pvalue & Count & Size & Term & direction \\ 
  \hline
0.00 &  13 &  45 & peptidase activity & Over \\ 
  0.01 &   7 &  18 & heme-copper terminal oxidase activity & Over \\ 
  0.01 &   7 &  18 & oxidoreductase activity, acting on a heme group of donors & Over \\ 
  0.01 &   7 &  18 & oxidoreductase activity, acting on a heme group of donors, oxygen as acceptor & Over \\ 
  0.01 &   7 &  18 & cytochrome-c oxidase activity & Over \\ 
  0.01 &  49 & 283 & catalytic activity & Over \\ 
  0.02 &  13 &  52 & transmembrane transporter activity & Over \\ 
  0.02 &   9 &  31 & monovalent inorganic cation transmembrane transporter activity & Over \\ 
  0.02 &   2 &   2 & L-amino acid transmembrane transporter activity & Over \\ 
  0.03 &   9 &  33 & inorganic cation transmembrane transporter activity & Over \\ 
  0.03 &  23 & 117 & hydrolase activity & Over \\ 
  0.03 &   8 &  29 & hydrogen ion transmembrane transporter activity & Over \\ 
  0.04 &   3 &   6 & ribonucleoprotein binding & Over \\ 
  0.04 &  13 &  58 & transporter activity & Over \\ 
  0.05 &  11 &  47 & substrate-specific transmembrane transporter activity & Over \\ 
  0.05 &  16 &  77 & oxidoreductase activity & Over \\ 
  0.00 &   1 &  53 & structural constituent of ribosome & Under \\ 
  0.03 &   7 &  93 & RNA binding & Under \\ 
  0.04 &   2 &  44 & transition metal ion binding & Under \\ 
  0.05 &   0 &  20 & protein binding transcription factor activity & Under \\ 
  0.05 &   0 &  20 & transcription factor binding transcription factor activity & Under \\ 
  0.05 &   0 &  20 & transcription cofactor activity & Under \\ 
   \hline
0.00 &  13 &  37 & brain development & Over \\ 
  0.00 &  14 &  45 & central nervous system development & Over \\ 
  0.00 &   6 &  12 & response to electrical stimulus & Over \\ 
  0.00 &   3 &   3 & branched chain family amino acid metabolic process & Over \\ 
  0.00 &   3 &   3 & branched chain family amino acid catabolic process & Over \\ 
  0.00 &  11 &  36 & ATP synthesis coupled electron transport & Over \\ 
  0.00 &  11 &  36 & mitochondrial ATP synthesis coupled electron transport & Over \\ 
  0.01 &   7 &  18 & mitochondrial electron transport, cytochrome c to oxygen & Over \\ 
  0.01 &  22 & 101 & nervous system development & Over \\ 
  0.01 &  11 &  38 & oxidative phosphorylation & Over \\ 
  0.01 &   6 &  15 & response to starvation & Over \\ 
  0.01 &  12 &  45 & cellular amino acid metabolic process & Over \\ 
  0.01 &   7 &  20 & positive regulation of cell cycle process & Over \\ 
  0.01 &  14 &  58 & amine metabolic process & Over \\ 
  0.01 &   4 &   8 & positive regulation of organelle organization & Over \\ 
  0.01 &   4 &   8 & spermatid development & Over \\ 
  0.01 &   4 &   8 & spermatid differentiation & Over \\ 
  0.01 &   5 &  12 & hindbrain development & Over \\ 
  0.01 &   5 &  12 & cerebellum development & Over \\ 
  0.01 &   5 &  12 & metencephalon development & Over \\ 
  0.01 &   5 &  12 & response to methylmercury & Over \\ 
  0.01 &   5 &  12 & autophagy & Over \\ 
  0.02 &  36 & 203 & response to stress & Over \\ 
  0.02 &   2 &   2 & embryonic body morphogenesis & Over \\ 
  0.02 &   2 &   2 & xylulose metabolic process & Over \\ 
  0.02 &   2 &   2 & L-amino acid transport & Over \\ 
  0.02 &   2 &   2 & neuromuscular process controlling balance & Over \\ 
  0.02 &   2 &   2 & response to sucrose stimulus & Over \\ 
  0.02 &   2 &   2 & NADP metabolic process & Over \\ 
  0.02 &   2 &   2 & response to disaccharide stimulus & Over \\ 
  0.02 &   2 &   2 & pentose metabolic process & Over \\ 
  0.02 &  15 &  66 & behavior & Over \\ 
  0.02 &   8 &  27 & interphase & Over \\ 
  0.02 &   8 &  27 & interphase of mitotic cell cycle & Over \\ 
  0.02 &  11 &  43 & electron transport chain & Over \\ 
  0.02 &  11 &  43 & respiratory electron transport chain & Over \\ 
  0.02 &  29 & 156 & catabolic process & Over \\ 
  0.02 &   3 &   5 & positive regulation of mitosis & Over \\ 
  0.02 &   3 &   5 & positive regulation of nuclear division & Over \\ 
  0.02 &  13 &  56 & cellular amine metabolic process & Over \\ 
  0.02 &  20 &  99 & aging & Over \\ 
  0.02 &  10 &  39 & regulation of cell cycle process & Over \\ 
  0.03 &  17 &  81 & apoptosis & Over \\ 
  0.03 &  16 &  75 & regulation of molecular function & Over \\ 
  0.03 &  13 &  57 & regulation of cell cycle & Over \\ 
  0.03 &   5 &  14 & mitotic cell cycle G1/S transition DNA damage checkpoint & Over \\ 
  0.03 &   5 &  14 & sleep & Over \\ 
  0.03 &   4 &  10 & cellular amino acid catabolic process & Over \\ 
  0.03 &  10 &  41 & reproductive structure development & Over \\ 
  0.03 &   3 &   6 & microtubule organizing center organization & Over \\ 
  0.03 &   3 &   6 & RNA catabolic process & Over \\ 
  0.03 &   3 &   6 & centrosome organization & Over \\ 
  0.03 &   8 &  30 & muscle organ development & Over \\ 
  0.04 &  11 &  47 & cellular respiration & Over \\ 
  0.04 &  13 &  59 & energy derivation by oxidation of organic compounds & Over \\ 
  0.04 &   7 &  25 & regulation of catabolic process & Over \\ 
  0.04 &   5 &  15 & signal transduction in response to DNA damage & Over \\ 
  0.04 &   5 &  15 & G1/S transition of mitotic cell cycle & Over \\ 
  0.04 &   5 &  15 & regulation of G1/S transition of mitotic cell cycle & Over \\ 
  0.04 &   5 &  15 & mitotic cell cycle G1/S transition checkpoint & Over \\ 
  0.04 &   5 &  15 & G1/S transition checkpoint & Over \\ 
  0.04 &   5 &  15 & DNA damage response, signal transduction by p53 class mediator & Over \\ 
  0.04 &   5 &  15 & regulation of cellular amine metabolic process & Over \\ 
  0.04 &   6 &  20 & response to copper ion & Over \\ 
  0.04 &  24 & 131 & cellular catabolic process & Over \\ 
  0.05 &   4 &  11 & imaginal disc development & Over \\ 
  0.05 &   4 &  11 & amine catabolic process & Over \\ 
  0.05 &   4 &  11 & skeletal muscle organ development & Over \\ 
  0.05 &  11 &  49 & mRNA metabolic process & Over \\ 
  0.05 &   2 &   3 & nuclear mRNA cis splicing, via spliceosome & Over \\ 
  0.05 &   2 &   3 & germ cell migration & Over \\ 
  0.05 &   2 &   3 & positive regulation of mitotic metaphase/anaphase transition & Over \\ 
  0.05 &   2 &   3 & mitotic centrosome separation & Over \\ 
  0.05 &   2 &   3 & oligosaccharide catabolic process & Over \\ 
  0.05 &   2 &   3 & spliceosomal conformational changes to generate catalytic conformation & Over \\ 
  0.05 &   2 &   3 & amino acid transport & Over \\ 
  0.05 &   2 &   3 & negative regulation of reproductive process & Over \\ 
  0.05 &   2 &   3 & centrosome duplication & Over \\ 
  0.05 &   2 &   3 & centrosome separation & Over \\ 
  0.05 &   2 &   3 & protein tetramerization & Over \\ 
  0.05 &   2 &   3 & protein homotetramerization & Over \\ 
  0.00 &  15 & 201 & gene expression & Under \\ 
  0.00 &   1 &  57 & cellular protein complex disassembly & Under \\ 
  0.00 &   1 &  57 & macromolecular complex disassembly & Under \\ 
  0.00 &   1 &  57 & protein complex disassembly & Under \\ 
  0.00 &   1 &  57 & cellular macromolecular complex disassembly & Under \\ 
  0.00 &   1 &  55 & pancreas development & Under \\ 
  0.00 &   1 &  55 & endocrine pancreas development & Under \\ 
  0.00 &   1 &  55 & endocrine system development & Under \\ 
  0.00 &   1 &  55 & viral genome expression & Under \\ 
  0.00 &   1 &  55 & viral transcription & Under \\ 
  0.00 &   8 & 131 & transcription & Under \\ 
  0.00 &   1 &  54 & translational termination & Under \\ 
  0.00 &   4 &  89 & translation & Under \\ 
  0.00 &   2 &  66 & cellular component disassembly & Under \\ 
  0.00 &   2 &  66 & cellular component disassembly at cellular level & Under \\ 
  0.01 &  14 & 178 & cellular macromolecule biosynthetic process & Under \\ 
  0.01 &  22 & 243 & biosynthetic process & Under \\ 
  0.01 &  22 & 240 & cellular biosynthetic process & Under \\ 
  0.01 &  15 & 181 & macromolecule biosynthetic process & Under \\ 
  0.01 &   2 &  57 & viral reproductive process & Under \\ 
  0.01 &   2 &  57 & viral infectious cycle & Under \\ 
  0.02 &   0 &  26 & positive regulation of intracellular protein kinase cascade & Under \\ 
  0.03 &   1 &  38 & positive regulation of response to stimulus & Under \\ 
  0.03 &   0 &  24 & oocyte differentiation & Under \\ 
  0.03 &   0 &  23 & oocyte development & Under \\ 
  0.03 &   0 &  23 & cation transport & Under \\ 
  0.04 &   0 &  22 & positive regulation of MAPKKK cascade & Under \\ 
  0.05 &  24 & 234 & growth & Under \\ 
   \hline
0.01 &   4 &   7 & small nuclear ribonucleoprotein complex & Over \\ 
  0.01 &  31 & 164 & mitochondrion & Over \\ 
  0.02 &   2 &   2 & Cajal body & Over \\ 
  0.02 &   2 &   2 & U5 snRNP & Over \\ 
  0.02 &   2 &   2 & U4/U6 x U5 tri-snRNP complex & Over \\ 
  0.03 &  17 &  80 & mitochondrial part & Over \\ 
  0.04 &   3 &   6 & nuclear speck & Over \\ 
  0.04 &   5 &  15 & nuclear body & Over \\ 
  0.04 &  14 &  65 & mitochondrial membrane & Over \\ 
  0.05 &  14 &  66 & mitochondrial envelope & Over \\ 
  0.05 &   2 &   3 & clathrin sculpted vesicle & Over \\ 
  0.05 &   2 &   3 & plasma membrane respiratory chain complex I & Over \\ 
  0.05 &   2 &   3 & plasma membrane respiratory chain & Over \\ 
  0.05 &   2 &   3 & basement membrane & Over \\ 
  0.05 &   2 &   3 & plant-type cell wall & Over \\ 
  0.00 &   0 &  37 & large ribosomal subunit & Under \\ 
  0.01 &   0 &  35 & cytosolic large ribosomal subunit & Under \\ 
  0.01 &  28 & 280 & nucleus & Under \\ 
  0.02 &  19 & 201 & non-membrane-bounded organelle & Under \\ 
  0.02 &  19 & 201 & intracellular non-membrane-bounded organelle & Under \\ 
  0.02 &   4 &  71 & nucleolus & Under \\ 
  0.02 &   3 &  60 & cytosolic ribosome & Under \\ 
  0.02 &   1 &  38 & plastid & Under \\ 
  0.03 &   4 &  68 & cytosolic part & Under \\ 
  0.03 &   1 &  36 & chloroplast & Under \\ 
  0.05 &   5 &  73 & ribosome & Under \\ 
  \hline\\
\caption[Over- and under-representation of GO-terms in positively
selected]{\textbf{Over- and under-representation of GO-terms in
    positively selected -} GO-terms over- or under-represented
  (direction) in contigs putatively under positive
  selection. Horizontal lines separate categories of the
  GO-ontology. First category is molecular function, second biological
  process, last cellular compartment. P values (Pval) for over- or
  under-representation are given along with the number of positively
  selected contigs (Count; dn/ds $>$ 0.5) and the number of contigs
  with this annotation for which a dn/ds was obtained (Size) and the
  description of the GO-term (Term).} \label{go-pos}\\
\end{longtable}


While the biological process and cellular compartment categories
provide less information for a nematode (highlighting e.g. brain or
pancreas development), underrepresented terms in both were connected
to ribosomal proteins, validating the analysis for the molecular
function category.

Other overrepresented terms abundant over categories pointed to
subunits of the respiratory chain e.g. ``heme-copper terminal oxidase
activity'' and ``cytochrome-c oxidase activity'' in molecular function
and ``mitochondrion'' in cellular compartment.

At both bitscore thresholds contigs novel in clade III and novel in
\textit{A. crassus} had a significantly higher dn/ds than other
contigs (novel.in.metazoa - novel.in.Ac, 0.005 and 0.015;
novel.in.nematoda - novel.in.Ac, 0.005 and 0.002; novel.in.nematoda -
novel.in.clade3, 0.207 and 0.045; comparison, p-value from bitscore of
50 and p-value from bitscore of 80, Nemenyi-Damico-Wolfe-Dunn test,
given only for significant comparisons; figure \ref{dn_ds_con}).

\figuremacro{dn_ds_con}{Positive selection and evolutionary
  conservation}{Box-plots for dn/ds in TUGs according to different
  categories of evolutionary conservation. Significant comparisons are
  novel.in.metazoa - novel.in.Ac (0.005 and 0.015), novel.in.nematoda
  - novel.in.Ac (0.005 and 0.002), novel.in.nematoda - novel.in.clade3
  (0.207 and 0.045; p-value for bitscore of 50 and 80,
  Nemenyi-Damico-Wolfe-Dunn test).}

Orthologs of \textit{C. elegans} transcripts with lethal rnai
phenotype are expected to evolve under stronger selective
constraints. Indeed the values of dn/ds showed a non-significant trend
towards lower values in TUGs with orthologs with a lethal phenotype
compared to a non-lethal phenotypes (p=0.138, two-sided U-test).

\section{SNP markers for single worms}

I used \texttt{Samtools}\cite{journals/bioinformatics/LiHWFRHMAD09}
and \texttt{Vcftools}\cite{pmid21653522} to call genotypes in single
worms (adult sequencing libraries). This resulted in 199 informative
sites in 152 contigs, where two alleles were found in at least one
assured genotype at least in one of the worms.

% latex table generated in R 2.13.0 by xtable 1.5-6 package
% Wed Nov 23 10:32:01 2011
\begin{table}[ht]
\begin{center}
\begin{tabular}{rrrrr}
  \hline
 & rel.het & int.rel & ho.loci & std.het \\ 
  \hline
T2 & 0.45 & -0.73 & 0.59 & 1.00 \\ 
  T1 & 0.93 & -0.95 & 0.34 & 1.62 \\ 
  M & 0.37 & -0.73 & 0.66 & 0.84 \\ 
  E1 & 0.38 & -0.83 & 0.60 & 0.91 \\ 
  E2 & 0.18 & -0.35 & 0.82 & 0.50 \\ 
   \hline
\end{tabular}
\caption[Measurements of multi-locus heterozygosity for single
worms]{\textbf{Measurements of multi-locus heterozygosity for single
    worms -} Genotyping for a set of 199 SNPs, different measurements
  were obtained to asses genome-wide heterozygosity.  Measurements for
  relative heterozygosity (rel.het; number of homozygous sites/ number
  of heterozygous sites), internal relatedness (int.rel;
  \cite{pmid11571049}), homozygosity by loci (ho.loci;
  \cite{pmid17107491}) and standardized heterozygosity (std.het;
  \cite{coltman81j}) are given.  All these measurements are pointing
  to sample T1 (Taiwanese worm from a wild population) as the most
  heterozygous and sample E2 (the European worm from Poland) as the
  least heterozygous individual. Heterozygote-heterozygote correlation
  \cite{pmid21565077} confirmed the genome-wide significance of these
  markers.}
\label{snp-sing}
\end{center}
\end{table}

Internal relatedness \cite{pmid11571049}, homozygosity by loci
\cite{pmid17107491} and standardised heterozygosity \cite{coltman81j}
were all highlighting the Taiwanese worm from the wild population
(sample T1) as the most and the European worm from Poland (sample E2)
as the least heterozygous individual. The other worms had intermediate
values between these two extremes (see table \ref{snp-sing}).

I confirmed the genome-wide significance of these estimates using
heterozygosity-heterozygosity correlation \cite{pmid21565077}. These
tests confirmed the representativeness of the 199 SNP-markers for the
whole genome in population genetic studies ($\mu$ = 0.78,
$ci_l$=0.444; $\mu$ = 0.86 and $ci_l$ = 0.596; $\mu$ = 0.87 and
$ci_l$= 0.632; mean and lower bound of 95\% confidence intervals from
1000 bootstrap replicates for internal relatedness, homozygosity by
loci and standardised heterozygosity). Using a higher number of
genotyped individuals these markers would allow to asses the amount of
inbreeding in populations of \textit{A. crassus}.


\section{Differential expression}

Using methods developed for sequencing data in the R-package DESeq
\cite{pmid20979621}, I analysed gene-expression inferred from
mapping. Of the 353055 reads 252388 (71.49\%) mapped uniquely (with
their best hit) to the fullest assembly (including the all assembled
contigs as a ``filter'' later removing screened out sequences for
analysis).

22 Contigs were clearly over-expressed at adjusted p-values $<$ 0.001
in the male worm compared to the female worms.  7 of these were
annotated as ``Major sperm protein'', all with different orthologs in
\textit{B. malayi} and with 5 different orthologs in
\textit{C. elegans}. Other annotation were ``putative P40'' for
``Contig2545'' and ``Contig94'', ``MFP2'' for ``Contig437'', ``KH
domain containing protein'' for ``Contig96'', ``Protein kinase domain
containing protein'' for ``Contig96'', ``Protein-tyrosine phosphatase
containing protein'' for ``Contig382'' and ``PDZ domain containing
protein'' for ``Contig106''. For all these contigs read-sums for
orthologous groups in the two model nematodes also showed significant
over-expression in the same direction.

An exception was ``Contig194'' (``Phosphoenolpyruvate
carboxykinase''), for which the sums of orthologous-counts for both
model-nematodes did not show a significant over-expression in the
male.

Comparing libraries from European worms to Taiwanese worms, two
contigs were observed at adjusted p-values $<$ 0.001: ``Contig5250''
was only expressed in library E2 and was annotated as an ortholog of
alpha tubulin in both \textit{C. elegans} and \textit{Brugia
  malayi}. This annotation was shared with 18 and 16 contigs
respectively. Contrary to the identified contig, the sums of the read
counts for this orthologous groups showed a non-significant trend
towards higher expression in the the Asian
libraries. ``Contig13931''was only expressed in sample E1 and showed
high similarity to \textit{Ascaris lumbricoides} small nuclear RNAs.

Two contigs were identified as differentially expressed at adjusted
p-values below 0.2 and were expressed highly in library E2:
``Contig5320'' shared annotation as ortholog of ``Cuticle collagen 7''
in both model-nematodes with 5 contigs.  Counts for the orthologous
groups showed a non-significant opposite trend of over-expression in
the in the Taiwanese libraries.  ``Contig200'' shared similarity only
with one predicted protein from \textit{C. elegans}.

This was followed by a set of 9 contigs over-expressed at exactly 20\%
false discovery rate. They were all found only in Taiwanese libraries
and the ortholog-groups showed a tendency towards expression in the
same direction. Three of these are worth further notice:
``Contig5164'' annotated as ``galactoside binding lectin'' was not
only expressed in one of the Taiwaneselibraries, but in
both. ``Contig110'' (``cysteine protease'') had adjusted p-values $<$
1 and ``Contig6355'' (``Trypsin family protein'') nearly significant
adjusted p-values $<$ 0.4 for orthologous groups in
\textit{C. elegans} and \textit{B. malayi}.


%%% Local Variables: ***
%%% mode:latex ***
%%% TeX-master: "../thesis.tex"  ***
%%% tex-main-file: "../thesis.tex" ***
%%% End: ***

