% this file is called up by thesis.tex
% content in this file will be fed into the main document

%: ----------------------- name of chapter  -------------------------
\chapter{Transcriptomic divergence in a common garden experiment}
% top level followed by section, subsection
\label{cha:tra-diff}

%: ----------------------- paths to graphics ------------------------

% change according to folder and file names
\ifpdf
    \graphicspath{{6_rna_seq/figures/PNG/}{6_rna_seq/figures/PDF/}{6_rna_seq/figures/}}
\else
    \graphicspath{{6_rna_seq/figures/EPS/}{6_rna_seq/figures/}}
\fi


%: ----------------------- contents from here ------------------------


\section{Infection experiments}

Dissection of eels 55-57 dpi showed higher recovery of European worms
in \textit{An. anguilla} and higher recovery of Taiwanes worms in
\textit{An. japonica}, compared to the other parasite populations. In
other words, in host-parasite combinations of matching origin,
parastites performed better.

\figuremacro{stages_plot}{Recovery of worms in cross-infection
  experiment}{Mean numbers of worms recovered after 55-57 dpi for
  sample sizes given as n=x. Error-bars indicate the standard error
  (s.e.) of the mean. Stages are L3-larvae (l3), L4-larvae (l4), adult
  females (adult.f) and adult males (adult.m).}

In host-species/parastite-population pairs found in nature roughly
eight or nine adult worms could be recoverd per eel. In the
transplanted host/parasite combinations only two or three adult worms
were recovered on average (see figure \ref{stages_plot}). In
\textit{An. anguilla} no differences in the recovery of larval stages
was recorded. In \textit{An. japonica} however roughly two individals
more were recorded from both larval stages in the host/parasite
combination found in nature.

These differences are highly significant especially for adult worms
(see table \ref{tab:ad-sig}) and are interpretable as a sign of local
adaptation, as adult survival and recovery can be regarded a fitness
component.

Recovery as a proportion of the 50 larvae eels were inocculated with,
was thus roughly 30\% for the adapted pairs compared to only roughly
10\% in non-adapted host-parasite pairs.

\begin{table}[ht]
\begin{center}
\begin{tabular}{rrrrr}
  \hline
 & Estimate & Std. Error & t value & Pr($>$$|$t$|$) \\ 
  \hline
  (Intercept) & 9.5000 & 1.1109 & 8.55 & 0.0000 \\ 
  host.spec.AJ & -8.0789 & 1.7472 & -4.62 & 0.0000 \\ 
  worm.pop.T & -5.2222 & 1.3689 & -3.81 & 0.0002 \\ 
  host.spec.AJ:worm.pop.T & 11.7345 & 2.2010 & 5.33 & 0.0000 \\ 
   \hline
\end{tabular}
\caption[Linear model for recovery]{Linear model for recovery of adult
  worms. The estimate gives the mean of the distribution of adult
  worms for the factor values in the rows. The intercept is set to Aa
  R (\textit{An. anguilla} and the European population) further rows
  give variations for each factor. Std. Error is the standard error of
  this value. Additonally the probabilty of a t-value as small or
  smaller then the oberserved t-value are given. The signature of
  local adaptation is visible in the highly significant interaction
  term.}
\label{tab:ad-sig}
\end{center}
\end{table}


\section{Sample preparation and sequencing }

Three biological replicates were obtained from each of the two worm
population in each of th two eel-host for each of the two sexes of
worms. This resulted in a total 24 RNA-extractions prepared for
sequencing: 3 individual female worms from each experimental group
were choosen randomly, to give in total twelve females. Additionally
from three individual male worms, and from 9 pools of male worms RNA
was extracted (see table \ref{tab:lib-prep}). Pools consisted of worms
from one infected eel individual each. All worms or worm-pools were
derived form infections of different eel individuals, with one small
exception from this form of statistical independence: From eel AJ/R3 a
male worms as well as a female worm had to be prepared. It was
impossible to extract enough RNA from all but the biggest male worms
especially of the Japanese eel/European worm combination, leaving no
other choice. Because of the small size of male worms it was generally
not possible to randomly choose individuals. Preparation of sufficient
amounts of RNA was only achieved in pools of the biggest
individuals. All male worms were thus choosen for preparation based on
large size, even when pools of worms were used.

% latex table generated in R 2.13.0 by xtable 1.5-6 package
% Tue Nov 29 10:05:46 2011
\begin{table}[h]
\begin{center}
\begin{tabular}{llllrrr}
  \hline
label & sex & host & population & intensity & worms in prep & conc in prep \\ 
  \hline
AA/T20F & female & AA & Taiwan & 1 & 1 & 5.60 \\ 
  AA/T12F & female & AA & Taiwan & 14 & 1 & 6.80 \\ 
  AA/T45F & female & AA & Taiwan & 5 & 1 & 8.00 \\ 
  AA/T24M & male & AA & Taiwan & 6 & 3 & 4.80 \\ 
  AA/T42M & male & AA & Taiwan & 11 & 1 & 5.60 \\ 
  AA/T3M & male & AA & Taiwan & 5 & 4 & 4.88 \\ 
  AA/R18F & female & AA & Europe & 4 & 1 & 4.80 \\ 
  AA/R28F & female & AA & Europe & 10 & 1 & 5.20 \\ 
  AA/R8F & female & AA & Europe & 27 & 1 & 5.20 \\ 
  AA/R16M & male & AA & Europe & 10 & 4 & 5.20 \\ 
  AA/R11M & male & AA & Europe & 25 & 14 & 6.40 \\ 
  AA/R2M & male & AA & Europe & 10 & 4 & 6.60 \\ 
  AJ/T8F & female & AJ & Taiwan & 10 & 1 & 5.91 \\ 
  AJ/T5F & female & AJ & Taiwan & 2 & 1 & 4.80 \\ 
  AJ/T26F & female & AJ & Taiwan & 2 & 1 & 2.40 \\ 
  AJ/T25M & male & AJ & Taiwan & 24 & 5 & 4.05 \\ 
  AJ/T19M & male & AJ & Taiwan & 24 & 7 & 3.50 \\ 
  AJ/T20M & male & AJ & Taiwan & 20 & 8 & 3.80 \\ 
  AJ/R1F & female & AJ & Europe & 3 & 1 & 5.92 \\ 
  AJ/R3F & female & AJ & Europe & 3 & 1 & 6.90 \\ 
  AJ/R5F & female & AJ & Europe & 10 & 1 & 4.04 \\ 
  AJ/R1M & male & AJ & Europe & 3 & 1 & 2.50 \\ 
  AJ/R3M & male & AJ & Europe & 3 & 2 & 2.60 \\ 
  AJ/R5M & male & AJ & Europe & 10 & 1 & 2.23 \\ 
   \hline
\end{tabular}
\caption[Summary of RNA preparation]{\textbf{A summary of all 24
    samples prepared for RNA-seq -} The label of the RNA preparation
  follows a convention based on the eel species (host; first two
  letter of label, AA for \textit{An. anguilla} AJ for
  \textit{An. japonica}), worm population (population - R for
  European, T for Taiwanese; letter after the slash) and sex of
  worm(s) in preparation (F for female, M for male; last letter in
  label). Additionally the intensity of infection (number of adult
  worms found in the infected eel; intensity) and the number of worms
  pooled in the preparation (only male worms are pooled for RNA
  extraction, individual female worms were used). Finally
  RNA-concentration (conc in prep) in the preparation is given in
  $\mu$g per ml}
\label{tab:lib-prep}
\end{center}
\end{table}

Sequencing was performed in three multiplexed pools of eight libraries
each. The samples were partitioned into these pools spreading
replicates over lanes in a blocking design to further guarantee
statistical independence from sequencing-lane effects. Each pool of
eight was sequenced on two lanes, giving in total six lanes of data
and two technical replicates for each library. Sequencing resulted in
a total of 263.668.952 raw sequencing read-pairs.

\section{Examination of data-quality}

Reads were mapped against the fullest assembly (see
\ref{sec:final-full-assembly}) using \texttt{BWA}
\cite{pmid20080505}. Of the 263.668.952 raw read-pairs 173.602.387
mapped uniquely to the assembly and were counted on a per-library
base.

Read counts for one library were originally obtained from two
different technical replicate lanes of a flow-cell. The technical
replicate of read-counts demonstrated very low differences as inferred
from a clustering analysis using variance stabilized data and
euclidian distances as implemented in \texttt{DESeq}
\cite{pmid20979621} (see \ref{heat_tech_rep}).

\figuremacroW{heat_all}{Distances between RNA-seq read-count for
  different samples}{Euclidean distance (square distance between the
  two count vectors) for variance stabilized read-counts for all
  libraries including technical replicates; Red indicates low distance
  (high similarity), blue heigh distance (low similarity). a) Data
  before screening and summation of technical replicates. All
  technical replicates are clusterd very closely, the distance between
  an outlier female sample (AJ\_T26F) is high. b) Same illustration
  after summation of technical replicates and screening. Distance
  between outlier-sample and other female samples is reduced.}{0.5}

158.232.523 read-pairs were left after removal of hits to contigs for
which non-\textit{A. crassus} origin had been inferred in the
curration of the 454-transcriptome assembly.

% latex table generated in R 2.13.0 by xtable 1.5-6 package
% Tue Nov 29 14:12:43 2011
\begin{table}[h]
\begin{center}
\begin{tabular}{llrrr}
  \hline
library & raw.reads & raw.mapped & tax.mapped & screened \\ 
  \hline
AA\_R11M & 11986442 & 8628520 & 7868814 & 6889551 \\ 
  AA\_R16M & 10810349 & 6858585 & 6217540 & 5276284 \\ 
  AA\_R18F & 9227615 & 6552527 & 5933235 & 5200958 \\ 
  AA\_R28F & 10135670 & 6665381 & 6005399 & 5171806 \\ 
  AA\_R2M & 12469746 & 7628428 & 6929651 & 5906422 \\ 
  AA\_R8F & 15270570 & 11527867 & 10758535 & 9453468 \\ 
  AA\_T12F & 11299438 & 7842479 & 7195621 & 6332396 \\ 
  AA\_T20F & 11740839 & 7744179 & 7114349 & 6323422 \\ 
  AA\_T24M & 8552723 & 5254194 & 4662053 & 3969305 \\ 
  AA\_T3M & 11031751 & 6460836 & 5800042 & 4993726 \\ 
  AA\_T42M & 11573501 & 7567845 & 6787375 & 5694801 \\ 
  AA\_T45F & 10646847 & 7714472 & 7173709 & 6283585 \\ 
  AJ\_R1F & 9855005 & 6400558 & 5890748 & 5167912 \\ 
  AJ\_R1M & 10211903 & 5851063 & 5313544 & 4506254 \\ 
  AJ\_R3F & 9897937 & 6425201 & 5948079 & 5124077 \\ 
  AJ\_R3M & 8775211 & 4562324 & 4073621 & 3422526 \\ 
  AJ\_R5F & 11949105 & 8442537 & 7830247 & 6882280 \\ 
  AJ\_R5M & 11231532 & 7504494 & 6772010 & 5913016 \\ 
  AJ\_T19M & 9195576 & 4798404 & 4293123 & 3635843 \\ 
  AJ\_T20M & 10862591 & 6880937 & 6251674 & 5280529 \\ 
  AJ\_T25M & 11195315 & 7162880 & 6480185 & 5645097 \\ 
  AJ\_T26F & 11195335 & 7439917 & 6641973 & 6031374 \\ 
  AJ\_T5F & 10357569 & 7413685 & 6794507 & 6007930 \\ 
  AJ\_T8F & 14196382 & 10275074 & 9496489 & 8364594 \\ 
   \hline
\end{tabular}
\caption[Mapping Summary]{\textbf{Maping summarized for all 24
    libraries -} Rows indicate different libraries (worms or
  worm-pools as indicated in \ref{tab:lib-prep}) raw.reads gives the
  number of read-paires sequenced, raw.mapped the number of reads
  mapping uniquely with their best hit, tax.mapped the number of reads
  after substraction of reads to putatively eel-host derived contigs
  and screened after substraction of all reads mapping not to the
  higCA-derived assembly or to contigs with overall counts less than
  32.}
\label{tab:read-clean}
\end{center}
\end{table}

After another screening for spurious read-counts to low covered
transcripts and to transcripts of low relieability (lowCA in the
454-assembly; see \ref{sec:final-full-assembly}) 137.477.156
read-pairs were left for further analysis. Distribution of these
read-pairs over libraries showd roughly three fold differences, with a
mean of 5.728.215 reads and a range from 3.422.526 read-pais for
library AJ\_R3M to 9.453.468 read-pairs for library AA\_R8F (see
\ref{tab:read-clean}).

\figuremacro{mds}{Principle coordinate plot for expression inRNA-seq
  libraries}{Distance between sample-pairs is the root-mean-square
  deviation (Euclidean distance) for the most differentially expressed
  (DE) genes. Distances can be interpreted as the log2-fold-change of
  the genes with the biggest changes, i.e. the log2-fold-change for
  the genes that distinguish the samples.}

These reads mapped to 7520 contigs from our 454 assembly, making them
the basis for all further inverstigations. Analysis of between-sample
distance confirmed the library clustering. Sex of the worms defined
the overall distances between libraries, host- or
population-differences were not visible in an overall effect in the
top differentially expressed (DE) genes (see figure \ref{mds}).

\section{Orthologous-screened expression differences}

For the 7520 contigs with analysed exprssion values 4382
\textit{C. elegans}-orthologs and 4292 \textit{B. malayi}-orthologs
with read more than 32 read-counts over all libraries were determined
based on the annotation of the assembly (see \ref{454-annot}). This
resulted in 3596 contigs with measured expression also having a
measurement for a corresponding orhtolog (or group of orthologs) in
both model-species and thus being available for analysis.

For all further evaluations the congruence of the basic contig-based
statistics with orthologous-derived statistics is considered.

\section{Expression differences in general linear models}

Generalized linear models (GLMs) were used as implemented in the
R-package \texttt{edgeR}. Using these models we obtained 2588 contigs
(34\% of total) de between male and female worms at a false discovery
rate (FDR) of 5\%. 1101 (31\% of total orthologous available) of these
contigs of were confirmed by contigs in the orthologous
evaluation. 1425 (556 orthologous confirmed; OC) of these were
upregulated in male worms 1163 (545 OC) upregulated in female worms.

At the same threshold 55 contigs (0.7\% of total; 9, 0.25\% OC) showed
significant differential response to the host-species. 38 (5 OC) were
upregulated upregulated in \textit{An. japonica}, 17 (4 OC)
upregulated in \textit{An. anguilla}.

68 contigs (0.9\% of total; 15, 0.42\% OC) showed differences
according to the population of the worm. 39 (11 OC) of these were
upregulated in the Taiwanese population, 29 (4 OC) in the European
population.

An important observation in these models is the prevalence of
co-occuring significance of simple main effects. Expression changes
overlapping for two main effects mean a significant difference in
expression according to both factors. These differences are in the
same direction for a combination of the factors. Most contigs DE
according to the main effects of host-species or worm-population were
also DE according to the sex of the worm. There was also a number of
contigs differing for all three predictors in the same way. No contigs
were observed DE in both the host-species and worm-populations in the
same direction but not according to worm-sex. From the 68 contigs DE
in different \textit{A. crassus}-populations, 38 were also DE
according to worm sex and 16 according to all 3 main effects (see
figure \ref{venn_mod}).

The benefit of also allowing contrasting significant differences in
interaction terms highlights the power of the GLM-approach. In these
interactions a difference according to both focal factors in different
directions for factor combinations is indicated. For interactions
between host-species and parasite-population (eel*pop) for example
this mirrors the result of adult recovery i.e. a differential
regulation according to host-species/parasite-population combinations
found in nature.

And indeed also interaction-effects were observed: 7 contigs (0 OC)
showed differential expression according to the worm-sex*eel-species
interaction, 12 (3 OC) to worm-sex*parasite-population, 13 (2 OC) to
host-species*parastie-population, 1 (0 OC) contig showed significance
for the 3-way interaciton (see figure \ref{venn_mod}).

\figuremacroW{venn_mod_man}{Venn diagramm of contigs significant for
  different terms in \texttt{edgeR}-GLMs}{Overlap between differences
  in simple main effects are given as black numbers in the
  Venn-Diagramm. Numbers outside the circles in the lower left corner
  idicated non-significant contigs. The number of significant contigs
  for interaction effects are indicated in red for comparison. In (a)
  values for all contigs are given in (b) for orhtolog-confirmed (OC)
  contigs.}{0.6}

In summary, a low amount of overlap in main effects between
populations and host-species compared to the other main-effect
overlaps and in relation a higher amount of interaction effects
between these two conditions was observed.

\section{Confirmation of contig categories through multivariate
  clustering}

We performed constrained redundancy analysis for the effects of
eel-host and worm-population. This technique similarly to principal
components analysis can partition the variance into orthogonal
components, and additionally constrain one of the components to the
factor of interest. We found 7\% of the variance in contigs DE between
eel-hosts and 11\% of the variance in contigs DE between
worm-population were explained by the corresponding factor. In both
evaluations more than 50\% of the remaining variance could be
explained by a single principal component, which was mainly consisting
of worm-sex (see figure \ref{pca_eel} a and \ref{pca_pop} a). When
only OC-DE contigs were considered the explained variance for
difference between eel-host droped to 3.3\% and the explained variance
for differences between worm-population was raised to 23\%, while the
sex-effect explained 70\% and 50\% of the variance (see figure
\ref{pca_eel} b and \ref{pca_pop} b). Significance of the constrained
component reported ba a premutation could be established at a p = 0.05
threshold for all but the OC eel-host DE subset.

\figuremacroW{pca_eel}{Constrained redundancy analysis for host-DE
  contigs}{Eel-host differences are displayed as constrained component
  on the x-axis, the principal component on the y-axis corresponds to
  the sex of the worm. (a) Host differences patition the variance in
  samples in like expcted for all contigs, the constrained component
  showed significance. (b) For OC contigs the constrained component
  fails to to partition the variance as expected, the component showed
  no significance}{0.65}

\figuremacroW{pca_pop}{Constrained redundancy analysis for
  population-DE contigs}{Population differences are displayed as
  constrained component on the x-axis, the principal component on the
  y-axis corresponds to the sex of the worm. Host differences patition
  the variance in samples like expcted for all contigs (a) as well as
  for OC-contigs (b). The constrained component showed significance in
  both subsets.}{0.65}

\section{Biological processes associated with DE contigs}

We employed tests for overrepresentation of categories in
gene-ontology (GO). These tests respect the structure of the ontology
and also consider overrepresentation of higher level (ancestor-)
terms. Summarizing annotations at higher levels it is therefore
possible to conceive higher-order responses to the conditions
investigated.

For the differences between male and female worms the 


\begin{table}[ht]
\begin{center}
\begin{tabular}{llrrrl}
  \hline
  GO.ID & Term & Annotated & Significant & Expected & p-value \\ 
  \hline
  \textbf{Molecular function} &  &   &   &  &  \\ 
  GO:0042578 & phosphoric ester hydrolase activity &  99 &  59 & 31.99 & 1.2e-08 \\ 
  GO:0016791 & phosphatase activity &  88 &  53 & 28.44 & 4.2e-08 \\ 
  GO:0004721 & phosphoprotein phosphatase activity &  65 &  42 & 21.00 & 6.5e-08 \\ 
  GO:0004722 & protein serine/threonine phosphatase act... &  34 &  24 & 10.99 & 4.8e-06 \\ 
  GO:0005509 & calcium ion binding &  78 &  43 & 25.21 & 2.1e-05 \\ 
  GO:0046873 & metal ion transmembrane transporter acti... &  32 &  21 & 10.34 & 0.00010 \\ 
  GO:0003824 & catalytic activity & 1354 & 482 & 437.55 & 0.00015 \\ 
  GO:0016614 & oxidoreductase activity, acting on CH-OH... &  46 &  27 & 14.86 & 0.00018 \\ 
  GO:0016616 & oxidoreductase activity, acting on the C... &  42 &  25 & 13.57 & 0.00023 \\ 
  GO:0017018 & myosin phosphatase activity &  10 &   9 & 3.23 & 0.00027 \\ 
  \hline
  \textbf{Biological process} &  &   &   &  &  \\ 
  GO:0050896 & response to stimulus & 1535 & 583 & 504.78 & 1.7e-10 \\ 
  GO:0006470 & protein dephosphorylation &  63 &  41 & 20.72 & 1.2e-07 \\ 
  GO:0007391 & dorsal closure &  32 &  25 & 10.52 & 1.7e-07 \\ 
  GO:0016476 & regulation of embryonic cell shape &  13 &  13 & 4.27 & 5.0e-07 \\ 
  GO:0001700 & embryonic development via the syncytial ... &  49 &  33 & 16.11 & 6.7e-07 \\ 
  GO:0007392 & initiation of dorsal closure &  15 &  14 & 4.93 & 1.7e-06 \\ 
  GO:0046664 & dorsal closure, amnioserosa morphology c... &  15 &  14 & 4.93 & 1.7e-06 \\ 
  GO:0016311 & dephosphorylation &  86 &  49 & 28.28 & 2.6e-06 \\ 
  GO:0042221 & response to chemical stimulus & 864 & 337 & 284.12 & 3.1e-06 \\ 
  GO:0007394 & dorsal closure, elongation of leading ed... &  11 &  11 & 3.62 & 4.7e-06 \\ 
  \hline
  \textbf{Cellular compartment} &  &   &   &  &  \\ 
  GO:0031224 & intrinsic to membrane & 372 & 164 & 118.85 & 8.4e-08 \\ 
  GO:0016021 & integral to membrane & 368 & 162 & 117.58 & 1.2e-07 \\ 
  GO:0005576 & extracellular region & 250 & 115 & 79.88 & 7.7e-07 \\ 
  GO:0031226 & intrinsic to plasma membrane & 176 &  86 & 56.23 & 1.0e-06 \\ 
  GO:0005887 & integral to plasma membrane & 172 &  84 & 54.95 & 1.4e-06 \\ 
  GO:0030054 & cell junction & 145 &  72 & 46.33 & 3.9e-06 \\ 
  GO:0000267 & cell fraction & 435 & 179 & 138.98 & 6.4e-06 \\ 
  GO:0016020 & membrane & 1154 & 417 & 368.70 & 3.6e-05 \\ 
  GO:0000164 & protein phosphatase type 1 complex &  14 &  12 & 4.47 & 4.9e-05 \\ 
  GO:0072357 & PTW/PP1 phosphatase complex &  14 &  12 & 4.47 & 4.9e-05 \\ 
  \hline
\end{tabular}
\caption[GO-terms enriched in DE between male and
female]{\textbf{GO-terms enriched in DE between male and female worms
    -} The top 10 enriched GO-categories are given for genes DE
  between the different male and female worms.}
\end{center}
\end{table}


\begin{table}[ht]
\begin{center}
\begin{tabular}{llrrrl}
  \hline
  GO.ID & Term & Annotated & Significant & Expected & p-value \\ 
  \hline
  \textbf{Molecular function} &  &   &   &  &  \\ 
  GO:0004190 & aspartic-type endopeptidase activity &   7 &   2 & 0.03 & 0.00044 \\ 
  GO:0070001 & aspartic-type peptidase activity &   7 &   2 & 0.03 & 0.00044 \\ 
  GO:0030248 & cellulose binding &   1 &   1 & 0.00 & 0.00478 \\ 
  GO:0030600 & feruloyl esterase activity &   1 &   1 & 0.00 & 0.00478 \\ 
  GO:0052689 & carboxylic ester hydrolase activity &  27 &   2 & 0.13 & 0.00694 \\ 
  GO:0045505 & dynein intermediate chain binding &   2 &   1 & 0.01 & 0.00955 \\ 
  GO:0016788 & hydrolase activity, acting on ester bond... & 193 &   4 & 0.92 & 0.01060 \\ 
  GO:0016787 & hydrolase activity & 604 &   7 & 2.89 & 0.01256 \\ 
  GO:0030235 & nitric-oxide synthase regulator activity &   3 &   1 & 0.01 & 0.01429 \\ 
  GO:0044183 & protein binding involved in protein fold... &   3 &   1 & 0.01 & 0.01429 \\ 
  \hline
  \textbf{Biological process} &  &   &   &  &  \\ 
  GO:0002478 & antigen processing and presentation of e... &   7 &   2 & 0.04 & 0.00055 \\ 
  GO:0019886 & antigen processing and presentation of e... &   7 &   2 & 0.04 & 0.00055 \\ 
  GO:0019884 & antigen processing and presentation of e... &   8 &   2 & 0.04 & 0.00073 \\ 
  GO:0002495 & antigen processing and presentation of p... &   9 &   2 & 0.05 & 0.00093 \\ 
  GO:0002504 & antigen processing and presentation of p... &   9 &   2 & 0.05 & 0.00093 \\ 
  GO:0048002 & antigen processing and presentation of p... &  13 &   2 & 0.07 & 0.00199 \\ 
  GO:0019882 & antigen processing and presentation &  15 &   2 & 0.08 & 0.00266 \\ 
  GO:0008219 & cell death & 406 &   7 & 2.16 & 0.00274 \\ 
  GO:0016265 & death & 406 &   7 & 2.16 & 0.00274 \\ 
  GO:0048102 & autophagic cell death &  19 &   2 & 0.10 & 0.00428 \\ 
  \hline
  \textbf{Cellular compartment} &  &   &   &  &  \\ 
  GO:0005768 & endosome & 109 &   4 & 0.48 & 0.00094 \\ 
  GO:0043230 & extracellular organelle &   2 &   1 & 0.01 & 0.00880 \\ 
  GO:0065010 & extracellular membrane-bounded organelle &   2 &   1 & 0.01 & 0.00880 \\ 
  GO:0070062 & extracellular vesicular exosome &   2 &   1 & 0.01 & 0.00880 \\ 
  GO:0043025 & neuronal cell body & 105 &   3 & 0.46 & 0.00951 \\ 
  GO:0000323 & lytic vacuole & 106 &   3 & 0.47 & 0.00976 \\ 
  GO:0044297 & cell body & 109 &   3 & 0.48 & 0.01054 \\ 
  GO:0000328 & fungal-type vacuole lumen &   3 &   1 & 0.01 & 0.01317 \\ 
  GO:0061200 & clathrin sculpted gamma-aminobutyric aci... &   3 &   1 & 0.01 & 0.01317 \\ 
  GO:0061202 & clathrin sculpted gamma-aminobutyric aci... &   3 &   1 & 0.01 & 0.01317 \\ 
  \hline
\end{tabular}
\caption[GO-terms enriched in DE between eel-hosts]{\textbf{GO-terms
    enriched in DE between eel-hosts -} The top 10 enriched
  GO-categories are given for genes DE between the different
  eel-hosts.}
\end{center}
\end{table}


\begin{table}[ht]
\begin{center}
\begin{tabular}{llrrrl}
  \hline
GO.ID & Term & Annotated & Significant & Expected & p-value \\ 
\hline
  \hline
  \textbf{Molecular function} &  &   &   &  &  \\ 
  GO:0016491 & oxidoreductase activity & 189 &   9 & 1.67 & 1.7e-05 \\ 
  GO:0004129 & cytochrome-c oxidase activity &  17 &   3 & 0.15 & 0.00038 \\ 
  GO:0015002 & heme-copper terminal oxidase activity &  17 &   3 & 0.15 & 0.00038 \\ 
  GO:0016676 & oxidoreductase activity, acting on a hem... &  17 &   3 & 0.15 & 0.00038 \\ 
  GO:0016616 & oxidoreductase activity, acting on the C... &  42 &   4 & 0.37 & 0.00042 \\ 
  GO:0004622 & lysophospholipase activity &   4 &   2 & 0.04 & 0.00044 \\ 
  GO:0016675 & oxidoreductase activity, acting on a hem... &  19 &   3 & 0.17 & 0.00054 \\ 
  GO:0016614 & oxidoreductase activity, acting on CH-OH... &  46 &   4 & 0.41 & 0.00060 \\ 
  GO:0004607 & phosphatidylcholine-sterol O-acyltransfe... &   5 &   2 & 0.04 & 0.00074 \\ 
  \hline
  \textbf{Biological process} &  &   &   &  &  \\ 
  GO:0034186 & apolipoprotein A-I binding &   5 &   2 & 0.04 & 0.00074 \\ 
  GO:0046688 & response to copper ion &  25 &   4 & 0.24 & 7.3e-05 \\ 
  GO:0006123 & mitochondrial electron transport, cytoch... &  11 &   3 & 0.11 & 0.00012 \\ 
  GO:0010035 & response to inorganic substance & 233 &   9 & 2.23 & 0.00019 \\ 
  GO:0010038 & response to metal ion & 182 &   8 & 1.74 & 0.00020 \\ 
  GO:0008202 & steroid metabolic process &  64 &   5 & 0.61 & 0.00028 \\ 
  GO:0034370 & triglyceride-rich lipoprotein particle r... &   4 &   2 & 0.04 & 0.00052 \\ 
  GO:0034372 & very-low-density lipoprotein particle re... &   4 &   2 & 0.04 & 0.00052 \\ 
  GO:0009408 & response to heat &  76 &   5 & 0.73 & 0.00063 \\ 
  GO:0009266 & response to temperature stimulus & 117 &   6 & 1.12 & 0.00065 \\ 
  \hline
  \textbf{Cellular compartment} &  &   &   &  &  \\  
  GO:0034375 & high-density lipoprotein particle remode... &   5 &   2 & 0.05 & 0.00087 \\ 
  GO:0034364 & high-density lipoprotein particle &   4 &   2 & 0.03 & 0.00037 \\ 
  GO:0032994 & protein-lipid complex &   5 &   2 & 0.04 & 0.00061 \\ 
  GO:0034358 & plasma lipoprotein particle &   5 &   2 & 0.04 & 0.00061 \\ 
  GO:0031090 & organelle membrane & 505 &  11 & 4.08 & 0.00078 \\ 
  GO:0044421 & extracellular region part & 174 &   6 & 1.41 & 0.00197 \\ 
  GO:0005576 & extracellular region & 250 &   7 & 2.02 & 0.00258 \\ 
  GO:0005739 & mitochondrion & 605 &  11 & 4.89 & 0.00372 \\ 
  GO:0005743 & mitochondrial inner membrane & 162 &   5 & 1.31 & 0.00807 \\ 
  GO:0031967 & organelle envelope & 313 &   7 & 2.53 & 0.00914 \\ 
  GO:0031975 & envelope & 314 &   7 & 2.54 & 0.00930 \\ 
   \hline
 \end{tabular}
 \caption[GO-terms enriched in DE between
 populations]{\textbf{GO-terms enriched in DE between wrom-populations
     -} The top 10 enriched GO-categories are given for genes DE
   between the different worm populations.}
\end{center}
\end{table}


\figuremacro{pop_all_heat}{Variance/mean stabilized expression values
  for contigs different between populations}{green indicates
  expression below the mean, red above the mean}



\section{Single gene differences}
\label{sec:single-gene-diff}

%% print(eel.detail, hline.after=1:length(ortho.l[[2]])*3)
% latex table generated in R 2.14.0 by xtable 1.6-0 package
% Sun Dec 11 18:21:30 2011
\begin{table}[ht]
\begin{center}
\begin{tabular}{p{5cm}rrrr}
  & Aa:EU & Aa:TW & Aj:EU & Aj:TW \\ 
 Contig1005.mean & 518.35 & 630.47 & 1512.31 & 831.26 \\ 
  Cytochrome P450 family protein & 1123.86 & 1204.98 & 2647.29 & 1620.76 \\ 
  ce.ortho.mean & 557.65 & 662.20 & 1658.80 & 1004.08 \\ 
   \hline
Contig12201.mean & 514.90 & 549.58 & 116.02 & 99.56 \\ 
  Lipase family protein & 502.48 & 553.48 & 119.47 & 101.09 \\ 
  ce.ortho.mean1 & 501.19 & 549.00 & 119.20 & 99.67 \\ 
   \hline
Contig26.mean & 11007.58 & 5406.06 & 3206.43 & 2541.48 \\ 
  Aspartic protease BmAsp-1, identical & 12994.14 & 7671.50 & 4466.98 & 4926.97 \\ 
  ce.ortho.mean2 & 12670.54 & 7237.48 & 4206.98 & 4402.80 \\ 
   \hline
Contig3754.mean & 490.23 & 901.35 & 922.95 & 663.19 \\ 
  MGC79044 protein, putative & 660.74 & 1110.31 & 1180.48 & 884.49 \\ 
  ce.ortho.mean3 & 488.55 & 883.91 & 971.48 & 682.95 \\ 
   \hline
Contig3896.mean & 123.17 & 85.71 & 109.09 & 60.18 \\ 
  Transcription factor AP-2 family protein & 119.36 & 86.89 & 111.08 & 59.46 \\ 
  ce.ortho.mean4 & 119.08 & 85.79 & 111.17 & 58.87 \\ 
   \hline
Contig566.mean & 642.74 & 484.47 & 337.05 & 691.06 \\ 
  Eukaryotic aspartyl protease family protein & 651.38 & 496.17 & 377.95 & 733.26 \\ 
  ce.ortho.mean5 & 654.89 & 491.93 & 381.14 & 724.47 \\ 
   \hline
Contig6778.mean & 39.00 & 768.10 & 1028.40 & 92.46 \\ 
  Nematode cuticle collagen N-terminal domain containing protein & 621.79 & 1259.66 & 1508.45 & 447.50 \\ 
  ce.ortho.mean6 & 38.62 & 752.61 & 1056.15 & 95.26 \\ 
   \hline
Contig6934.mean & 449.66 & 639.22 & 632.23 & 572.12 \\ 
  Serine/threonine-protein phosphatase & 788.16 & 1133.91 & 1236.79 & 1041.83 \\ 
  ce.ortho.mean7 & 448.17 & 628.16 & 663.55 & 591.01 \\ 
   \hline
Contig7580.mean & 240.34 & 1318.57 & 2215.65 & 38.30 \\ 
  Cuticular collagen Bmcol-2 & 286.57 & 1490.40 & 2531.07 & 227.23 \\ 
  ce.ortho.mean8 & 231.55 & 1298.61 & 2272.71 & 38.23 \\ 
   \hline
\end{tabular}
\end{center}
\end{table}

%% print(pop.detail, hline.after=1:length(ortho.l[[3]])*3)
% latex table generated in R 2.14.0 by xtable 1.6-0 package
% Sun Dec 11 18:22:04 2011
\begin{table}[ht]
\begin{center}
\begin{tabular}{p{5cm}rrrr}
  & Aa:EU & Aa:TW & Aj:EU & Aj:TW \\ 
 Contig13267.mean & 103.86 & 38.57 & 111.01 & 83.54 \\ 
  ABC transporter family protein & 101.36 & 37.67 & 114.79 & 94.25 \\ 
  ce.ortho.mean & 101.74 & 37.76 & 115.19 & 89.28 \\ 
   \hline
Contig157.mean & 362.46 & 394.14 & 369.26 & 449.27 \\ 
  Probable 3-hydroxyacyl-CoA dehydrogenase B0272.3, putative & 361.60 & 378.14 & 381.70 & 545.36 \\ 
  ce.ortho.mean1 & 362.40 & 367.51 & 380.95 & 504.83 \\ 
   \hline
Contig2099.mean & 289.41 & 327.82 & 367.54 & 556.00 \\ 
  Malate/L-lactate dehydrogenase family protein & 316.68 & 360.99 & 418.67 & 754.71 \\ 
  ce.ortho.mean2 & 319.36 & 357.47 & 421.73 & 699.56 \\ 
   \hline
Contig236.mean & 266.65 & 164.76 & 183.18 & 840.76 \\ 
  Lecithin:cholesterol acyltransferase family protein & 2797.98 & 2969.10 & 2306.91 & 6119.67 \\ 
  ce.ortho.mean3 & 2716.28 & 2886.46 & 2225.58 & 5278.32 \\ 
   \hline
Contig2442.mean & 284.39 & 360.83 & 521.53 & 408.18 \\ 
  Putative uncharacterized protein & 782.07 & 1102.11 & 1432.12 & 960.61 \\ 
  ce.ortho.mean4 & 797.22 & 1131.03 & 1448.22 & 970.06 \\ 
   \hline
Contig2531.mean & 21.38 & 53.89 & 25.65 & 35.20 \\ 
  Cutical collagen 6, putative & 20.78 & 52.54 & 26.07 & 37.82 \\ 
  ce.ortho.mean5 & 20.86 & 51.95 & 26.08 & 36.53 \\ 
   \hline
Contig3453.mean & 269.89 & 209.33 & 277.53 & 1032.13 \\ 
  Lecithin:cholesterol acyltransferase family protein1 & 2797.98 & 2969.10 & 2306.91 & 6119.67 \\ 
  ce.ortho.mean6 & 2716.28 & 2886.46 & 2225.58 & 5278.32 \\ 
   \hline
Contig566.mean & 642.74 & 484.47 & 337.05 & 691.06 \\ 
  Eukaryotic aspartyl protease family protein & 651.38 & 496.17 & 377.95 & 733.26 \\ 
  ce.ortho.mean7 & 654.89 & 491.93 & 381.14 & 724.47 \\ 
   \hline


\begin{table}[ht]
\begin{center}
\begin{tabular}{p{5cm}rrrr}
  
Contig6043.mean & 1003.44 & 841.34 & 942.26 & 631.00 \\ 
  Putative uncharacterized protein1 & 977.73 & 834.03 & 964.85 & 670.11 \\ 
  ce.ortho.mean8 & 978.45 & 823.82 & 967.65 & 647.85 \\ 
   \hline
Contig6386.mean & 68.17 & 31.29 & 68.01 & 48.09 \\ 
  Matrixin family protein & 66.79 & 30.60 & 69.64 & 53.52 \\ 
  ce.ortho.mean9 & 72.76 & 36.38 & 72.47 & 55.31 \\ 
   \hline
Contig6759.mean & 47.39 & 12737.30 & 115.48 & 28013.11 \\ 
  Cytochrome c oxidase subunit 2 & 5647.97 & 19163.28 & 9116.07 & 43335.23 \\ 
  ce.ortho.mean10 & 5865.67 & 19455.08 & 9437.50 & 41673.94 \\ 
   \hline
Contig6778.mean & 39.00 & 768.10 & 1028.40 & 92.46 \\ 
  Nematode cuticle collagen N-terminal domain containing protein & 621.79 & 1259.66 & 1508.45 & 447.50 \\ 
  ce.ortho.mean11 & 38.62 & 752.61 & 1056.15 & 95.26 \\ 
   \hline
Contig6934.mean & 449.66 & 639.22 & 632.23 & 572.12 \\ 
  Serine/threonine-protein phosphatase & 788.16 & 1133.91 & 1236.79 & 1041.83 \\ 
  ce.ortho.mean12 & 448.17 & 628.16 & 663.55 & 591.01 \\ 
   \hline
Contig7580.mean & 240.34 & 1318.57 & 2215.65 & 38.30 \\ 
  Cuticular collagen Bmcol-2 & 286.57 & 1490.40 & 2531.07 & 227.23 \\ 
  ce.ortho.mean13 & 231.55 & 1298.61 & 2272.71 & 38.23 \\ 
   \hline
Contig8758.mean & 390.97 & 715.11 & 602.46 & 494.53 \\ 
  Protein B0207.11, putative & 383.10 & 687.32 & 626.45 & 510.14 \\ 
  ce.ortho.mean14 & 389.74 & 701.10 & 633.78 & 511.74 \\ 
   \hline
\end{tabular}
\end{center}
\end{table}





%%% Local Variables: ***
%%% mode:latex ***
%%% TeX-master: "../thesis.tex"  ***
%%% tex-main-file: "../thesis.tex" ***
%%% End: ***
