% this file is called up by thesis.tex
% content in this file will be fed into the main document

%: ----------------------- name of chapter  -------------------------
\chapter{Evaluation of an assembly strategy for pyrosequencing
  reads} % top level followed by section, subsection
\label{chap:eval-ass}

%: ----------------------- paths to graphics ------------------------

% change according to folder and file names
\ifpdf
    \graphicspath{{4_eval_ass/figures/PNG/}{4_eval_ass/figures/PDF/}{4_eval_ass/figures/}}
\else
    \graphicspath{{4_eval_ass/figures/EPS/}{4_eval_ass/figures/}}
\fi

%: ----------------------- contents from here ------------------------


\section{Overview}
\label{sec:over-eval}

This chapter reports on an important methodical detail of chapter
\ref{cha:pyro}: the sequence-assembly. The quality of this sequence
assembly constitutes a fundamental foundation of the later chapters.

The pre-processed \textit{A. crassus} data-set consisting of
100,491,819 bases in 353,055 reads (58,617 generated using
``FLX-chemistry'', 294,438 using ``Titanium-chemistry'') was assembled
following an approach proposed by \cite{pmid20950480}: two assemblies
were generated, one using \texttt{Newbler v2.6} \cite{pmid16056220},
the other using \texttt{Mira v3.2.1} \cite{miraEST}. The resulting
assemblies (referred to as first-order assemblies) were merged with
\texttt{Cap3} \cite{Cap3_Huang} into a combined assembly (referred to
as second-order assembly).

Summary statistics for the assemblies, demonstrating the superiority
of the second-order assembly are reported as well as summary
statistics for single contigs. These metadata on contigs are important
for the evaluation of downstream results. As a perfect assembly with
each contig representing a single full transcript is illusive and
every contig constitutes a hypothesis, it becomes important to
validate and question analyses based on as much information as
possible. Thus a comprehensive set of assembly derived statistics is
presented.

\section{The \texttt{Newbler} first-order assembly}
\label{sec:new-fist}

During transcriptome-assembly \texttt{Newbler} can split individual
reads spanning the breakpoints of alternate isoforms, to assemble, for
example, the first portion of the reads in one contig, the second
portion in two different contigs. Later multiple so called isotigs
would be constructed and reported, one for each putative
transcript-variant. While this approach could be helpful for the
detection of alternate isoforms, it also produces short contigs
(especially at error-prone edges of high-coverage transcripts) when
the building of isotigs fails. The read-status report and the assembly
output in ace-format the program provides include short contigs only
used during the assembly-process, but not reported in the contigs-file
used in transcriptome-assembly projects
(\texttt{454Isotigs.fna}). Therefore to get all reads not included in
contigs (i.e. a consistent definition of ``singleton'') it was
necessary to add all reads appearing only in contigs not reported in
the fasta-file to the reported singletons. The number of singletons
increased in this step from the 26,211 reported to 109,052. I later
also address the usefulness of \texttt{Newbler's} report vs. the
expanded singleton-category, but in the meantime I define singletons
as all reads not present in a given assembly.

\figuremacro{split_454}{Number of contigs/isotigs split}{A histogram
  of the number of contigs or isotigs \texttt{Newbler} split a single
  read into.}

As mentioned above, the splitting of reads in the \texttt{Newbler}
assembly can give useful information on possible isoforms, however,
the number of contigs \texttt{Newbler} split one read into (in some
cases more than 100 contigs) seems artificially inflated (see figure
\ref{split_454}). If information would correspond to real isoforms it
should be about an order of magnitude lower. This fact emphasises the
need for further processing of the contigs. The maximum number of
read-splits in a given contig and its usefulness will be discussed
later in greater detail.

\section{The \texttt{Mira}-assembly and the second-order assembly}
\label{sec:assembly-sec}

The \texttt{Mira}-assembly provided a second estimate of the
transcriptome. In this assembly individual reads are not split. The
number of reads not used in the \texttt{Mira}-assembly was 65368.

To combine the two assemblies \texttt{cap3} was used with default
parameters and including the quality information from first-order
assemblies. The reminder of this chapter deals with the exploratory
analysis of how information from both estimates of the transcriptome
are integrated into the final second-order assembly.

% latex table generated in R 2.13.0 by xtable 1.5-6 package
% Tue Nov 15 23:52:26 2011
\begin{table}[ht]
\begin{center}
\begin{tabular}{rrrr}
  \hline
 & Newbler & Mira & Second-order(MN) \\ 
  \hline
Max length & 6,300 & 6,352 & 6,377 \\ 
  Number of contigs & 15,934 & 22,596 & 14,064 \\ 
  Number of Bases & 8,085,922 & 12,010,349 & 8,139,143 \\ 
  N50 & 579 & 579 & 662 \\ 
  Number of contigs in N50 & 4,301 & 6,749 & 3,899 \\ 
  non ATGC bases & 375 & 29,962 & 5,245 \\ 
  Mean length & 508 & 532 & 579 \\ 
   \hline
\end{tabular}
\caption[Statistics for the first-order assemblies]{\textbf{Statistics
    for the first-order assemblies} - Basic statistics for the
  first-order assemblies and the second-order assembly (for which only
  the most reliable category of contigs (MN) is shown; see
  \ref{sec:data-categ-second}).}
\label{tab:pc}
\end{center}
\end{table}


Table \ref{tab:pc} gives basic summary-statistics of the different
assemblies. \texttt{Mira} clearly produced the biggest assembly, both
in terms of number of contigs and bases. The second-order assembly is
of slightly smaller size than the \texttt{Newbler} assembly. The
second-order assembly had on average longer contigs than both
first-order assemblies and a higher weighted median contig size (N50).

\section{Data-categories in the second-order assembly}
\label{sec:data-categ-second}

Three main categories of assembled sequence data can be distinguished
in the second-order assembly, with different reliability and purpose
in downstream applications: The first category of data obtained are
the singletons of the final second-order assembly. It comprises raw
sequencing reads that neither of the first-order assemblers used. It
is therefore the intersection of the \texttt{Newbler}-singletons (as
defined in \ref{sec:new-fist}) and the
\texttt{Mira}-singletons. 47,669 reads fell into this category. A
second category of sequence contains the first-order contigs which
could not be assembled in the second-order assembly (the singletons in
the \texttt{cap3}-assembly; M\_1 and N\_1 in table
\ref{tab:categ}). Furthermore, second-order contigs in which
first-order contigs from only one assembler are combined (M\_n and
N\_n in table \ref{tab:categ}) also have to be included in this
category. Sequences in this category should be considered only
moderately reliable as they are supported by only one assembly
algorithm.

Finally the category of contigs considered most reliable contains all
second-order contigs with contributions from both first-order
assemblies (MN in table \ref{tab:categ}). For this last, most reliable
(MN) category, reads contained in the assembly can be categorised
depending on whether they entered the assembly via both or only via
one first-order assembly.

% latex table generated in R 2.13.0 by xtable 1.5-6 package
% Tue Nov 15 23:52:34 2011
\begin{table}[ht]
\begin{center}
\begin{tabular}{rlllll}
  \hline
 & M\_1 & M\_n & MN & N\_n & N\_1 \\ 
  \hline
Snd.o.con &   & 164 & 13887 & 13 &   \\ 
  Fst.o.con & 2347 & 897 & Mira=19352/Newbler=14410 & 40 & 1484 \\ 
  reads & 42172 & 21153 & one=269868/both=193308 & 1538 & 13100 \\ 
   \hline
\end{tabular}
\caption[Number of reads in assemblies]{\textbf{Number of reads in
    assemblies} - For first-order contigs (Fst.o.con) and second-order
  contigs (Snd.o.con) numbers for different categories of contigs are
  given: M\_1 and N\_1 = first-order contigs not assembled in
  second-order assembly, from \texttt{Mira} and \texttt{Newbler}
  respectively; M\_n and N\_n = assembled in second-order contigs only
  with contigs from the same first-order assembly; MN = assembled in
  second-order contigs with first order contigs from both first order
  assemblies.}
\label{tab:categ}
\end{center}
\end{table}

\figuremacro{read_origin}{Origin of reads}{Reads in the most reliable
  (MN) assembly-category are categorised by the way they entered the
  assembly: Although they are in a highly credible contig, reads can
  still have entered from only one first order assembly (Mira\_in\_MN
  or Newbler\_in\_MN). The intersection gives the reads which entered
  via both routes. The duplicated category gives the number of reads
  split by \texttt{Newbler} and the intersection reads, which were
  split and entered the assembly.}

Figure \ref{read_origin} gives a more detailed view of the fate of the
reads \texttt{Newbler} split during first-order
assembly. Interestingly, most reads \texttt{Newbler} split ended in
the high-quality category of the second order assembly only.

\section{Contribution of first-order assemblies to second-order contigs}
\label{sec:contr-firs-order}

Looking at the contribution of contigs from each of the assemblies to
one second-order contig in figure \ref{ass_contributions}a it becomes
clear that the \texttt{Mira}-assembly had a high number of redundant
contigs. These were assembled into the same contig by \texttt{Newbler}
and finally also in one second-order contig by \texttt{Cap3}.

\figuremacro{ass_contributions}{Contribution to second-order assembly}
{Number of first-order contigs from both first-order assemblies for
  each second order contig (a) number of reads through
  \texttt{Newbler} and \texttt{Mira} for each second-order contig
  (b).}

A different picture emerges from the contribution of reads through
each of the first-order assemblies (figure
\ref{ass_contributions}b). Here, for most second-order contigs many
more reads are contributed through \texttt{Newbler}-contigs. This is
because \texttt{Newbler} has more reads summed over all contigs caused
by the duplication due to the splitting of reads.

\section{Evaluation of the assemblies}
\label{sec:eval-three-assembl}

To further compare assemblies (\texttt{Mira, Newbler} first-order
assemblies including or excluding their singletons) and the
second-order assembly (including different contigs-categories and
singletons) I evaluated the number of bases or proteins their contigs
and singletons (partially) cover in the related model-nematodes,
\textit{Caenorhabditis elegans} and \textit{Brugia malayi}.

In addition, the size of the assembly can give an indication of
redundancy or artificially assembled data. If it increases without
improving the reference-coverage the dataset is likely to contain more
redundant or artificial information, a more parsimonious assembly
should be preferred.

The database-coverage for the two reference species can then be
plotted against the size of the assembly-dataset to estimate the
completeness conditional to the size of the assembly (figures
\ref{base_ref_b}, \ref{base_ref_p}, \ref{base_ref_p}).

From the assemblies excluding singletons (in the lower left corner
with lower size and database-coverage) the highly reliable
contig-category of the second-order assembly produced the highest
per-base coverage in both reference-species, with the \texttt{Newbler}
assembly in second place and \texttt{Mira} producing the lowest
reference-coverage. When adding the contigs considered lower quality
supported by only one assembler to the second-order assembly the
reference-coverage increased moderately.

Including singletons the \texttt{Mira} and \texttt{Newbler} assemblies
were of increased size. A comparison of the \texttt{Newbler's}
reported singletons with all singletons added to the
\texttt{Newbler}-assembly shows that the reported singletons increased
reference-coverage to the same amount as all singletons, while the
non-reported singletons only increased the size of the assembly. It
can be concluded that the latter contain hardly any additional
information but only error-prone or variant reads.

The second-order assembly including the intersection of first-order
singletons performed similarly to the \texttt{Newbler} assembly for
the number of bases covered, but was larger in size. Adding the less
reliable set of one-assembler supported second-order-contigs the
assembly covered the most bases in both references. When the singleton
of the second-order assembly (as defined in \ref{sec:new-fist}) were
not included but only the intersection of \texttt{Newbler's}
``reported singletons'' and \texttt{Mira's} singletons, a very
parsimonious assembly with high reference-coverage (termed fullest
assembly; and labeled FU in the plots above) was obtained.


\figuremacro{base_ref_b}{Base-content and reference-transcriptome
  coverage in percent of bases}{for different assemblies and
  assembly-combinations; M = \texttt{Mira}; N = \texttt{Newbler};
  $M+S$ = \texttt{Mira} + singletons; $N+S$ = \texttt{Newbler} plus
  singletons; $N+rS$ = \texttt{Newbler} plus singletons reported in
  readstatus.txt; MN = second-order contigs supported by both
  first-order; $MN+N\_x$ = second-order MN plus contigs only supported
  by \texttt{Newbler} ($N\_x$ = $N\_n$ and $N\_1$); $MN+M\_x$ = same
  for \texttt{Mira}-first-order-contigs; $MN+M\_x+S$ and $MN+N\_x+S$
  same with singletons; FU = second-order contigs supported by both or
  one assembler plus the intersection of \texttt{Newbler} reported
  singletons and \texttt{Mira}-singletons = the basis for the
  ``fullest assembly'' used in later analyses}


\figuremacro{base_ref_p}{Base-content and reference-transcriptome
  coverage in percent of proteins hit}{in percent of proteins hit for
  different assemblies and assembly-combinations (for
  category-abbreviations see figure \ref{base_ref_b})}

Considering the reference-database with any kind of coverage the
second-order assembly performed less well. Excluding singletons it
covered similar numbers of database-proteins to the
\texttt{Newbler}-assembly and and was outperformed by the
\texttt{Mira}-assembly, although the latter was again shown to be
least parsimonious. The same general picture emerged from this
analysis when singletons were considered
additionally. \texttt{Newbler} and second-order assemblies covered
similar amounts of reference-data.

\figuremacro{base_ref_8}{Base-content and reference-transcriptome
  coverage in percent of proteins covered to at least 80\%}{of their
  length for different assemblies and assembly-combinations (for
  category-abbreviations see figure \ref{base_ref_b})}

When database-proteins covered for at least 80\% of their length are
considered, the second-order assembly showed its superiority: both ex-
and including singletons the second-order assembly outperformed the
first-order assemblies. Moderate gains in reference coverage were made
again for the addition of dubious single-assembler supported
second-order contigs. I give most weight in my analysis to these
results as in average longer correct contigs will allow finding the
highest number of putative full-length genes.

Given this evaluation I defined a ``minimal adequate'' assembly as the
subset of contigs of the second-order assembly supported by both
assemblers (labeled MN above). Given the performance of the singletons
\texttt{Newbler} reported. I defined a ``fullest-assembly'' as all
second-order contigs (including those supported by only one assembler)
plus the intersection of reported \texttt{Newbler}-singletons and
\texttt{Mira} singletons.

\section{Measurements on second-order assembly}

Based on the tracking of reads through the complicated assembly
process, I calculated the following statistics for each contig in the
second-order assembly.

\begin{itemize}
\item number of \texttt{Mira} and \texttt{Newbler} first-order contigs
\item number of reads through \texttt{Mira} and reads through \texttt{Newbler}
\item number of reads being split by \texttt{Newbler} in first-order
  assembly 
\item number of read-split events in the first-order assembly (equals
  the sum of reads multiplied by number of contigs a read has been
  split into)
\item maximal number of first-order contigs a read in the contig has
  been split into during \texttt{Newbler}-assembly 
\item the number of same-read-pairs from the \texttt{Newbler} and
  \texttt{Mira} first order-assembly merged in a second order contig
\item cluster-id of the contig: All contigs ``connected'' by sharing
  reads were assigned the same id (similar to the graph clustering
  reported in \cite{pmid21138572}).
\item number of other second order contigs containing the same read
  (size of the cluster)
\end{itemize}


\subsection{Contig coverage}

As well defined coverage-information is not readily available from the
output of this combined assembly approach (although I followed
individual reads through the process) I inferred coverage by mapping
the reads used for assembly against the fullest assembly using
\texttt{ssaha2} \cite{pmid11591649} and obtained:

\begin{itemize}
\item mean per base coverage
\item mean unique per base coverage
\end{itemize}

The ratio of mean per base coverage and unique per base coverage (the
standard for assessing coverage) can be used as to asses the redundancy
of a contig.

\subsection{Example use of the contig-measurements}

Based on these measurements the emergence of a given contig from the
assembly process can be reconstructed. Table \ref{tab:ex-me} gives an
excerpt of the contig-measurements reported in additional-file
\texttt{contig-data.csv}. The example contigs are all from large
contig-clusters (cluster.size), where interpretation of the assembly
history is complicated, but not impossible:

% latex table generated in R 2.13.0 by xtable 1.5-6 package
% Tue Nov 15 23:54:13 2011
\begin{table}[ht]
\begin{center}
\begin{tabular}{rllll}
  \hline
 & Contig1047 & Contig10719 & Contig104 & Contig13672 \\ 
  \hline
reads\_through\_Newbler &   16 & 1351 &    0 &   14 \\ 
  reads\_through\_Mira &  26 & 651 & 135 &   0 \\ 
  Newbler\_contigs & 1 & 5 & 0 & 2 \\ 
  Mira\_contigs & 1 & 9 & 4 & 0 \\ 
  category & MN & MN & M\_n & N\_n \\ 
  num.new.split &    8 & 1314 &    0 &    0 \\ 
  sum.new.split &   16 & 2628 &    0 &    0 \\ 
  max.new.split & 2 & 2 & 0 & 0 \\ 
  num.SndO.pair &  13 & 644 &   0 &   0 \\ 
  cluster.id & CL62 & CL6 & CL176 & CL235 \\ 
  cluster.size & 24 & 18 &  5 &  5 \\ 
  coverage &   4.200342 & 267.495458 &  41.003369 &   2.920755 \\ 
  uniq\_coverage & 4.248960 & 7.425507 & 2.568000 & 1.196078 \\ 
   \hline
\end{tabular}
\caption[Example for assembly-measurements]{\textbf{Example for
    assembly-measurements} - Measurements on contigs (as given in
  additional file \texttt{contig-data.csv}), row-labels are explained
  in a detailed example in the main text}
\label{tab:ex-me}
\end{center}
\end{table}
\textbf{Contig1047} is in the well trusted MN category of contigs. It
consists of only one contig from each first-order assembly
(Newbler\_contigs and Mira\_contigs), each containing a set of reads
of moderate size: 16 from \texttt{Newbler} (reads\_through\_Newbler)
26 from \texttt{Mira} (reads\_through\_Mira). 8 of the 16 reads
\texttt{Newbler} used in its one assembled contig were also assembled
to a different \texttt{Newbler}-contig (num.new.split). That each of
the 8 reads was only appearing in one other \texttt{Newbler}-contig is
visible from the fact, that the number of split events is 16
(sum.new.split) and the maximal number of splits for one read is 2
(max.new.split). 13 (num.SndO.pair) same-read-pairs from the two
different first-order assemblies were merged in this second-order
contig, leaving 3 (16-13) reads in \texttt{Newbler}-contigs and 13
(26-13) reads in \texttt{Mira} contigs, which all could potentially
have ended up in other contigs. The contig is in a cluster (CL62),
which contains in total 24 contigs (cluster.size). It has to be
admitted that the whole graph-structure linking this 24 contigs can't
be reconstructed from this contig summary data. On the other hand the
summary data makes clear, from what source the links for
cluster-affiliation have resulted: In this case from 3 and 13 unlinked
read-pairs from both first-order assemblies and 8 split-reads from
\texttt{Newbler}-fist order contigs.

A comprehensive interpretation of the other example-contigs depicted
is left to the reader. It should just be remarked that in case of
one-assembler supported contigs, all reads in that contig could
potentially be represented in other contigs, making average
cluster-size in these contigs bigger than in the MN category.

One of the most interesting measurement calculated for each contig is
the cluster-membership and cluster-size. Such clusters can represent
close paralogs, duplicated genes, isoforms from alternative splicing
or allelic variants. Cluster size correlates as expected with the
ratio of unique/non-unique coverage, as contigs in clusters contain
redundant sequences also found in other contigs.

These measurements were used in all later analyses to evaluate
likelihood of misassembly artefacts as an influence on a given set of
biological relevant contigs. All gene-sets mentioned later (in chapter
\ref{cha:pyro}) were thus, as a matter of routine, controlled for
unusual patterns in the contig meta-data.

\section{Finalising the fullest assembly set}
\label{sec:final-full-assembly}

As additional measure in order to minimise the amount of sequence with
artificially inferred isoform-breakpoints, I used the
unique-mapping-information described above to detect contigs and
singletons not supported by any raw data (reads). Table
\ref{tab:cov.ex} gives a summary of these unsupported data by
contig-category. For all downstream-analysis I removed all well
trusted MN-category contigs having no coverage at all and the contigs
(and singletons) from other categories having no unique coverage.

% latex table generated in R 2.13.0 by xtable 1.5-6 package
% Tue Nov 15 23:54:14 2011
\begin{table}[ht]
\begin{center}
\begin{tabular}{rrrrrrr}
  \hline
  & singletons & M\_1 & M\_n & MN & N\_1 & N\_n \\ 
  \hline
  coverage == 0 & 546 &  34 &  2 &  36 & 158 &  0 \\ 
  unique coverage == 0 & 584 &  48 &   2 &  \textbf{42(-36)} & 210 &   3 \\ 
  \hline
\end{tabular}
\caption[Final filtering of the assembly]{\textbf{Final filtering of
    the assembly} - Number of contigs with a coverage and
  unique-coverage of zero, inferred from mapping of raw reads, listed
  by contig-category. Only the contigs in bold listed here were not
  screened from the assembly (7 MN-contigs).}
\label{tab:cov.ex}
\end{center}
\end{table}

Thereby I reduced my dataset to 40187 tentative unique genes (TUGs),
redefining the ``fullest assembly'' dataset. Based on the above
evaluation I decided to treat the MN-category of contigs as high
credibility assembly (highCA) and to subsume the M\_n, N\_n, M\_1,
N\_1 and \texttt{Newbler's} reported singletons as additional low
credibility assembly (lowCA).

%%% Local Variables: ***
%%% mode:latex ***
%%% TeX-master: "../thesis.tex"  ***
%%% tex-main-file: "../thesis.tex" ***
%%% End: ***
