% this file is called up by thesis.tex
% content in this file will be fed into the main document

\chapter{Discussion} % top level followed by section, subsection


% ----------------------- paths to graphics ------------------------

% change according to folder and file names
\ifpdf
    \graphicspath{{7/figures/PNG/}{7/figures/PDF/}{7/figures/}}
\else
    \graphicspath{{7/figures/EPS/}{7/figures/}}
\fi


% ----------------------- contents from here ------------------------

\section{Sanger-method pilot-sequencing}
\label{sec:sanger-pil}

In was not achieved to reproducibly alleviate the rRNA-levels in
libraries prepared for sequencing.

This has probably been due to the fact that extraction of total-RNA
from worms filled with host blood resulted in low amounts of starting
material, and amplification using standard kits did not allow to
reproducibly alleviate rRNA abundance. As the same problems existed in
preparation of liver tissue of the host species it seems likely that
the blood of eels contains substances limiting the success of specific
amplification protocols. In fact it is known that compounds like
hemoglobin can inhibit PCR reactions \cite{pmid9327537} and reverse
transcription \cite{pmid16109794}.


Nevertheless the stringent quality trimming and processing of raw
reads, as summarized in \ref{pilot-seq}, made the remaining ESTs a
valuable resource for comparison with future 454-sequencing-data.



\section{454-pyrosequencing}
\label{sec:454-pyr}


We are providing transcriptome-data for the parasite
\textit{A. crassus}, enabling a broad spectrum of molecular research
on this ecologically and economically important species.

We emphasize the importance of screening for xenobiotics. We consider
this aspect important in any deep transcriptome project. First the
depth of sequencing is leading to the generation of large amounts of
off-target data from a ``metatranscriptomic community'' associated with a
target organism. Second due to the abundance of laboratory
contamination and the possibility of cross-contamination if libraries
are sequenced only on a subset of a picotiter-plate (i.e. without the
use of barcodes distinguishing between samples \cite{pmid20137071})
non-biological contamination can be introduced. 
However, in the context of a parasite (or an infected host) the
screening for off-target data and contamination becomes even more
important: Correct inference of biological origin for a given contig
constitutes a prerequisite for most downstream analysis or the
interpretation of results.

Cross-contamination from different compartments of a picolitre-plate
was ruled out by our sequence provider, using Multiplex Indexes (MID)
for one library and similarity searches to neighboring lanes for the
other libraries.

For the remaining off-target and contamination problem we archived
separation of sequences in two steps, one before assembly, one
afterward. Both screening-steps had to rely solely on sequence
comparison. The screening-step before assembly has to employ lower
stringency as sequence comparisons on sequence as short as reads are
less informative than on longer contig-sequence. In our case of
\textit{A. crassus}, neither of the two host species has genomic data
available for use in similarity searches. A publicly available
transcriptome-data-set for European eel \cite{pmid21080939} in addition
to a unpublished data-set for the same species was augmented with a
data-set generated from the Japanese eel sequenced for the purpose of
screening \textit{A. crassus}-sequences in the present project. The
pre-assembly screening had the rationale of facilitating the assembly
process reducing the amount of divergent sequence from two
host-species and the amount of extensively covered rRNA sequence. In
our sequencing we were not able to reproducibly alleviate the rRNA
coverage. This has probably been due to the fact that extraction of
total-RNA from worms filled with host blood resulted in low amounts of
starting material, and amplification using standard kits did not allow
to reproducibly alleviate rRNA abundance. As the same problems
existed in preparation of liver tissue of the host species it seems
likely that the blood of eels contains substances limiting the success
of specific amplification protocols. In fact it is known that
compounds like hemoglobin can inhibit PCR reactions \cite{pmid9327537}
and reverse transcription \cite{pmid16109794}.

Although raw reads with rRNA hits were screened out prior to assembly,
it was still possible to gain insights from these off-target data, as
we assembled and annotated screening databases. Some of the rRNA data
especially from the L2 library showed high similarity to flagellate
eukaryotes. It could be possibly derived from an unknown protist
living in the swimbladder of eels (possibly as a commensal of
\textit{A. crassus}), from where the L2 larvae for RNA-preparation
were washed out. This seems worth further investigation, especially as
it has been controversial whether encapsulated objects in the
swimbladder of eels could be attributed solely to \textit{A. crassus}
\cite{heitlinger_massive_2009} or to opportunist coinfections.
 
We were able to demonstrate, that screening of SNPs in or adjacent to
homopolymer regions ``improved'' overall measurements on SNP-quality:

First the ratio of transitions to transversions (ti/tv) increased.
Such an increase is explainable by the removal of ``noise'' associated
with common homopolymer-errors \cite{pmid21685085}. Assuming that
errors would be independent of transversion-transition bias erroneous
SNPs would have a ti/tv of 0.5 and thereby lower the overall
value. Other explanations for these observations are hard to find so
it can be concluded that removing noise from homopolymer
sequencing-error ti/tv increases.  The value of XXX XXXX outside, XXX
inside ORFs) is in good agreement with the overall ti/tv of humans
(2.16) or \textit{Drosophila} (2.07 \cite{pmid21143862}).

The ratio of non-synonymous SNPs per non-synonymous site to synonymous
SNPs per synonymous site (dn/ds) decreased with removal of SNPs
adjacent to homopolymer regions from XXX  to
XXXX after full screening. Similar to ti/tv it the most
plausible explanation is the removal of error, as unbiased error would
lead to a dn/ds of 1. While dn/ds is not unproblematic to interpret
within populations \cite{pmid19081788}, assuming negative (purifying)
selection on most protein-coding genes lower values seem more
plausible, also in comparison with other studies (see further text).

We used a threshold value for the minority allele of 7\% for exclusion
of SNPs, this corresponds to the ca. 10 ``haploid equivalents'' (5
individual worms plus an negligible amount of L2 larvae - in the L2
library and within the female adult worms - bearing possibly
additional diversity). It is hard to explain, that ti/tv decreased in
this filtering step, while dn/ds still further decreased.

The benefit of this screening was mainly a reduction of non-synonymous
SNPs in high coverage contigs. When it was applied dn/ds did not
scale with coverage. Working with an estimate of dn/ds independent of
coverage, efforts to control for sampling a biased by sampling depth
(i.e. coverage) like developed \cite{pmid18590545} and used
\cite{pmid20478048} could be avoided.  

\section{Experimental infections}
\label{sec:exp-inf}

Such experiments have their problems because environmental factors,
such as the general quality of the environment (i.e. water
temperature) can interact with the host-environment
\cite{kaltz_shykoff_rev}.

%%% Local Variables: ***
%%% mode:latex ***
%%% TeX-master: "../thesis.tex"  ***
%%% tex-main-file: "../thesis.tex" ***
%%% End: ***
